{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 10,
   "id": "d5b4b5aa",
   "metadata": {},
   "outputs": [],
   "source": [
    "import table_utils\n",
    "import process_paper_utils as utils\n",
    "import os\n",
    "from bs4 import BeautifulSoup\n",
    "import html\n",
    "from datetime import datetime\n",
    "import pandas as pd"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "4c9c2d1b",
   "metadata": {},
   "outputs": [],
   "source": [
    "#Enter PMC here:\n",
    "PMC = \"PMC5514907\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "id": "80847829",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "<?xml version=\"1.0\" encoding=\"UTF-8\"?><!DOCTYPE collection SYSTEM \"BioC.dtd\"><collection><source>BioC-API</source><date>20250708</date><key>collection.key</key><document><id>PMC7767363</id><infon key=\"license\">CC BY</infon><passage><infon key=\"article-id_doi\">10.3390/ijms21249764</infon><infon key=\"article-id_pmc\">PMC7767363</infon><infon key=\"article-id_pmid\">33371333</infon><infon key=\"article-id_publisher-id\">ijms-21-09764</infon><infon key=\"elocation-id\">9764</infon><infon key=\"issue\">24</infon><infon key=\"kwd\">HER2-positive cancer RNA aptamer mertansine aptamer delivery</infon><infon key=\"license\">Licensee MDPI, Basel, Switzerland. This article is an open access article distributed under the terms and conditions of the Creative Commons Attribution (CC BY) license (http://creativecommons.org/licenses/by/4.0/).</infon><infon key=\"name_0\">surname:Jeong;given-names:Hwa Yeon</infon><infon key=\"name_1\">surname:Kim;given-names:Hyeri</infon><infon key=\"name_2\">surname:Lee;given-names:Myunghwa</infon><infon key=\"name_3\">surname:Hong;given-names:Jinju</infon><infon key=\"name_4\">surname:Lee;given-names:Joo Han</infon><infon key=\"name_5\">surname:Kim;given-names:Jeonghyeon</infon><infon key=\"name_6\">surname:Choi;given-names:Moon Jung</infon><infon key=\"name_7\">surname:Park;given-names:Yong Serk</infon><infon key=\"name_8\">surname:Kim;given-names:Sung-Chun</infon><infon key=\"section_type\">TITLE</infon><infon key=\"type\">front</infon><infon key=\"volume\">21</infon><infon key=\"year\">2020</infon><offset>0</offset><text>Development of HER2-Specific Aptamer-Drug Conjugate for Breast Cancer Therapy</text></passage><passage><infon key=\"section_type\">ABSTRACT</infon><infon key=\"type\">abstract</infon><offset>78</offset><text>In this study, HER2 RNA aptamers were conjugated to mertansine (DM1) and the anti-cancer effectiveness of the conjugate was evaluated in HER2-overexpressing breast cancer models. The conjugate of HER2 aptamer and anticancer drug DM1 (aptamer-drug conjugate, ApDC) was prepared and analyzed using HPLC and mass spectrometry. The cell-binding affinity and cytotoxicity of the conjugate were determined using confocal microscopy and WST-1 assay. The in vivo anti-tumoral efficacy of ApDC was also evaluated in mice carrying BT-474 breast tumors overexpressing HER2. The synthesized HER2-specific RNA aptamers were able to specifically and efficiently bind to HER-positive BT-474 breast cancer cells, but not to HER2-negative MDA-MB-231 breast cancer cells. Also, the HER2-specific ApDC showed strong toxicity to the target cells, BT-474, but not to MDA-MB-231 cells. According to the in vivo analyses drawn from the mouse xenografts of BT-747 tumor, the ApDC was able to more effectively inhibit the tumor growth. Compared to the control group, the mice treated with the ApDC showed a significant reduction of tumor growth. Besides, any significant body weight losses or hepatic toxicities were monitored in the ApDC-treated mice. This research suggests the HER2 aptamer-DM1 conjugate as a target-specific anti-cancer modality and provides experimental evidence supporting its enhanced effectiveness for HER2-overexpressing target tumors. This type of aptamer-conjugated anticancer drug would be utilized as a platform structure for the development of versatile targeted high-performance anticancer drugs by adopting the easy deformability and high affinity of aptamers.</text></passage><passage><infon key=\"section_type\">INTRO</infon><infon key=\"type\">title_1</infon><offset>1746</offset><text>1. Introduction</text></passage><passage><infon key=\"section_type\">INTRO</infon><infon key=\"type\">paragraph</infon><offset>1762</offset><text>Targeted therapy is an essential procedure for precision and personalized therapy to improve patient care. In order to bring target-specificity, varied target ligands are coupled to therapeutic drugs by various linking strategies such as direct coupling, via biocompatible spacers, cleavable linkers, or other payload chemistries. In all these cases, the efficiency and safety of the conjugate are primarily determined by the nature of payload and targeting ligand. Compared to untargeted therapies, for example, chemotherapy or radiotherapy of cancer, targeted drug delivery intends to deliver toxic therapeutic cargos specifically into diseased cells and avoids off-target uptake by healthy normal cells. To find adoptable targeting ligands, a variety of small organic molecules, proteins, and nucleic acid scaffolds have been extensively searched based on the biomarkers selected.</text></passage><passage><infon key=\"section_type\">INTRO</infon><infon key=\"type\">paragraph</infon><offset>2646</offset><text>Among ligands for targeted therapy, antibodies are well-established for specific recognition and/or biological regulation, and, therefore, antibody-drug conjugates (ADCs) have attracted tremendous attention for targeted cancer therapy. Besides, certain structures of aptamers and peptides have been also recognized as a useful targeting moiety for targeted therapy. Recently, exciting advances in oncology have been realized from an old idea about clinical applications of ADCs which was first tested 30 years ago. Brentuximab vedotin (anti-CD30-monomethyl auristatin E, MMAE) and ado-trastuzumab emtansine (anti-HER2-DM1), prepared through improved antibody-drug linker chemistry, were recently approved by the Food and Drug Administration (FDA) for cancer treatment. The conjugate of a prostate-specific membrane antigen (PSMA) antibody (SCL44A4) and highly toxic monomethyl auristatin E (MMAE) has been also testing as a target-specific ADC.</text></passage><passage><infon key=\"section_type\">INTRO</infon><infon key=\"type\">paragraph</infon><offset>3599</offset><text>Separately, various structures of aptamers have been developed as targeting ligands. Aptamers which are single-stranded oligonucleotides with specific recognition capabilities to target molecules are primarily screened through the Systematic Evolution of Ligands by Exponential Enrichment (SELEX) in combination with purification and analysis techniques such as capillary electrophoresis, microfluidics, and fluorescence-activated cell sorting. The molecular targets of aptamers range from small molecules and proteins to intact cells. A wide array of aptamers with high binding affinities (Kds of pM-nM) has been developed for biomarkers of high therapeutic interest. Recent advances in biochemistry and biophysics have facilitated the resolution of crystal structures of aptamer-target complexes which brings elucidation of aptamer-target interactions in detail. This facilitates the development of aptamers and aptamer-drug conjugates (ApDCs) through rational drug design and screening, as well as the discovery of new druggable sites on biomarkers.</text></passage><passage><infon key=\"section_type\">INTRO</infon><infon key=\"type\">paragraph</infon><offset>4656</offset><text>In addition to target binding, aptamers can also act as therapeutics to regulate the biological functions of target molecules. For example, pegaptanib (marketed as Macugen ), an anti-VEGF aptamer that recognizes most human VEGF-A isoforms, has been approved by the FDA for the treatment of age-related macular degeneration (AMD). Another example, AS1411, is a nucleolin-targeting aptamer currently under Phase II clinical investigation for the treatment of acute myeloid leukemia (AML). The several advantages over antibody such as low immunogenicity, rapid tissue penetration, cost-effectiveness in synthesis and modification, and long shelf life accelerate their clinical applications as therapeutics and targeting ligands.</text></passage><passage><infon key=\"section_type\">INTRO</infon><infon key=\"type\">paragraph</infon><offset>5383</offset><text>ErbB family is one of the most popular receptors for targeted therapies. In particular, EGFR (ErbB1 and HER1) are associated with a variety of solid tumor malignancies and the overexpression of ErbB2 (HER2) is also found in 20-30% of breast cancers. To date, synthetic tyrosine kinase inhibitors (e.g., Erlotinib and Gefitinib) and monoclonal antibodies (mAbs; e.g., Cetuximab and Trastuzumab) have been developed to inhibit certain signaling pathways in cancer cells or recruit the immune system to cancer cells. Several anti-ErbB2 aptamers were already developed as an alternative therapeutic modality. The anti-ErbB2 RNA aptamer was used in this study for ApDC development.</text></passage><passage><infon key=\"section_type\">INTRO</infon><infon key=\"type\">paragraph</infon><offset>6062</offset><text>In this study, we developed a HER2-specific aptamer-drug conjugate for targeted cancer therapy. The HER2-specific aptamer was conjugated with anti-cancer mertansine (DM1) by a cleavable disulfide bond at the 3' end of aptamer. A polyethylene glycol molecule (M.W. 20 kDa) was also conjugated to the 5' end of aptamer for prolonged circulation in vivo. The HER2-specific ApDC was tested in HER2-positive cancer cells to verify its target-specific binding capability and cytotoxicity. Also, the ApDC was systemically administered into mice carrying tumor xenografts to evaluate its targeted anti-cancer therapeutic efficacy.</text></passage><passage><infon key=\"section_type\">RESULTS</infon><infon key=\"type\">title_1</infon><offset>6689</offset><text>2. Results and Discussion</text></passage><passage><infon key=\"section_type\">RESULTS</infon><infon key=\"type\">title_2</infon><offset>6715</offset><text>2.1. Characterization of HER2 RNA Aptamer</text></passage><passage><infon key=\"section_type\">RESULTS</infon><infon key=\"type\">paragraph</infon><offset>6757</offset><text>It is known that RNA molecules are generally unstable and easily degraded by RNases in various environments. However, the stability of RNA can be secured by various conventional modification methods, and the RNA aptamer used in the present study (Figure 1A) was also modified at the pyrimidine by 2'-fluorinated pyrimidine, thereby achieving much higher stability than the unmodified RNA aptamer. The gel retardation analysis of aptamer molecules (Figure 1B) shows the relative stabilities of unmodified RNA aptamer and 2'-pyrimidine-modified RNA aptamer in 10% FBS-containing medium. The results of gel retardation analysis of the unmodified RNA aptamer revealed that the unmodified molecules degraded more than 50% in 1 h. On the other hand, 2'-pyrimidine-modified RNA aptamers were able to maintain their structure over 50% even 48 h later.</text></passage><passage><infon key=\"section_type\">RESULTS</infon><infon key=\"type\">paragraph</infon><offset>7607</offset><text>The 2'-pyrimidine-modified RNA aptamers utilized in this study exhibited a strong binding affinity to HER2 receptors. According to the qPCR measurement, the dissociation constant of the HER2 aptamer was 16.5 +- 4.4 nM (Figure 1C). Their HER2-specific binding was also confirmed by microscopic assessment (Figure 1D). Fluorescent HER2 aptamers effectively bind to BT-474 and A549 cancer cells, both over-expressing HER2 receptors. Meanwhile, the same molecules little bind MCF7 low-expressing HER2 and MDA-MB-231 cells deficient of HER2. These results suggest that the 2'-pyrimidine-modified HER2 RNA aptamer prepared for this study can be utilized as a ligand for HER2-specific delivery of anticancer drugs in vivo.</text></passage><passage><infon key=\"section_type\">RESULTS</infon><infon key=\"type\">title_2</infon><offset>8327</offset><text>2.2. Chemistry of HER2 Aptamer-DM1 Conjugate</text></passage><passage><infon key=\"section_type\">RESULTS</infon><infon key=\"type\">paragraph</infon><offset>8372</offset><text>For the synthesis of HER2 aptamer-drug conjugate (ApDC), a 20 kDa PEG molecule was coupled to the 5' end of HER2 RNA aptamer and a molecule of DM1 was then attached to the other side 3' end (Figure 2A). DM1 is a widely utilized anticancer drug and has a structural advantage at the coupling to carrier molecules since it is a thiol form of mertansine. The disulfide bond between DM1 and the 3' end of the aptamer is expected to be broken in the reducing environment of early and late endosomes. The linkage fragility of ApDC could facilitate the release of DM1 from the endocytic pathway, elevating the anticancer therapeutic efficacy of the drug. Also, PEGylation of the aptamer at 5' end could enhance the half-life of ApDC in circulation by avoiding renal clearance. The prolonged circulation of the ApDC is expected to increase the availability and targetability of ApDC molecules to target cancer cells, presumably resulting in increased anticancer therapeutic efficacy.</text></passage><passage><infon key=\"section_type\">RESULTS</infon><infon key=\"type\">paragraph</infon><offset>9356</offset><text>According to the HPLC analysis (Figure 2B) and mass spectrometry (Figure 2C), the final product of ApDC exhibited 98% of conjugation yield and ~33,000 g/mole of molecular weight. The strict and stable procedure of ApDC synthesis provide strong advantages over biological ligand conjugates in terms of structural modification and quality control. Because of the stability of aptamer chemistry, there was little variation from batch to batch and their derivatives such as aptamer-drug conjugates were able to maintain targetability as well as therapeutic efficacy.</text></passage><passage><infon key=\"section_type\">RESULTS</infon><infon key=\"type\">title_2</infon><offset>9919</offset><text>2.3. In Vitro Cytotoxicity of ApDC</text></passage><passage><infon key=\"section_type\">RESULTS</infon><infon key=\"type\">paragraph</infon><offset>9954</offset><text>The target-specific cytotoxicity of HER2 aptamer-DM1 conjugate was analyzed by the WST-1 assay (n = 8) on HER2-positive BT-474 and HER2-negative MDA-MB-231 cancer cells. In the target BT-474 cancer cells, free DM1 and the HER2 ApDC exhibited similar levels of IC50 value, 230.4 nM and 201.7 nM, respectively (Figure 3A). However, the MDA-MB-231 cells deficient of HER2 did not respond to the HER2 ApDC while the same cells showed cytotoxic response to the free DM 1 (Figure 3B). The ApDC hardly affect the viability of MDA-MB-231 even at the highest concentration tested (2 muM). This implies that the cytotoxicity of the ApDC is dependent upon the HER2 expressed on the surface of cancer cells.</text></passage><passage><infon key=\"section_type\">RESULTS</infon><infon key=\"type\">paragraph</infon><offset>10650</offset><text>The free form of DM1 was able to penetrate the plasma membranes regardless HER2 expression. Therefore, it can be stated that the ApDC has cell-specific cytotoxicity. Presumably, the aptamer-drug conjugates were too big and too hydrophilic to directly penetrate through the plasma membranes via a certain transport and appeared to be internalized via HER2-mediated endocytosis. The disulfide bond was easily cleaved in the in the reducing environment of endocytic vesicles, resulting in facilitated escapes of the freed DM1 from endosomes. Free HER aptamers did not show any cell toxicity even to BT-474 cells at the highest concentrations. This implies the cytotoxicity of the ApDC solely comes from the drug DM1, not from the HER2 aptamer.</text></passage><passage><infon key=\"section_type\">RESULTS</infon><infon key=\"type\">title_2</infon><offset>11391</offset><text>2.4. In Vivo Tumor Growth Inhibition by ApDC</text></passage><passage><infon key=\"section_type\">RESULTS</infon><infon key=\"type\">paragraph</infon><offset>11436</offset><text>In order to examine the antitumoral activity of the HER2 aptamer-DM1 conjugate, the HER2 ApDC was intravenously administered to mice carrying BT-474 tumors three times at intervals of 3 days. Tumor volumes and body weights were measured for a total of 36 days. Compared with the saline-treated group of mice, both the free DM1 and the ApDC exhibited a reduction of tumor size just after treatment (Figure 4A) and then more effectively suppressed the growth of the tumor. When the tumor-carrying mice were treated with the same amount of DM1 (60 mug/kg), the tumors of mice injected with the ApDC were statically smaller than those with free DM1 after the 19th-day post-treatment. These results tell that the chemically modified DM1 is still effective in inhibition of tumor growth and the HER2 aptamers conjugated to the drugs presumably enhance the anti-cancer therapeutic efficacy of DM1.</text></passage><passage><infon key=\"section_type\">RESULTS</infon><infon key=\"type\">paragraph</infon><offset>12327</offset><text>DM1 is known to be a strong inhibitor for microtubule assembly, yielding inherent systemic toxicity. However, the ApDC and free DM1 appeared to be safe at the treated dose, 60 mug DM1/kg. The mice administered with the suggested dose of ApDC or free DM1 did not show any significant body weight loss during treatment (Figure 4B). Aberrant side-effects of DM1 under in vivo conditions were already reported and based on the analyses, preclinical and clinical trials of DM1 conjugates have been allowed and then proceeded. Since any systemic cytotoxicity of RNA aptamers has not been reported and only some allergic reaction of PEG molecules has been reported, the enhanced anti-cancer therapeutic efficacy would certainly result from the inherent toxicity of DM1 which were more efficiently delivered to the cancer cells. However, at this moment it is uncertain whether providing HER2 targetability and PEG conjugation to DM1 may reduce the systemic toxicity of the drug. Upscaled animal studies would further verify the enhanced safety of the ApDC in vivo.</text></passage><passage><infon key=\"section_type\">RESULTS</infon><infon key=\"type\">title_2</infon><offset>13384</offset><text>2.5. Histological Changes by ApDC Treatment</text></passage><passage><infon key=\"section_type\">RESULTS</infon><infon key=\"type\">paragraph</infon><offset>13428</offset><text>At the end point of tumor growth analysis, major internal organs such as tumor, heart, liver, lungs, and spleen were resected, and any forms of cell death were then analyzed through hematoxylin and eosin (H&amp;E) staining and terminal deoxynucleotidyl transferase dUTP nick end labeling (TUNEL) assay. The TUNEL assay verified some apoptotic cells in the superficial area of the tumor tissue of the ApDC-administered mice (Figure 5). In the case of mice treated with free DM1, apoptotic cells were found in tumor tissues and spleen as well.</text></passage><passage><infon key=\"section_type\">RESULTS</infon><infon key=\"type\">paragraph</infon><offset>13966</offset><text>According to the microscopic analysis, no significant infiltration of immune cells was found in the tissues of major organs of mice treated with the ApDC or free DM1 at 60 mug DM1/kg (Figure S1). These results imply that the intravenous administration of the target specific ApDC appeared to be rather a safer procedure to treat cancer, compared to the direct administration of free DM1. The tumor-targeting aptamer conjugated to DM1 molecule presumably enhanced DM1 accumulation in the tumor tissue. Meanwhile, the free DM1 happened to be more off-targeted to spleen resulting in aberrant tissue damages, due to a lack of tumor-targeting capability.</text></passage><passage><infon key=\"section_type\">RESULTS</infon><infon key=\"type\">title_2</infon><offset>14617</offset><text>2.6. Changes of Hematological and Biochemical Parameters by ApDC Treatment</text></passage><passage><infon key=\"section_type\">RESULTS</infon><infon key=\"type\">paragraph</infon><offset>14692</offset><text>In order to further verify the in vivo toxicity of HER2 aptamer-DM1 conjugate, mice were intravenously injected with varied concentrations of free DM1 and the ApDC. Two days later, the hematological and biochemical parameters were analyzed using the blood infraorbital taken. According to the hematological analysis, the counts of platelets, RBC, and hematocrit were not significantly changed at the test dose, 60 mug DM1/kg and the equivalent dose of ApDC (2.7 mg/kg). Any significant changes in the numbers of white blood cells including monocytes, eosinophils, and basophils were not monitored either. These imply that the administration of ApDC and free DM1 at the controlled dose does not elicit serious side-effects such as aberrant immune stimulation and anemia, reported major side-effects of DM1. The biochemical analysis (Table 1) also showed that the administration of ApDC and free DM1 at the dose of 60 mug DM1/kg did not seriously affect the hepatic functions and hematologic characteristics of a mouse. These results suggest that the ApDC and DM1 can be systemically utilized for cancer treatment at the controlled doses.</text></passage><passage><infon key=\"section_type\">RESULTS</infon><infon key=\"type\">paragraph</infon><offset>15829</offset><text>A reducing environment of the body may readily break the disulfide bonding between the aptamer and free DM1 and the premature exposure of free DM1 in the blood circulation may cause the aberrant side-effects of DM1. Therefore, when combining a delivery carrier with a drug, it is important to consider the stability of the chemical linkage in the body. The disulfide linkage for the combination of aptamer molecules and drugs can enhance the efficiency of the drug but may not guarantee the stability of the conjugates in the body. In order to make a further analysis regarding the in vivo side-effects of ApDC, the distribution and pharmacokinetics of the drug in the body should be analyzed. A more systematic analysis on a larger scale would tell us a genuine benefit of ApDC in terms of efficacy and toxicity, differentiating from those of free DM1. At the same time, to expand the utilization of aptamer in drug targeting, it is necessary to prove beneficial merits of aptamer-mediated targeting over antibody-mediated drug delivery.</text></passage><passage><infon key=\"section_type\">METHODS</infon><infon key=\"type\">title_1</infon><offset>16868</offset><text>3. Materials and Methods</text></passage><passage><infon key=\"section_type\">METHODS</infon><infon key=\"type\">title_2</infon><offset>16893</offset><text>3.1. Cell Lines and Cell Culture</text></passage><passage><infon key=\"section_type\">METHODS</infon><infon key=\"type\">paragraph</infon><offset>16926</offset><text>BT-474 (breast mammary gland, human), MDA-MB-231 (breast mammary gland, human), MCF-7 (breast adenocarcinoma, human), and A549 (lung carcinoma, human) were purchased from the Korean Cell Line Bank (Seoul, Korea). Following the suppliers' instructions, the cells were cultured in RPMI media with l-glutamine, 25 mM HEPES supplemented with 10% fetal bovine serum (FBS), and penicillin-streptomycin sulfate at 37  C in a 5% CO2 incubator.</text></passage><passage><infon key=\"section_type\">METHODS</infon><infon key=\"type\">title_2</infon><offset>17365</offset><text>3.2. Aptamer and Chemicals</text></passage><passage><infon key=\"section_type\">METHODS</infon><infon key=\"type\">paragraph</infon><offset>17392</offset><text>The DNA template of HER2 RNA aptamer and FITC-labeled RNA aptamer were purchased from Bioneer (Daejeon, Korea). Double-stranded DNA templates for transcription were generated via PCR, and the dsDNA templates were purified with a QIAquick PCR purification kit (Qiagen, Valencia, CA, USA). HER2 RNA aptamers were in vitro transcribed from the purified PCR product by a DuraScribe T7 transcription kit (Epicentre, Madison, WI, USA). RNase A- resistant RNA was accomplished by replacing CTP and UTP with 2'-fluorine-dCTP (2'-F-dCTP) and 2'-fluorine-dUTP (2'-F-dUTP) in the DuraScribe in vitro transcription reaction. The sequence of RNA aptamer was 5'-AGC CGC GAG GGG AGG GAU AGG GUA GGG CGC GGC U-3' with 2'-fluorinated pyrimidines. Mertansine, also called DM1 compound, was purchased from Cayman Chemical (Ann Arbor, MI, USA).</text></passage><passage><infon key=\"section_type\">METHODS</infon><infon key=\"type\">paragraph</infon><offset>18231</offset><text>For the determination of aptamer dissociation constant, the HER2 aptamers were diluted to several different concentrations (from 0 to 100 nM) and incubated in the presence of His-tagged HER2 protein (3 nM) in the 200 muL of binding buffer (30 mM Tris-HCl, 150 mM NaCl, 1.5 mM MgCl2) for 30 min at room temperature. Ni-NTA agarose beads (Qiagen) were blocked with a buffer (30 mM Tris-HCl, 150 mM NaCl, 1.5 mM MgCl2, 1% BSA) for 1 h, washed with a binding buffer (30 mM Tris-HCl, 150 mM NaCl, 1.5 mM MgCl2) and then treated with the aptamer-protein mixture for 1 h. The aptamers were eluted by heating the beads to 95  C and then quantified by real-time q-PCR: 50  C for 3 min, 95  C for 2 min, and 40 cycles of 95  C for 15 s, 60  C for 30 s, and 72  C for 30 s (Table S1). The dissociation constants (Kd) of aptamers were obtained from the binding curves created with GraphPad Prism 5.0 software (GraphPad, San Diego, CA, USA).</text></passage><passage><infon key=\"section_type\">METHODS</infon><infon key=\"type\">title_2</infon><offset>19166</offset><text>3.3. Preparation of HER2 Aptamer-DM1 Conjugate</text></passage><passage><infon key=\"section_type\">METHODS</infon><infon key=\"type\">paragraph</infon><offset>19213</offset><text>The HER2 aptamer-DM1 conjugate is comprised of HER2 aptamer, DM1, and PEG (polyethylene glycol, 20 kDa) with a bifunctional linker. Aptamer and DM1 were conjugated with a disulfide bond using a succinimidyl 3-(2-pyridyldithio)propionate (SPDP) bifunctional linker. The synthesized HER2 aptamer-DM1-PEG conjugates (Bio-Synthesis, Lewisville, TX, USA), referred to as ApDC, were stored at -20  C until usage.</text></passage><passage><infon key=\"section_type\">METHODS</infon><infon key=\"type\">title_2</infon><offset>19623</offset><text>3.4. Serum Stability of HER2 RNA Aptamer</text></passage><passage><infon key=\"section_type\">METHODS</infon><infon key=\"type\">paragraph</infon><offset>19664</offset><text>The HER2 RNA aptamers were incubated in RPMI 1640 media including 10% fetal bovine serum at 37  C for 0 ~ 72 h and analyzed in 2% agarose gel. The shifted RNAs were stained with Loading Star (Dynebio, Seongnam, Korea), quantified using Gel Quant NET software (BiochemLabSolutions.com), and plotted with GraphPad Prism 5.0 software (GraphPad, San Diego, CA, USA).</text></passage><passage><infon key=\"section_type\">METHODS</infon><infon key=\"type\">title_2</infon><offset>20028</offset><text>3.5. Confocal Image Analysis</text></passage><passage><infon key=\"section_type\">METHODS</infon><infon key=\"type\">paragraph</infon><offset>20057</offset><text>BT474, MDA-MB-231, MCF-7, and A549 cells (each 1.0 x 106) were seeded on a 4-chamber cell culture plate (SPL life sciences, Pocheon, Korea) and incubated in a CO2 incubator at 37  C for 24 h. After washing with a buffer (30 mM Tris-HCl, 150 mM NaCl, and 1.5 mM MgCl2), the cells were treated with FITC-labeled RNA aptamers (25 nM) and incubated at 37  C for 30 min to 4 h with gentle shaking. The treated cells were washed, stained with Antifade Mounting Medium with DAPI (Vector laboratories, Burlingame, CA, USA) for 30 min and then visualized with a confocal microscope (LSM800, Carl Zeiss, Jena, Germany).</text></passage><passage><infon key=\"section_type\">METHODS</infon><infon key=\"type\">title_2</infon><offset>20670</offset><text>3.6. Cytotoxicity Assay</text></passage><passage><infon key=\"section_type\">METHODS</infon><infon key=\"type\">paragraph</infon><offset>20694</offset><text>To verify the cytotoxicity of ApDC, BT474 (HER2-positive) and MDA-MB-231 (HER2-negative) cells were seeded in 96-well plates and cultured for 24 h. The plated cells were treated with HER2 aptamer, DM1, or ApDC (0 to 500 nM, 100 muL, n = 8) in serum-free culture medium at 37  C for 4 h. Fresh 20% FBS-containing media (100 muL) were added to the media which were further incubated for 72 h. For live cell counting, 10 muL of WST solution (EZ-cytox, Seoul, Korea) was added to each well which was additionally incubated for 4 h. The absorbance of media in the wells was measured at 450 nm wavelength using Tecan Sunrise microplate reader (Tecan, Mannedorf, Switzerland).</text></passage><passage><infon key=\"section_type\">METHODS</infon><infon key=\"type\">title_2</infon><offset>21366</offset><text>3.7. In Vivo Xenograft Mouse Model</text></passage><passage><infon key=\"section_type\">METHODS</infon><infon key=\"type\">paragraph</infon><offset>21401</offset><text>The animal experiments were performed with the approval of the Institutional Animal Care and Ethics Committee of Yonsei University at Wonju College of Medicine (approval number: YWCI-201808-012-02). BT-474 cells were suspended at a concentration of 1.5 x 107 cells in 200 muL of serum-free media containing 50% of Matrigel (BD Biosciences, Bedford, MA, USA) and subcutaneously injected to the right flank of 7-week-old female BALB/c nude mice (Orient Bio, Seongnam, Korea).</text></passage><passage><infon key=\"section_type\">METHODS</infon><infon key=\"type\">title_2</infon><offset>21876</offset><text>3.8. Analyses of In Vivo Anticancer Activity of ApDC</text></passage><passage><infon key=\"section_type\">METHODS</infon><infon key=\"type\">paragraph</infon><offset>21929</offset><text>The anticancer therapeutic activity of the HER2 aptamer-DM1-PEG conjugate was evaluated in the mice bearing BT-474 xenografts. When the volumes of implanted tumors reached approximately 100 mm3, BT-474 tumor-bearing mice were randomly separated into five groups (n = 4, group I: PBS-treated, group II: free DM1-treated, group III: ApDC-treated). DM1 and ApDC were dissolved in PBS and then intravenously administered into mice (60 mug of DM1/kg each) three times every three days. Body weights and tumor volumes were measured every other day (tumor volume = a x b2 x 0.52, a; long diameter, b; short diameter). The mice were sacrificed on day 36, and the major organs including tumors were excised, fixed, and embedded in paraffin for histological analysis.</text></passage><passage><infon key=\"section_type\">METHODS</infon><infon key=\"type\">title_2</infon><offset>22689</offset><text>3.9. Histological Analysis</text></passage><passage><infon key=\"section_type\">METHODS</infon><infon key=\"type\">paragraph</infon><offset>22716</offset><text>On day 35 post injection, the tumor, liver, lungs, spleen, and heart of all treated groups (PBS: PBS-treated, DM1 low: 12 mug of DM1/kg each-treated, DM1 medium: 60 mug of DM1/kg each-treated, DM1 high: 300 mug of DM1/kg each-treated, ApDC: 60 mug of DM1/kg each-treated) were dissected, fixed in 4% formalin, embedded in paraffin, and sectioned at 5 mum thickness. The sections were deparaffinized, hydrated, and stained with H&amp;E reagent for histological analysis. Apoptotic cells in tumor tissues were stained with Click-itTM TUNEL colorimetric kit (Invitrogen, Carlsbad, CA, USA) according to the manufacturer's instruction. TUNEL-positive cells were manually quantified.</text></passage><passage><infon key=\"section_type\">METHODS</infon><infon key=\"type\">title_2</infon><offset>23393</offset><text>3.10. Analysis of Dematological and Biochemical Parameters</text></passage><passage><infon key=\"section_type\">METHODS</infon><infon key=\"type\">paragraph</infon><offset>23452</offset><text>To analyze the in vivo toxicity of HER2 ApDC, the hematological and biochemical parameters of the BALB/c normal mice were measured after intravenous administration of a single dose of PBS, free DM1 (60 mug/kg), ApDC (2.7 mg/kg, equivalent to 60 mug of DM1/kg). Subsequently, blood was collected in an EDTA-coated microtainer tube (BD Biosciences, Franklin Lakes, NJ, USA) for complete blood counts (CBC) from the intraorbital vein under isoflurane anesthesia on day 3 and then centrifuged at 1800x g for 10 min. Hematological parameters such as counts of platelets, reticulocytes, lymphocytes, and neutrophils were measured with HEMAVET 950 hematology system (Drew Scientific, Waterbury, CT, USA). Mouse sera were also obtained after centrifugation of the blood at 2000x g for 20 min. The biochemical parameters of the treated mouse such as serum alanine aminotransferase (ALT), aspartate aminotransferase (AST), alkaline phosphatase (ALP), gamma-glutamyl transpeptidase, and total bilirubin sera were measured with an automated biochemistry analyzer (Konelab 20XT, Thermo Fisher Scientific, Waltham, MA, USA).</text></passage><passage><infon key=\"section_type\">METHODS</infon><infon key=\"type\">title_2</infon><offset>24565</offset><text>3.11. Statistical Analysis</text></passage><passage><infon key=\"section_type\">METHODS</infon><infon key=\"type\">paragraph</infon><offset>24592</offset><text>The statistical significance of experimental results was determined by the Student's t test and ANOVA using GraphPad Prism software (GraphPad, San Diego, CA, USA).</text></passage><passage><infon key=\"section_type\">CONCL</infon><infon key=\"type\">title_1</infon><offset>24758</offset><text>4. Conclusions</text></passage><passage><infon key=\"section_type\">CONCL</infon><infon key=\"type\">paragraph</infon><offset>24773</offset><text>This study shows a representative application of aptamer-based targeted delivery of therapeutics for the treatment of HER2-overexpressing breast cancer. According to the results of the cell-specific binding, cytotoxicity, and in vivo evaluation of the aptamer-drug conjugate ApDC, the HER2 aptamer conjugated to anticancer drug DM1 was able to specifically recognize the target cancer cells and therefore enhance the therapeutic efficacy of the drug. This report provides strong evidence supporting a platform technology of aptamer-guided therapeutics and drug delivery.</text></passage><passage><infon key=\"section_type\">CONCL</infon><infon key=\"type\">footnote</infon><offset>25344</offset><text>Publisher's Note: MDPI stays neutral with regard to jurisdictional claims in published maps and institutional affiliations.</text></passage><passage><infon key=\"section_type\">SUPPL</infon><infon key=\"type\">title</infon><offset>25470</offset><text>Supplementary Materials</text></passage><passage><infon key=\"section_type\">SUPPL</infon><infon key=\"type\">paragraph</infon><offset>25494</offset><text>Supplementary materials can be found at .</text></passage><passage><infon key=\"section_type\">AUTH_CONT</infon><infon key=\"type\">title</infon><offset>25536</offset><text>Author Contributions</text></passage><passage><infon key=\"section_type\">AUTH_CONT</infon><infon key=\"type\">paragraph</infon><offset>25557</offset><text>Conceptualization, H.Y.J. and S.-C.K.; methodology, H.Y.J., H.K., M.L., J.H., J.H.L., J.K. and M.J.C.; data curation, H.Y.J. and Y.S.P.; writing:original draft preparation, H.Y.J.; writing:review and editing, Y.S.P.; supervision, Y.S.P. and S.-C.K.; project administration, S.-C.K.; funding acquisition, S.-C.K. All authors have read and agreed to the published version of the manuscript.</text></passage><passage><infon key=\"section_type\">ACK_FUND</infon><infon key=\"type\">title</infon><offset>25950</offset><text>Funding</text></passage><passage><infon key=\"section_type\">ACK_FUND</infon><infon key=\"type\">paragraph</infon><offset>25958</offset><text>This research received no external funding.</text></passage><passage><infon key=\"section_type\">COMP_INT</infon><infon key=\"type\">title</infon><offset>26002</offset><text>Conflicts of Interest</text></passage><passage><infon key=\"section_type\">COMP_INT</infon><infon key=\"type\">paragraph</infon><offset>26024</offset><text>The authors declare no conflict of interest.</text></passage><passage><infon key=\"section_type\">ABBR</infon><infon key=\"type\">title</infon><offset>26069</offset><text>Abbreviations</text></passage><passage><infon key=\"file\">no_id_0.xml</infon><infon key=\"id\">no_id_0</infon><infon key=\"section_type\">TABLE</infon><infon key=\"type\">table</infon><infon key=\"xml\">&lt;?xml version=&quot;1.0&quot; encoding=&quot;UTF-8&quot;?&gt;\n",
      "&lt;tbody&gt;&lt;tr&gt;&lt;td align=&quot;left&quot; valign=&quot;top&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;HER2&lt;/td&gt;&lt;td align=&quot;left&quot; valign=&quot;top&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;Human epidermal growth factor receptor 2&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td align=&quot;left&quot; valign=&quot;top&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;DM1&lt;/td&gt;&lt;td align=&quot;left&quot; valign=&quot;top&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;Mertansine&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td align=&quot;left&quot; valign=&quot;top&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;ApDC&lt;/td&gt;&lt;td align=&quot;left&quot; valign=&quot;top&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;Aptamer drug conjugate&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td align=&quot;left&quot; valign=&quot;top&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;ADC&lt;/td&gt;&lt;td align=&quot;left&quot; valign=&quot;top&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;Antibody drug conjugate&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td align=&quot;left&quot; valign=&quot;top&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;MMAE&lt;/td&gt;&lt;td align=&quot;left&quot; valign=&quot;top&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;Monomethyl auristatin E&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td align=&quot;left&quot; valign=&quot;top&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;FDA&lt;/td&gt;&lt;td align=&quot;left&quot; valign=&quot;top&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;Food and Drug Administration&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td align=&quot;left&quot; valign=&quot;top&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;PSMA&lt;/td&gt;&lt;td align=&quot;left&quot; valign=&quot;top&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;Prostate specific membrane antigen&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td align=&quot;left&quot; valign=&quot;top&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;SELEX&lt;/td&gt;&lt;td align=&quot;left&quot; valign=&quot;top&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;Systematic evolution of ligands by exponential enrichment&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td align=&quot;left&quot; valign=&quot;top&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;Kds&lt;/td&gt;&lt;td align=&quot;left&quot; valign=&quot;top&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;Equilibrium dissociation constant&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td align=&quot;left&quot; valign=&quot;top&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;VEGF&lt;/td&gt;&lt;td align=&quot;left&quot; valign=&quot;top&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;Vascular endothelial growth factor&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td align=&quot;left&quot; valign=&quot;top&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;EGFR&lt;/td&gt;&lt;td align=&quot;left&quot; valign=&quot;top&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;Epidermal growth factor receptor&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td align=&quot;left&quot; valign=&quot;top&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;FITC&lt;/td&gt;&lt;td align=&quot;left&quot; valign=&quot;top&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;Fluorescein isothiocyanate&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td align=&quot;left&quot; valign=&quot;top&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;qPCR&lt;/td&gt;&lt;td align=&quot;left&quot; valign=&quot;top&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;Quantitative real time polymerase chain reaction&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td align=&quot;left&quot; valign=&quot;top&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;PEG&lt;/td&gt;&lt;td align=&quot;left&quot; valign=&quot;top&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;Polyethylene glycol&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td align=&quot;left&quot; valign=&quot;top&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;HPLC&lt;/td&gt;&lt;td align=&quot;left&quot; valign=&quot;top&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;High performance liquid chromatography&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td align=&quot;left&quot; valign=&quot;top&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;WST-1&lt;/td&gt;&lt;td align=&quot;left&quot; valign=&quot;top&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;Water soluble tetrazolium salt&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td align=&quot;left&quot; valign=&quot;top&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;TUNEL&lt;/td&gt;&lt;td align=&quot;left&quot; valign=&quot;top&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;Terminal deoxynucleotidyl transferase dUTP nick end labeling&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td align=&quot;left&quot; valign=&quot;top&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;H&amp;amp;E&lt;/td&gt;&lt;td align=&quot;left&quot; valign=&quot;top&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;Hematoxylin and eosin&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td align=&quot;left&quot; valign=&quot;top&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;ALT&lt;/td&gt;&lt;td align=&quot;left&quot; valign=&quot;top&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;Alanine transaminase&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td align=&quot;left&quot; valign=&quot;top&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;AST&lt;/td&gt;&lt;td align=&quot;left&quot; valign=&quot;top&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;Aspartate aminotransferase&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td align=&quot;left&quot; valign=&quot;top&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;ALP&lt;/td&gt;&lt;td align=&quot;left&quot; valign=&quot;top&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;Alkaline phosphatase&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td align=&quot;left&quot; valign=&quot;top&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;GGT&lt;/td&gt;&lt;td align=&quot;left&quot; valign=&quot;top&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;Gamma-glutamyl transferase&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td align=&quot;left&quot; valign=&quot;top&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;SPDP&lt;/td&gt;&lt;td align=&quot;left&quot; valign=&quot;top&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;Succinimidyl 3-(2-pyridyldithio)propionate&lt;/td&gt;&lt;/tr&gt;&lt;/tbody&gt;\n",
      "</infon><offset>26083</offset><text>HER2\tHuman epidermal growth factor receptor 2\t \tDM1\tMertansine\t \tApDC\tAptamer drug conjugate\t \tADC\tAntibody drug conjugate\t \tMMAE\tMonomethyl auristatin E\t \tFDA\tFood and Drug Administration\t \tPSMA\tProstate specific membrane antigen\t \tSELEX\tSystematic evolution of ligands by exponential enrichment\t \tKds\tEquilibrium dissociation constant\t \tVEGF\tVascular endothelial growth factor\t \tEGFR\tEpidermal growth factor receptor\t \tFITC\tFluorescein isothiocyanate\t \tqPCR\tQuantitative real time polymerase chain reaction\t \tPEG\tPolyethylene glycol\t \tHPLC\tHigh performance liquid chromatography\t \tWST-1\tWater soluble tetrazolium salt\t \tTUNEL\tTerminal deoxynucleotidyl transferase dUTP nick end labeling\t \tH&amp;E\tHematoxylin and eosin\t \tALT\tAlanine transaminase\t \tAST\tAspartate aminotransferase\t \tALP\tAlkaline phosphatase\t \tGGT\tGamma-glutamyl transferase\t \tSPDP\tSuccinimidyl 3-(2-pyridyldithio)propionate\t \t</text></passage><passage><infon key=\"section_type\">REF</infon><infon key=\"type\">title</infon><offset>26973</offset><text>References</text></passage><passage><infon key=\"fpage\">507</infon><infon key=\"lpage\">522</infon><infon key=\"name_0\">surname:Euan;given-names:A.</infon><infon key=\"pub-id_pmid\">27528417</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">Nat. Rev. Genet.</infon><infon key=\"type\">ref</infon><infon key=\"volume\">17</infon><infon key=\"year\">2016</infon><offset>26984</offset><text>The sequencing of the human genome led many to specu late on the near term potential for clinical medi cine</text></passage><passage><infon key=\"fpage\">33</infon><infon key=\"lpage\">46</infon><infon key=\"name_0\">surname:Tsuchikama;given-names:K.</infon><infon key=\"name_1\">surname:An;given-names:Z.</infon><infon key=\"pub-id_doi\">10.1007/s13238-016-0323-0</infon><infon key=\"pub-id_pmid\">27743348</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">Protein Cell</infon><infon key=\"type\">ref</infon><infon key=\"volume\">9</infon><infon key=\"year\">2018</infon><offset>27092</offset><text>Antibody-drug conjugates: Recent advances in conjugation and linker chemistries</text></passage><passage><infon key=\"elocation-id\">1683</infon><infon key=\"name_0\">surname:Ni;given-names:S.</infon><infon key=\"name_1\">surname:Yao;given-names:H.</infon><infon key=\"name_2\">surname:Wang;given-names:L.</infon><infon key=\"name_3\">surname:Lu;given-names:J.</infon><infon key=\"name_4\">surname:Jiang;given-names:F.</infon><infon key=\"name_5\">surname:Lu;given-names:A.</infon><infon key=\"name_6\">surname:Zhang;given-names:G.</infon><infon key=\"pub-id_doi\">10.3390/ijms18081683</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">Int. J. Mol. Sci.</infon><infon key=\"type\">ref</infon><infon key=\"volume\">18</infon><infon key=\"year\">2017</infon><offset>27172</offset><text>Chemical Modifications of Nucleic Acid Aptamers for Therapeutic Purposes</text></passage><passage><infon key=\"fpage\">176</infon><infon key=\"lpage\">192</infon><infon key=\"name_0\">surname:Agarwal;given-names:P.</infon><infon key=\"name_1\">surname:Bertozzi;given-names:C.R.</infon><infon key=\"pub-id_doi\">10.1021/bc5004982</infon><infon key=\"pub-id_pmid\">25494884</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">Bioconjug. Chem.</infon><infon key=\"type\">ref</infon><infon key=\"volume\">26</infon><infon key=\"year\">2015</infon><offset>27245</offset><text>Site-Specific Antibody-Drug Conjugates: The Nexus of Bioorthogonal Chemistry, Protein Engineering, and Drug Development</text></passage><passage><infon key=\"fpage\">198</infon><infon key=\"lpage\">205</infon><infon key=\"name_0\">surname:Bae;given-names:Y.H.</infon><infon key=\"name_1\">surname:Park;given-names:K.</infon><infon key=\"pub-id_doi\">10.1016/j.jconrel.2011.06.001</infon><infon key=\"pub-id_pmid\">21663778</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">J. Control. Release</infon><infon key=\"type\">ref</infon><infon key=\"volume\">153</infon><infon key=\"year\">2011</infon><offset>27367</offset><text>Targeted drug delivery to tumors: Myths, reality and possibility</text></passage><passage><infon key=\"fpage\">215</infon><infon key=\"lpage\">223</infon><infon key=\"name_0\">surname:Singh;given-names:R.</infon><infon key=\"name_1\">surname:Lillard;given-names:J.W.</infon><infon key=\"pub-id_doi\">10.1016/j.yexmp.2008.12.004</infon><infon key=\"pub-id_pmid\">19186176</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">Exp. Mol. Pathol.</infon><infon key=\"type\">ref</infon><infon key=\"volume\">86</infon><infon key=\"year\">2009</infon><offset>27432</offset><text>Nanoparticle-based targeted drug delivery</text></passage><passage><infon key=\"fpage\">5</infon><infon key=\"issue\">Suppl. S6679</infon><infon key=\"lpage\">10</infon><infon key=\"name_0\">surname:Langer;given-names:R.</infon><infon key=\"pub-id_doi\">10.1201/b12801</infon><infon key=\"pub-id_pmid\">9579855</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">Nature</infon><infon key=\"type\">ref</infon><infon key=\"volume\">392</infon><infon key=\"year\">1998</infon><offset>27474</offset><text>Drug Delivery and Targeting</text></passage><passage><infon key=\"fpage\">548</infon><infon key=\"lpage\">552</infon><infon key=\"name_0\">surname:Sawyers;given-names:C.L.</infon><infon key=\"pub-id_doi\">10.1038/nature06913</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">Nat. Cell Biol.</infon><infon key=\"type\">ref</infon><infon key=\"volume\">452</infon><infon key=\"year\">2008</infon><offset>27502</offset><text>The cancer biomarker problem</text></passage><passage><infon key=\"fpage\">807</infon><infon key=\"lpage\">819</infon><infon key=\"name_0\">surname:Stopeck;given-names:A.T.</infon><infon key=\"name_1\">surname:Brown-Glaberman;given-names:U.</infon><infon key=\"name_2\">surname:Wong;given-names:H.Y.</infon><infon key=\"name_3\">surname:Park;given-names:B.H.</infon><infon key=\"name_4\">surname:Barnato;given-names:S.E.</infon><infon key=\"name_5\">surname:Gradishar;given-names:W.J.</infon><infon key=\"name_6\">surname:Hudis;given-names:C.A.</infon><infon key=\"name_7\">surname:Rugo;given-names:H.S.</infon><infon key=\"pub-id_doi\">10.1007/s10585-012-9496-y</infon><infon key=\"pub-id_pmid\">22692561</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">Clin. Exp. Metastasis</infon><infon key=\"type\">ref</infon><infon key=\"volume\">29</infon><infon key=\"year\">2012</infon><offset>27531</offset><text>The role of targeted therapy and biomarkers in breast cancer treatment</text></passage><passage><infon key=\"fpage\">20</infon><infon key=\"lpage\">27</infon><infon key=\"name_0\">surname:Gheibi Hayat;given-names:S.M.</infon><infon key=\"name_1\">surname:Sahebkar;given-names:A.H.</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">J. Babol. Univ. Med. Sci.</infon><infon key=\"type\">ref</infon><infon key=\"volume\">19</infon><infon key=\"year\">2017</infon><offset>27602</offset><text>Antibody drug conjugates for cancer therapy</text></passage><passage><infon key=\"fpage\">2186</infon><infon key=\"lpage\">2197</infon><infon key=\"name_0\">surname:Zhu;given-names:G.</infon><infon key=\"name_1\">surname:Niu;given-names:G.</infon><infon key=\"name_2\">surname:Chen;given-names:X.</infon><infon key=\"pub-id_doi\">10.1021/acs.bioconjchem.5b00291</infon><infon key=\"pub-id_pmid\">26083153</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">Bioconjug. Chem.</infon><infon key=\"type\">ref</infon><infon key=\"volume\">26</infon><infon key=\"year\">2015</infon><offset>27646</offset><text>Aptamer-Drug Conjugates</text></passage><passage><infon key=\"fpage\">1893</infon><infon key=\"lpage\">1904</infon><infon key=\"name_0\">surname:Burke;given-names:J.F.</infon><infon key=\"name_1\">surname:Mark;given-names:V.H.</infon><infon key=\"name_2\">surname:Soloway;given-names:A.H.</infon><infon key=\"name_3\">surname:Leskowitz;given-names:S.</infon><infon key=\"pub-id_pmid\">4162674</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">Cancer Res.</infon><infon key=\"type\">ref</infon><infon key=\"volume\">26</infon><infon key=\"year\">1966</infon><offset>27670</offset><text>The effect of antibody to L-phenylalanine mustard conjugate on malignant cells selectively marked through &quot;early inflammatory-like&quot; vascular permeability</text></passage><passage><infon key=\"fpage\">145</infon><infon key=\"lpage\">151</infon><infon key=\"name_0\">surname:Milunovic;given-names:V.</infon><infon key=\"name_1\">surname:Misura Jakobac;given-names:K.</infon><infon key=\"name_2\">surname:Kursar;given-names:M.</infon><infon key=\"name_3\">surname:Mandac Rogulj;given-names:I.</infon><infon key=\"name_4\">surname:Ostojic Kolonic;given-names:S.</infon><infon key=\"pub-id_doi\">10.1111/ejh.13269</infon><infon key=\"pub-id_pmid\">31166030</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">Eur. J. Haematol.</infon><infon key=\"type\">ref</infon><infon key=\"volume\">103</infon><infon key=\"year\">2019</infon><offset>27828</offset><text>FDA's and EMA's approval of brentuximab vedotin for advanced Hodgkin lymphoma: Another player in the town?</text></passage><passage><infon key=\"fpage\">405</infon><infon key=\"lpage\">414</infon><infon key=\"name_0\">surname:Boyraz;given-names:B.</infon><infon key=\"name_1\">surname:Sendur;given-names:M.A.</infon><infon key=\"name_2\">surname:Aksoy;given-names:S.</infon><infon key=\"name_3\">surname:Babacan;given-names:T.</infon><infon key=\"name_4\">surname:Roach;given-names:E.C.</infon><infon key=\"name_5\">surname:Kizilarslanoglu;given-names:M.C.</infon><infon key=\"name_6\">surname:Petekkaya;given-names:I.</infon><infon key=\"name_7\">surname:Altundag;given-names:K.</infon><infon key=\"pub-id_doi\">10.1185/03007995.2013.775113</infon><infon key=\"pub-id_pmid\">23402224</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">Curr. Med. Res. Opin.</infon><infon key=\"type\">ref</infon><infon key=\"volume\">29</infon><infon key=\"year\">2013</infon><offset>27939</offset><text>Trastuzumab emtansine (T-DM1) for HER2-positive breast cancer</text></passage><passage><infon key=\"fpage\">604</infon><infon key=\"lpage\">613</infon><infon key=\"name_0\">surname:Petrylak;given-names:D.P.</infon><infon key=\"name_1\">surname:Kantoff;given-names:P.</infon><infon key=\"name_2\">surname:Vogelzang;given-names:N.J.</infon><infon key=\"name_3\">surname:Mega;given-names:A.</infon><infon key=\"name_4\">surname:Fleming;given-names:M.T.</infon><infon key=\"name_5\">surname:Stephenson;given-names:J.J.;suffix:Jr.</infon><infon key=\"name_6\">surname:Frank;given-names:R.</infon><infon key=\"name_7\">surname:Shore;given-names:N.D.</infon><infon key=\"name_8\">surname:Dreicer;given-names:R.</infon><infon key=\"name_9\">surname:McClay;given-names:E.F.</infon><infon key=\"pub-id_doi\">10.1002/pros.23765</infon><infon key=\"pub-id_pmid\">30663074</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">Prostate</infon><infon key=\"type\">ref</infon><infon key=\"volume\">79</infon><infon key=\"year\">2019</infon><offset>28001</offset><text>Phase 1 study of PSMA ADC, an antibody-drug conjugate targeting prostate-specific membrane antigen, in chemotherapy-refractory prostate cancer</text></passage><passage><infon key=\"elocation-id\">957</infon><infon key=\"name_0\">surname:Wang;given-names:Y.</infon><infon key=\"name_1\">surname:Fan;given-names:S.-Y.</infon><infon key=\"name_2\">surname:Xiao;given-names:D.</infon><infon key=\"name_3\">surname:Xie;given-names:F.</infon><infon key=\"name_4\">surname:Zhao;given-names:L.</infon><infon key=\"name_5\">surname:Zhou;given-names:X.</infon><infon key=\"name_6\">surname:Zhou;given-names:X.</infon><infon key=\"pub-id_doi\">10.3390/cancers11070957</infon><infon key=\"pub-id_pmid\">31288450</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">Cancers</infon><infon key=\"type\">ref</infon><infon key=\"volume\">11</infon><infon key=\"year\">2019</infon><offset>28144</offset><text>Novel Silyl Ether-Based Acid-Cleavable Antibody-MMAE Conjugates with Appropriate Stability and Efficacy</text></passage><passage><infon key=\"fpage\">831</infon><infon key=\"lpage\">841</infon><infon key=\"name_0\">surname:Kawasaki;given-names:A.M.</infon><infon key=\"name_1\">surname:Casper;given-names:M.D.</infon><infon key=\"name_2\">surname:Freier;given-names:S.M.</infon><infon key=\"name_3\">surname:Lesnik;given-names:E.A.</infon><infon key=\"name_4\">surname:Zounes;given-names:M.C.</infon><infon key=\"name_5\">surname:Cummins;given-names:L.L.</infon><infon key=\"name_6\">surname:Gonzalez;given-names:C.</infon><infon key=\"name_7\">surname:Cook;given-names:P.D.</infon><infon key=\"pub-id_doi\">10.1021/jm00059a007</infon><infon key=\"pub-id_pmid\">8464037</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">J. Med. Chem.</infon><infon key=\"type\">ref</infon><infon key=\"volume\">36</infon><infon key=\"year\">1993</infon><offset>28248</offset><text>Uniformly modified 2'-deoxy-2'-fluoro-phosphorothioate oligonucleotides as nuclease-resistant antisense compounds with high affinity and specificity for RNA targets</text></passage><passage><infon key=\"fpage\">803</infon><infon key=\"lpage\">812</infon><infon key=\"name_0\">surname:Beutel;given-names:B.A.</infon><infon key=\"name_1\">surname:Gold;given-names:L.</infon><infon key=\"pub-id_doi\">10.1016/0022-2836(92)90865-H</infon><infon key=\"pub-id_pmid\">1469714</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">J. Mol. Biol.</infon><infon key=\"type\">ref</infon><infon key=\"volume\">228</infon><infon key=\"year\">1992</infon><offset>28417</offset><text>In vitro evolution of intrinsically bent DNA</text></passage><passage><infon key=\"fpage\">97</infon><infon key=\"lpage\">107</infon><infon key=\"name_0\">surname:Klug;given-names:S.J.</infon><infon key=\"name_1\">surname:Famulok;given-names:M.</infon><infon key=\"pub-id_doi\">10.1007/BF00996358</infon><infon key=\"pub-id_pmid\">7536299</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">Mol. Biol. Rep.</infon><infon key=\"type\">ref</infon><infon key=\"volume\">20</infon><infon key=\"year\">1994</infon><offset>28462</offset><text>All you wanted to know about SELEX</text></passage><passage><infon key=\"fpage\">862</infon><infon key=\"lpage\">869</infon><infon key=\"name_0\">surname:Schneider;given-names:D.</infon><infon key=\"name_1\">surname:Tuerk;given-names:C.</infon><infon key=\"name_2\">surname:Gold;given-names:L.</infon><infon key=\"pub-id_doi\">10.1016/0022-2836(92)90870-P</infon><infon key=\"pub-id_pmid\">1469719</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">J. Mol. Biol.</infon><infon key=\"type\">ref</infon><infon key=\"volume\">228</infon><infon key=\"year\">1992</infon><offset>28497</offset><text>Selection of high affinity RNA ligands to the bacteriophage R17 coat protein</text></passage><passage><infon key=\"fpage\">171</infon><infon key=\"lpage\">182</infon><infon key=\"name_0\">surname:Gopinath;given-names:S.C.</infon><infon key=\"pub-id_doi\">10.1007/s00216-006-0826-2</infon><infon key=\"pub-id_pmid\">17072603</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">Anal. Bioanal. Chem.</infon><infon key=\"type\">ref</infon><infon key=\"volume\">387</infon><infon key=\"year\">2007</infon><offset>28574</offset><text>Methods developed for SELEX</text></passage><passage><infon key=\"fpage\">5317</infon><infon key=\"lpage\">5338</infon><infon key=\"name_0\">surname:Cai;given-names:S.</infon><infon key=\"name_1\">surname:Yan;given-names:J.H.</infon><infon key=\"name_2\">surname:Xiong;given-names:H.</infon><infon key=\"name_3\">surname:Liu;given-names:Y.</infon><infon key=\"name_4\">surname:Peng;given-names:D.</infon><infon key=\"name_5\">surname:Liu;given-names:Z.</infon><infon key=\"pub-id_doi\">10.1039/C8AN01467A</infon><infon key=\"pub-id_pmid\">30357118</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">Analyst</infon><infon key=\"type\">ref</infon><infon key=\"volume\">143</infon><infon key=\"year\">2018</infon><offset>28602</offset><text>Investigations on the interface of nucleic acid aptamers and binding targets</text></passage><passage><infon key=\"fpage\">141</infon><infon key=\"lpage\">154</infon><infon key=\"name_0\">surname:Katz;given-names:B.</infon><infon key=\"name_1\">surname:Goldbaum;given-names:M.</infon><infon key=\"pub-id_doi\">10.1097/01.iio.0000212130.91136.31</infon><infon key=\"pub-id_pmid\">17060800</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">Int. Ophthalmol. Clin.</infon><infon key=\"type\">ref</infon><infon key=\"volume\">46</infon><infon key=\"year\">2006</infon><offset>28679</offset><text>Macugen (Pegaptanib Sodium), a Novel Ocular Therapeutic That Targets Vascular Endothelial Growth Factor (VEGF)</text></passage><passage><infon key=\"fpage\">178</infon><infon key=\"lpage\">187</infon><infon key=\"name_0\">surname:Rosenberg;given-names:J.</infon><infon key=\"name_1\">surname:Figlin;given-names:R.A.</infon><infon key=\"name_2\">surname:Smith;given-names:G.W.</infon><infon key=\"name_3\">surname:Choueiri;given-names:T.</infon><infon key=\"name_4\">surname:Erlandsson;given-names:F.</infon><infon key=\"pub-id_doi\">10.1007/s10637-013-0045-6</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">Investig. New Drugs</infon><infon key=\"type\">ref</infon><infon key=\"volume\">32</infon><infon key=\"year\">2015</infon><offset>28790</offset><text>A phase II trial of the nucleolin-targeted DNA aptamer AS1411 in metastatic refractory renal cell carcinoma</text></passage><passage><infon key=\"fpage\">181</infon><infon key=\"lpage\">202</infon><infon key=\"name_0\">surname:Zhou;given-names:J.</infon><infon key=\"name_1\">surname:Rossi;given-names:J.J.</infon><infon key=\"pub-id_doi\">10.1038/nrd.2016.199</infon><infon key=\"pub-id_pmid\">27807347</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">Nat. Rev. Drug Discov.</infon><infon key=\"type\">ref</infon><infon key=\"volume\">16</infon><infon key=\"year\">2017</infon><offset>28898</offset><text>Aptamers as targeted therapeutics: Current potential and challenges</text></passage><passage><infon key=\"fpage\">177</infon><infon key=\"lpage\">182</infon><infon key=\"name_0\">surname:Slamon;given-names:D.J.</infon><infon key=\"name_1\">surname:Clark;given-names:G.M.</infon><infon key=\"name_2\">surname:Wong;given-names:S.G.</infon><infon key=\"name_3\">surname:Levin;given-names:W.J.</infon><infon key=\"name_4\">surname:Ullrich;given-names:A.</infon><infon key=\"name_5\">surname:McGuire;given-names:W.L.</infon><infon key=\"pub-id_doi\">10.1126/science.3798106</infon><infon key=\"pub-id_pmid\">3798106</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">Science</infon><infon key=\"type\">ref</infon><infon key=\"volume\">235</infon><infon key=\"year\">1987</infon><offset>28966</offset><text>Human breast cancer: Correlation of relapse and survival with amplification of the HER-2/neu oncogene</text></passage><passage><infon key=\"fpage\">1160</infon><infon key=\"lpage\">1174</infon><infon key=\"name_0\">surname:Ciardiello;given-names:F.</infon><infon key=\"name_1\">surname:Tortora;given-names:G.</infon><infon key=\"pub-id_doi\">10.1056/NEJMra0707704</infon><infon key=\"pub-id_pmid\">18337605</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">N. Engl. J. Med.</infon><infon key=\"type\">ref</infon><infon key=\"volume\">358</infon><infon key=\"year\">2008</infon><offset>29068</offset><text>EGFR Antagonists in Cancer Treatment</text></passage><passage><infon key=\"fpage\">148</infon><infon key=\"name_0\">surname:Liu;given-names:Z.</infon><infon key=\"name_1\">surname:Duan;given-names:J.-H.</infon><infon key=\"name_2\">surname:Song;given-names:Y.-M.</infon><infon key=\"name_3\">surname:Ma;given-names:J.</infon><infon key=\"name_4\">surname:Wang;given-names:F.-D.</infon><infon key=\"name_5\">surname:Lu;given-names:X.</infon><infon key=\"name_6\">surname:Yang;given-names:X.-D.</infon><infon key=\"pub-id_doi\">10.1186/1479-5876-10-148</infon><infon key=\"pub-id_pmid\">22817844</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">J. Transl. Med.</infon><infon key=\"type\">ref</infon><infon key=\"volume\">10</infon><infon key=\"year\">2012</infon><offset>29105</offset><text>Novel HER2 Aptamer Selectively Delivers Cytotoxic Drug to HER2-positive Breast Cancer Cells in Vitro</text></passage><passage><infon key=\"fpage\">173</infon><infon key=\"lpage\">178</infon><infon key=\"name_0\">surname:Kim;given-names:M.Y.</infon><infon key=\"name_1\">surname:Jeong;given-names:S.</infon><infon key=\"pub-id_doi\">10.1089/nat.2011.0283</infon><infon key=\"pub-id_pmid\">21749294</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">Nucleic Acid Ther.</infon><infon key=\"type\">ref</infon><infon key=\"volume\">21</infon><infon key=\"year\">2011</infon><offset>29206</offset><text>In Vitro Selection of RNA Aptamer and Specific Targeting of ErbB2 in Breast Cancer Cells</text></passage><passage><infon key=\"fpage\">18780</infon><infon key=\"lpage\">18785</infon><infon key=\"name_0\">surname:Feener;given-names:E.P.</infon><infon key=\"name_1\">surname:Shen;given-names:W.C.</infon><infon key=\"name_2\">surname:Ryser;given-names:H.J.</infon><infon key=\"pub-id_pmid\">2229041</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">J. Biol. Chem.</infon><infon key=\"type\">ref</infon><infon key=\"volume\">265</infon><infon key=\"year\">1990</infon><offset>29295</offset><text>Cleavage of disulfide bonds in endocytosed macromolecules. A processing not associated with lysosomes or endosomes</text></passage><passage><infon key=\"fpage\">199</infon><infon key=\"lpage\">215</infon><infon key=\"name_0\">surname:Saito;given-names:G.</infon><infon key=\"name_1\">surname:Swanson;given-names:J.A.</infon><infon key=\"name_2\">surname:Lee;given-names:K.-D.</infon><infon key=\"pub-id_doi\">10.1016/S0169-409X(02)00179-5</infon><infon key=\"pub-id_pmid\">12564977</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">Adv. Drug Deliv. Rev.</infon><infon key=\"type\">ref</infon><infon key=\"volume\">55</infon><infon key=\"year\">2003</infon><offset>29410</offset><text>Drug delivery strategy utilizing conjugation via reversible disulfide linkages: Role and site of cellular reducing activities</text></passage><passage><infon key=\"fpage\">3286</infon><infon key=\"lpage\">3306</infon><infon key=\"name_0\">surname:Bauhuber;given-names:S.</infon><infon key=\"name_1\">surname:Hozsa;given-names:C.</infon><infon key=\"name_2\">surname:Breunig;given-names:M.</infon><infon key=\"name_3\">surname:Goepferich;given-names:A.</infon><infon key=\"pub-id_doi\">10.1002/adma.200802453</infon><infon key=\"pub-id_pmid\">20882498</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">Adv. Mater.</infon><infon key=\"type\">ref</infon><infon key=\"volume\">21</infon><infon key=\"year\">2009</infon><offset>29536</offset><text>Delivery of Nucleic Acids via Disulfide-Based Carrier Systems</text></passage><passage><infon key=\"fpage\">335</infon><infon key=\"lpage\">344</infon><infon key=\"name_0\">surname:Kratschmer;given-names:C.</infon><infon key=\"name_1\">surname:Levy;given-names:M.</infon><infon key=\"pub-id_doi\">10.1089/nat.2017.0680</infon><infon key=\"pub-id_pmid\">28945147</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">Nucleic Acid Ther.</infon><infon key=\"type\">ref</infon><infon key=\"volume\">27</infon><infon key=\"year\">2017</infon><offset>29598</offset><text>Effect of Chemical Modifications on Aptamer Stability in Serum</text></passage><passage><infon key=\"fpage\">2532</infon><infon key=\"lpage\">2540</infon><infon key=\"name_0\">surname:Yang;given-names:Q.</infon><infon key=\"name_1\">surname:Deng;given-names:Z.</infon><infon key=\"name_2\">surname:Wang;given-names:D.</infon><infon key=\"name_3\">surname:He;given-names:J.</infon><infon key=\"name_4\">surname:Zhang;given-names:D.</infon><infon key=\"name_5\">surname:Tan;given-names:Y.</infon><infon key=\"name_6\">surname:Peng;given-names:T.</infon><infon key=\"name_7\">surname:Wang;given-names:X.-Q.</infon><infon key=\"name_8\">surname:Tan;given-names:W.</infon><infon key=\"pub-id_doi\">10.1021/jacs.9b12409</infon><infon key=\"pub-id_pmid\">31910340</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">J. Am. Chem. Soc.</infon><infon key=\"type\">ref</infon><infon key=\"volume\">142</infon><infon key=\"year\">2020</infon><offset>29661</offset><text>Conjugating Aptamer and Mitomycin C with Reductant-Responsive Linker Leading to Synergistically Enhanced Anticancer Effect</text></passage><passage><infon key=\"fpage\">84</infon><infon key=\"lpage\">92</infon><infon key=\"name_0\">surname:Erickson;given-names:H.K.</infon><infon key=\"name_1\">surname:Widdison;given-names:W.C.</infon><infon key=\"name_2\">surname:Mayo;given-names:M.F.</infon><infon key=\"name_3\">surname:Whiteman;given-names:K.</infon><infon key=\"name_4\">surname:Audette;given-names:C.</infon><infon key=\"name_5\">surname:Wilhelm;given-names:S.D.</infon><infon key=\"name_6\">surname:Singh;given-names:R.</infon><infon key=\"pub-id_doi\">10.1021/bc900315y</infon><infon key=\"pub-id_pmid\">19891424</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">Bioconjug. Chem.</infon><infon key=\"type\">ref</infon><infon key=\"volume\">21</infon><infon key=\"year\">2010</infon><offset>29784</offset><text>Tumor Delivery and In Vivo Processing of Disulfide-Linked and Thioether-Linked Antibody-Maytansinoid Conjugates</text></passage><passage><infon key=\"fpage\">537</infon><infon key=\"lpage\">550</infon><infon key=\"name_0\">surname:Keefe;given-names:A.D.</infon><infon key=\"name_1\">surname:Pai;given-names:S.</infon><infon key=\"name_2\">surname:Ellington;given-names:A.D.</infon><infon key=\"pub-id_doi\">10.1038/nrd3141</infon><infon key=\"pub-id_pmid\">20592747</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">Nat. Rev. Drug Discov.</infon><infon key=\"type\">ref</infon><infon key=\"volume\">9</infon><infon key=\"year\">2010</infon><offset>29898</offset><text>Aptamers as therapeutics</text></passage><passage><infon key=\"fpage\">25</infon><infon key=\"lpage\">33</infon><infon key=\"name_0\">surname:Burmeister;given-names:P.E.</infon><infon key=\"name_1\">surname:Lewis;given-names:S.D.</infon><infon key=\"name_2\">surname:Silva;given-names:R.F.</infon><infon key=\"name_3\">surname:Preiss;given-names:J.R.</infon><infon key=\"name_4\">surname:Horwitz;given-names:L.R.</infon><infon key=\"name_5\">surname:Pendergrast;given-names:P.S.</infon><infon key=\"name_6\">surname:McCauley;given-names:T.G.</infon><infon key=\"name_7\">surname:Kurz;given-names:J.C.</infon><infon key=\"name_8\">surname:Epstein;given-names:D.M.</infon><infon key=\"name_9\">surname:Wilson;given-names:C.</infon><infon key=\"pub-id_doi\">10.1016/j.chembiol.2004.10.017</infon><infon key=\"pub-id_pmid\">15664512</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">Chem. Biol.</infon><infon key=\"type\">ref</infon><infon key=\"volume\">12</infon><infon key=\"year\">2005</infon><offset>29923</offset><text>Direct In Vitro Selection of a 2'-O-Methyl Aptamer to VEGF</text></passage><passage><infon key=\"fpage\">1</infon><infon key=\"lpage\">26</infon><infon key=\"name_0\">surname:Cassady;given-names:J.M.</infon><infon key=\"name_1\">surname:Chan;given-names:K.K.</infon><infon key=\"name_2\">surname:Floss;given-names:H.G.</infon><infon key=\"name_3\">surname:Leistner;given-names:E.</infon><infon key=\"pub-id_doi\">10.1248/cpb.52.1</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">Chem. Pharm. Bull.</infon><infon key=\"type\">ref</infon><infon key=\"volume\">52</infon><infon key=\"year\">2004</infon><offset>29984</offset><text>Recent Developments in the Maytansinoid Antitumor Agents</text></passage><passage><infon key=\"fpage\">298</infon><infon key=\"lpage\">313</infon><infon key=\"name_0\">surname:Poon;given-names:K.A.</infon><infon key=\"name_1\">surname:Flagella;given-names:K.</infon><infon key=\"name_2\">surname:Beyer;given-names:J.</infon><infon key=\"name_3\">surname:Tibbitts;given-names:J.</infon><infon key=\"name_4\">surname:Kaur;given-names:S.</infon><infon key=\"name_5\">surname:Saad;given-names:O.</infon><infon key=\"name_6\">surname:Yi;given-names:J.-H.</infon><infon key=\"name_7\">surname:Girish;given-names:S.</infon><infon key=\"name_8\">surname:Dybdal;given-names:N.</infon><infon key=\"name_9\">surname:Reynolds;given-names:T.</infon><infon key=\"pub-id_doi\">10.1016/j.taap.2013.09.003</infon><infon key=\"pub-id_pmid\">24035823</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">Toxicol. Appl. Pharmacol.</infon><infon key=\"type\">ref</infon><infon key=\"volume\">273</infon><infon key=\"year\">2013</infon><offset>30041</offset><text>Preclinical safety profile of trastuzumab emtansine (T-DM1): Mechanism of action of its cytotoxic component retained with improved tolerability</text></passage><passage><infon key=\"fpage\">202</infon><infon key=\"lpage\">209</infon><infon key=\"name_0\">surname:Peddi;given-names:P.F.</infon><infon key=\"name_1\">surname:Hurvitz;given-names:S.A.</infon><infon key=\"pub-id_doi\">10.1177/1758834014539183</infon><infon key=\"pub-id_pmid\">25342987</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">Ther. Adv. Med. Oncol.</infon><infon key=\"type\">ref</infon><infon key=\"volume\">6</infon><infon key=\"year\">2014</infon><offset>30185</offset><text>Ado-trastuzumab emtansine (T-DM1) in human epidermal growth factor receptor 2 (HER2)-positive metastatic breast cancer: Latest evidence and clinical potential</text></passage><passage><infon key=\"fpage\">1610</infon><infon key=\"lpage\">1613.e7</infon><infon key=\"name_0\">surname:Ganson;given-names:N.J.</infon><infon key=\"name_1\">surname:Povsic;given-names:T.J.</infon><infon key=\"name_2\">surname:Sullenger;given-names:B.A.</infon><infon key=\"name_3\">surname:Alexander;given-names:J.H.</infon><infon key=\"name_4\">surname:Zelenkofske;given-names:S.L.</infon><infon key=\"name_5\">surname:Sailstad;given-names:J.M.</infon><infon key=\"name_6\">surname:Rusconi;given-names:C.P.</infon><infon key=\"name_7\">surname:Hershfield;given-names:M.S.</infon><infon key=\"pub-id_doi\">10.1016/j.jaci.2015.10.034</infon><infon key=\"pub-id_pmid\">26688515</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">J. Allergy Clin. Immunol.</infon><infon key=\"type\">ref</infon><infon key=\"volume\">137</infon><infon key=\"year\">2016</infon><offset>30344</offset><text>Pre-existing anti-polyethylene glycol antibody linked to first-exposure allergic reactions to pegnivacogin, a PEGylated RNA aptamer</text></passage><passage><infon key=\"elocation-id\">32</infon><infon key=\"name_0\">surname:Dan;given-names:N.</infon><infon key=\"name_1\">surname:Setua;given-names:S.</infon><infon key=\"name_2\">surname:Kashyap;given-names:V.K.</infon><infon key=\"name_3\">surname:Khan;given-names:S.</infon><infon key=\"name_4\">surname:Jaggi;given-names:M.</infon><infon key=\"name_5\">surname:Yallapu;given-names:M.M.</infon><infon key=\"name_6\">surname:Chauhan;given-names:S.C.</infon><infon key=\"pub-id_doi\">10.3390/ph11020032</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">Pharmaceuticals</infon><infon key=\"type\">ref</infon><infon key=\"volume\">11</infon><infon key=\"year\">2018</infon><offset>30478</offset><text>Antibody-Drug Conjugates for Cancer Therapy: Chemistry to Clinical Implications</text></passage><passage><infon key=\"fpage\">869</infon><infon key=\"lpage\">881</infon><infon key=\"name_0\">surname:Perez;given-names:H.L.</infon><infon key=\"name_1\">surname:Cardarelli;given-names:P.M.</infon><infon key=\"name_2\">surname:Deshpande;given-names:S.</infon><infon key=\"name_3\">surname:Gangwar;given-names:S.</infon><infon key=\"name_4\">surname:Schroeder;given-names:G.M.</infon><infon key=\"name_5\">surname:Vite;given-names:G.D.</infon><infon key=\"name_6\">surname:Borzilleri;given-names:R.M.</infon><infon key=\"pub-id_doi\">10.1016/j.drudis.2013.11.004</infon><infon key=\"pub-id_pmid\">24239727</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">Drug Discov. Today</infon><infon key=\"type\">ref</infon><infon key=\"volume\">19</infon><infon key=\"year\">2014</infon><offset>30558</offset><text>Antibody-drug conjugates: Current status and future directions</text></passage><passage><infon key=\"fpage\">3470</infon><infon key=\"lpage\">3479</infon><infon key=\"name_0\">surname:Kamath;given-names:A.V.</infon><infon key=\"name_1\">surname:Iyer;given-names:S.</infon><infon key=\"pub-id_doi\">10.1007/s11095-014-1584-z</infon><infon key=\"pub-id_pmid\">25446773</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">Pharm. Res.</infon><infon key=\"type\">ref</infon><infon key=\"volume\">32</infon><infon key=\"year\">2015</infon><offset>30623</offset><text>Preclinical Pharmacokinetic Considerations for the Development of Antibody Drug Conjugates</text></passage><passage><infon key=\"elocation-id\">10</infon><infon key=\"name_0\">surname:Lucas;given-names:A.T.</infon><infon key=\"name_1\">surname:Price;given-names:L.S.L.</infon><infon key=\"name_2\">surname:Schorzman;given-names:A.N.</infon><infon key=\"name_3\">surname:Storrie;given-names:M.</infon><infon key=\"name_4\">surname:Piscitelli;given-names:J.A.</infon><infon key=\"name_5\">surname:Razo;given-names:J.</infon><infon key=\"name_6\">surname:Zamboni;given-names:W.C.</infon><infon key=\"pub-id_doi\">10.3390/antib7010010</infon><infon key=\"pub-id_pmid\">31544862</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">Antibodies</infon><infon key=\"type\">ref</infon><infon key=\"volume\">7</infon><infon key=\"year\">2018</infon><offset>30714</offset><text>Factors Affecting the Pharmacology of Antibody-Drug Conjugates</text></passage><passage><infon key=\"file\">ijms-21-09764-g001.jpg</infon><infon key=\"id\">ijms-21-09764-f001</infon><infon key=\"section_type\">FIG</infon><infon key=\"type\">fig_caption</infon><offset>30779</offset><text>The structure and characterization of HER2 aptamer. (A) The secondary structure of HER2 RNA aptamer was estimated using the secondary structure prediction service of Mathews Lab. (B) Unmodified RNA aptamer and 2'-pyrimidine-modified RNA aptamer incubated in 10% FBS-containing medium were analyzed by gel retardation analysis (*** p &lt; 0.001). (C) The dissociation constant of HER2 RNA aptamer was performed using the bead binding assay and analyzed by qPCR. (D) In vitro cell bindings of FITC-labeled HER2 RNA aptamer to BT-474, A549, MCF7, and MDA-MB-231 cells were analyzed using confocal microscopy.</text></passage><passage><infon key=\"file\">ijms-21-09764-g002.jpg</infon><infon key=\"id\">ijms-21-09764-f002</infon><infon key=\"section_type\">FIG</infon><infon key=\"type\">fig_caption</infon><offset>31384</offset><text>Structure and analysis of HER2 aptamer-DM1 conjugate. (A) The aptamer-DM1 conjugate consists of polyethylene glycol (20kDa) and DM1 coupled to the 5' end and 3' end of HER2 aptamer, respectively. The HER2 aptamer-DM1 conjugate was purified and analyzed by HPLC (B) and mass spectrometry (C).</text></passage><passage><infon key=\"file\">ijms-21-09764-g003.jpg</infon><infon key=\"id\">ijms-21-09764-f003</infon><infon key=\"section_type\">FIG</infon><infon key=\"type\">fig_caption</infon><offset>31680</offset><text>The in vitro cytotoxicity of HER2 aptamer-DM1 conjugate. BT-474 (A) or MDA-MB-231 (B) cells were treated with the HER2 aptamer-DM1 conjugate and their viability was analyzed using WST-1 assay 72 h later.</text></passage><passage><infon key=\"file\">ijms-21-09764-g004.jpg</infon><infon key=\"id\">ijms-21-09764-f004</infon><infon key=\"section_type\">FIG</infon><infon key=\"type\">fig_caption</infon><offset>31884</offset><text>The in vivo tumor growth inhibition by HER2 aptamer-DM1 conjugate. BT-474 cells were subcutaneously injected into the right flank of nude mice. When the size of tumor reached a volume of ~100 mm3, the mice began to be treated with the ApDC or free DM1. The mice were treated trice every three days, and the tumor size (A) and mouse body weight (B) were measured every other day. Red arrows indicate the injection days. * p &lt; 0.05, *** p &lt; 0.001.</text></passage><passage><infon key=\"file\">ijms-21-09764-g005.jpg</infon><infon key=\"id\">ijms-21-09764-f005</infon><infon key=\"section_type\">FIG</infon><infon key=\"type\">fig_caption</infon><offset>32330</offset><text>Histochemical changes by the administration of HER2 aptamer-DM1 conjugate. The tissues of tumor, spleen, and liver were stained with a terminal deoxynucleotidyl transferase dUTP nick end labeling (TUNEL) kit on the final day of in vivo study and examined with a microscope (200x).</text></passage><passage><infon key=\"file\">ijms-21-09764-t001.xml</infon><infon key=\"id\">ijms-21-09764-t001</infon><infon key=\"section_type\">TABLE</infon><infon key=\"type\">table_caption</infon><offset>32612</offset><text>Hematological and biochemical parameters measured after ApDC treatment (n = 3).</text></passage><passage><infon key=\"file\">ijms-21-09764-t001.xml</infon><infon key=\"id\">ijms-21-09764-t001</infon><infon key=\"section_type\">TABLE</infon><infon key=\"type\">table</infon><infon key=\"xml\">&lt;?xml version=&quot;1.0&quot; encoding=&quot;UTF-8&quot;?&gt;\n",
      "&lt;table frame=&quot;hsides&quot; rules=&quot;groups&quot;&gt;&lt;thead&gt;&lt;tr&gt;&lt;th align=&quot;center&quot; valign=&quot;middle&quot; style=&quot;border-top:solid thin&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;\n",
      "&lt;/th&gt;&lt;th align=&quot;center&quot; valign=&quot;middle&quot; style=&quot;border-top:solid thin&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;\n",
      "&lt;/th&gt;&lt;th colspan=&quot;2&quot; align=&quot;center&quot; valign=&quot;middle&quot; style=&quot;border-top:solid thin;border-bottom:solid thin&quot; rowspan=&quot;1&quot;&gt;Control&lt;break/&gt;(PBS)&lt;/th&gt;&lt;th colspan=&quot;2&quot; align=&quot;center&quot; valign=&quot;middle&quot; style=&quot;border-top:solid thin;border-bottom:solid thin&quot; rowspan=&quot;1&quot;&gt;DM1&lt;break/&gt;(60 mug/kg)&lt;/th&gt;&lt;th colspan=&quot;2&quot; align=&quot;center&quot; valign=&quot;middle&quot; style=&quot;border-top:solid thin;border-bottom:solid thin&quot; rowspan=&quot;1&quot;&gt;ApDC&lt;break/&gt;(3.85 mg/kg)&lt;/th&gt;&lt;/tr&gt;&lt;tr&gt;&lt;th align=&quot;center&quot; valign=&quot;middle&quot; style=&quot;border-bottom:solid thin&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;\n",
      "&lt;/th&gt;&lt;th align=&quot;center&quot; valign=&quot;middle&quot; style=&quot;border-bottom:solid thin&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;\n",
      "&lt;/th&gt;&lt;th align=&quot;center&quot; valign=&quot;middle&quot; style=&quot;border-bottom:solid thin&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;Mean&lt;/th&gt;&lt;th align=&quot;center&quot; valign=&quot;middle&quot; style=&quot;border-bottom:solid thin&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;SD&lt;/th&gt;&lt;th align=&quot;center&quot; valign=&quot;middle&quot; style=&quot;border-bottom:solid thin&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;Mean&lt;/th&gt;&lt;th align=&quot;center&quot; valign=&quot;middle&quot; style=&quot;border-bottom:solid thin&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;SD&lt;/th&gt;&lt;th align=&quot;center&quot; valign=&quot;middle&quot; style=&quot;border-bottom:solid thin&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;Mean&lt;/th&gt;&lt;th align=&quot;center&quot; valign=&quot;middle&quot; style=&quot;border-bottom:solid thin&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;SD&lt;/th&gt;&lt;/tr&gt;&lt;/thead&gt;&lt;tbody&gt;&lt;tr&gt;&lt;td rowspan=&quot;5&quot; align=&quot;center&quot; valign=&quot;middle&quot; style=&quot;border-bottom:solid thin&quot; colspan=&quot;1&quot;&gt;Hematology,&lt;break/&gt;10&lt;sup&gt;3&lt;/sup&gt;/muL&lt;/td&gt;&lt;td align=&quot;center&quot; valign=&quot;middle&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;Platelet, counts&lt;/td&gt;&lt;td align=&quot;center&quot; valign=&quot;middle&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;697.50&lt;/td&gt;&lt;td align=&quot;center&quot; valign=&quot;middle&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;64.16&lt;/td&gt;&lt;td align=&quot;center&quot; valign=&quot;middle&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;632.40&lt;/td&gt;&lt;td align=&quot;center&quot; valign=&quot;middle&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;101.64&lt;/td&gt;&lt;td align=&quot;center&quot; valign=&quot;middle&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;425.33&lt;/td&gt;&lt;td align=&quot;center&quot; valign=&quot;middle&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;66.11&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td align=&quot;center&quot; valign=&quot;middle&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;Red blood cells, M/muL&lt;/td&gt;&lt;td align=&quot;center&quot; valign=&quot;middle&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;8.38&lt;/td&gt;&lt;td align=&quot;center&quot; valign=&quot;middle&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;0.27&lt;/td&gt;&lt;td align=&quot;center&quot; valign=&quot;middle&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;8.08&lt;/td&gt;&lt;td align=&quot;center&quot; valign=&quot;middle&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;0.35&lt;/td&gt;&lt;td align=&quot;center&quot; valign=&quot;middle&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;7.69&lt;/td&gt;&lt;td align=&quot;center&quot; valign=&quot;middle&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;0.34&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td align=&quot;center&quot; valign=&quot;middle&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;Hematocrits, %&lt;/td&gt;&lt;td align=&quot;center&quot; valign=&quot;middle&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;45.86&lt;/td&gt;&lt;td align=&quot;center&quot; valign=&quot;middle&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;2.37&lt;/td&gt;&lt;td align=&quot;center&quot; valign=&quot;middle&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;46.38&lt;/td&gt;&lt;td align=&quot;center&quot; valign=&quot;middle&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;1.41&lt;/td&gt;&lt;td align=&quot;center&quot; valign=&quot;middle&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;43.17&lt;/td&gt;&lt;td align=&quot;center&quot; valign=&quot;middle&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;1.20&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td align=&quot;center&quot; valign=&quot;middle&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;Neutrophil, absolute&lt;/td&gt;&lt;td align=&quot;center&quot; valign=&quot;middle&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;0.54&lt;/td&gt;&lt;td align=&quot;center&quot; valign=&quot;middle&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;0.09&lt;/td&gt;&lt;td align=&quot;center&quot; valign=&quot;middle&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;0.37&lt;/td&gt;&lt;td align=&quot;center&quot; valign=&quot;middle&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;0.15&lt;/td&gt;&lt;td align=&quot;center&quot; valign=&quot;middle&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;0.31&lt;/td&gt;&lt;td align=&quot;center&quot; valign=&quot;middle&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;0.11&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td align=&quot;center&quot; valign=&quot;middle&quot; style=&quot;border-bottom:solid thin&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;Lymphocyte&lt;/td&gt;&lt;td align=&quot;center&quot; valign=&quot;middle&quot; style=&quot;border-bottom:solid thin&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;2.22&lt;/td&gt;&lt;td align=&quot;center&quot; valign=&quot;middle&quot; style=&quot;border-bottom:solid thin&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;0.50&lt;/td&gt;&lt;td align=&quot;center&quot; valign=&quot;middle&quot; style=&quot;border-bottom:solid thin&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;1.78&lt;/td&gt;&lt;td align=&quot;center&quot; valign=&quot;middle&quot; style=&quot;border-bottom:solid thin&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;0.89&lt;/td&gt;&lt;td align=&quot;center&quot; valign=&quot;middle&quot; style=&quot;border-bottom:solid thin&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;1.55&lt;/td&gt;&lt;td align=&quot;center&quot; valign=&quot;middle&quot; style=&quot;border-bottom:solid thin&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;0.18&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td rowspan=&quot;5&quot; align=&quot;center&quot; valign=&quot;middle&quot; style=&quot;border-bottom:solid thin&quot; colspan=&quot;1&quot;&gt;Clinical chemistry,&lt;break/&gt;U/L&lt;/td&gt;&lt;td align=&quot;center&quot; valign=&quot;middle&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;ALT&lt;/td&gt;&lt;td align=&quot;center&quot; valign=&quot;middle&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;26.28&lt;/td&gt;&lt;td align=&quot;center&quot; valign=&quot;middle&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;6.33&lt;/td&gt;&lt;td align=&quot;center&quot; valign=&quot;middle&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;32.29&lt;/td&gt;&lt;td align=&quot;center&quot; valign=&quot;middle&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;14.33&lt;/td&gt;&lt;td align=&quot;center&quot; valign=&quot;middle&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;42.93&lt;/td&gt;&lt;td align=&quot;center&quot; valign=&quot;middle&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;14.86&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td align=&quot;center&quot; valign=&quot;middle&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;AST&lt;/td&gt;&lt;td align=&quot;center&quot; valign=&quot;middle&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;89.46&lt;/td&gt;&lt;td align=&quot;center&quot; valign=&quot;middle&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;17.86&lt;/td&gt;&lt;td align=&quot;center&quot; valign=&quot;middle&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;89.20&lt;/td&gt;&lt;td align=&quot;center&quot; valign=&quot;middle&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;11.66&lt;/td&gt;&lt;td align=&quot;center&quot; valign=&quot;middle&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;95.35&lt;/td&gt;&lt;td align=&quot;center&quot; valign=&quot;middle&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;20.89&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td align=&quot;center&quot; valign=&quot;middle&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;ALP&lt;/td&gt;&lt;td align=&quot;center&quot; valign=&quot;middle&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;295.11&lt;/td&gt;&lt;td align=&quot;center&quot; valign=&quot;middle&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;99.65&lt;/td&gt;&lt;td align=&quot;center&quot; valign=&quot;middle&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;292.01&lt;/td&gt;&lt;td align=&quot;center&quot; valign=&quot;middle&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;84.44&lt;/td&gt;&lt;td align=&quot;center&quot; valign=&quot;middle&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;211.76&lt;/td&gt;&lt;td align=&quot;center&quot; valign=&quot;middle&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;44.13&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td align=&quot;center&quot; valign=&quot;middle&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;GGT&lt;/td&gt;&lt;td align=&quot;center&quot; valign=&quot;middle&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;4.97&lt;/td&gt;&lt;td align=&quot;center&quot; valign=&quot;middle&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;1.00&lt;/td&gt;&lt;td align=&quot;center&quot; valign=&quot;middle&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;7.02&lt;/td&gt;&lt;td align=&quot;center&quot; valign=&quot;middle&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;2.11&lt;/td&gt;&lt;td align=&quot;center&quot; valign=&quot;middle&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;6.28&lt;/td&gt;&lt;td align=&quot;center&quot; valign=&quot;middle&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;3.67&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td align=&quot;center&quot; valign=&quot;middle&quot; style=&quot;border-bottom:solid thin&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;Total bilirubin&lt;/td&gt;&lt;td align=&quot;center&quot; valign=&quot;middle&quot; style=&quot;border-bottom:solid thin&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;0.06&lt;/td&gt;&lt;td align=&quot;center&quot; valign=&quot;middle&quot; style=&quot;border-bottom:solid thin&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;0.03&lt;/td&gt;&lt;td align=&quot;center&quot; valign=&quot;middle&quot; style=&quot;border-bottom:solid thin&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;0.07&lt;/td&gt;&lt;td align=&quot;center&quot; valign=&quot;middle&quot; style=&quot;border-bottom:solid thin&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;0.03&lt;/td&gt;&lt;td align=&quot;center&quot; valign=&quot;middle&quot; style=&quot;border-bottom:solid thin&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;0.07&lt;/td&gt;&lt;td align=&quot;center&quot; valign=&quot;middle&quot; style=&quot;border-bottom:solid thin&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;0.02&lt;/td&gt;&lt;/tr&gt;&lt;/tbody&gt;&lt;/table&gt;\n",
      "</infon><offset>32692</offset><text>\t\tControl(PBS)\tDM1(60 mug/kg)\tApDC(3.85 mg/kg)\t \t\t\tMean\tSD\tMean\tSD\tMean\tSD\t \tHematology,103/muL\tPlatelet, counts\t697.50\t64.16\t632.40\t101.64\t425.33\t66.11\t \tRed blood cells, M/muL\t8.38\t0.27\t8.08\t0.35\t7.69\t0.34\t \tHematocrits, %\t45.86\t2.37\t46.38\t1.41\t43.17\t1.20\t \tNeutrophil, absolute\t0.54\t0.09\t0.37\t0.15\t0.31\t0.11\t \tLymphocyte\t2.22\t0.50\t1.78\t0.89\t1.55\t0.18\t \tClinical chemistry,U/L\tALT\t26.28\t6.33\t32.29\t14.33\t42.93\t14.86\t \tAST\t89.46\t17.86\t89.20\t11.66\t95.35\t20.89\t \tALP\t295.11\t99.65\t292.01\t84.44\t211.76\t44.13\t \tGGT\t4.97\t1.00\t7.02\t2.11\t6.28\t3.67\t \tTotal bilirubin\t0.06\t0.03\t0.07\t0.03\t0.07\t0.02\t \t</text></passage><passage><infon key=\"file\">ijms-21-09764-t001.xml</infon><infon key=\"id\">ijms-21-09764-t001</infon><infon key=\"section_type\">TABLE</infon><infon key=\"type\">table_footnote</infon><offset>33284</offset><text>Hematological parameters and biochemical parameters were measured with a hematology system and an automated biochemistry analyzer 3 days after inoculation, respectively.</text></passage></document>\n",
      "</collection>\n"
     ]
    }
   ],
   "source": [
    "os.makedirs(\"papers\", exist_ok=True)\n",
    "if os.path.isfile(f\"papers/{PMC}.xml\"):\n",
    "    with open(f\"papers/{PMC}.xml\", 'r', encoding='utf-8') as f:\n",
    "        paper_xml = f.read()\n",
    "else:\n",
    "    paper_xml = utils.fetch_xml(PMC).replace(\"|\", \"/\")\n",
    "    with open(f\"papers/{PMC}.xml\", \"w\", encoding=\"utf-8\") as f:\n",
    "        f.write(paper_xml)\n",
    "print(paper_xml)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "id": "90330ec3",
   "metadata": {},
   "outputs": [],
   "source": [
    "soup = BeautifulSoup(paper_xml, \"lxml-xml\")\n",
    "\n",
    "#abstract\n",
    "abstract_passages = [p for p in soup.find_all(\"passage\") if p.find(\"infon\", {\"key\": \"section_type\"}) and \n",
    "        p.find(\"infon\", {\"key\": \"section_type\"}).text.strip().upper() == \"ABSTRACT\"]\n",
    "abstract_text_list = [p.find(\"text\").get_text() for p in abstract_passages if p.find(\"text\")]\n",
    "abstract = \"\\n\".join(abstract_text_list)\n",
    "\n",
    "#methods\n",
    "methods_passages = [p for p in soup.find_all(\"passage\") if p.find(\"infon\", {\"key\": \"section_type\"}) and \n",
    "        p.find(\"infon\", {\"key\": \"section_type\"}).text.strip().upper() == \"METHODS\"]\n",
    "methods_text_list = [p.find(\"text\").get_text() for p in methods_passages if p.find(\"text\")]\n",
    "methods = \"\\n\".join(methods_text_list)\n",
    "\n",
    "#results\n",
    "results_passages = [p for p in soup.find_all(\"passage\") if p.find(\"infon\", {\"key\": \"section_type\"}) and \n",
    "        p.find(\"infon\", {\"key\": \"section_type\"}).text.strip().upper() == \"RESULTS\"]\n",
    "results_title_passages = [p for p in soup.find_all(\"passage\") if p.find(\"infon\", {\"key\": \"section_type\"}) and \n",
    "        p.find(\"infon\", {\"key\": \"section_type\"}).text.strip().upper() == \"RESULTS\" and\n",
    "        p.find(\"infon\", {\"key\": \"type\"}) and \"TITLE\" in p.find(\"infon\", {\"key\": \"type\"}).text.strip().upper()]\n",
    "results_text_list = [p.find(\"text\").get_text() for p in results_passages if p.find(\"text\")]\n",
    "results_title_list = [p.find(\"text\").get_text() for p in results_title_passages if p.find(\"text\")]\n",
    "results = \"\\n\".join(results_text_list)\n",
    "\n",
    "#discussion\n",
    "discussion_passages = [p for p in soup.find_all(\"passage\") if p.find(\"infon\", {\"key\": \"section_type\"}) and \n",
    "        p.find(\"infon\", {\"key\": \"section_type\"}).text.strip().upper() == \"DISCUSSION\"]\n",
    "discussion_text_list = [p.find(\"text\").get_text() for p in discussion_passages if p.find(\"text\")]\n",
    "discussion = \"\\n\".join(discussion_text_list)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "id": "01bb6ae3",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "control|mouse|4||||||| \n",
      "PBS|mouse|4||||||| \n",
      "free DM1|mouse|4|DM1|60|μg/kg||| \n",
      "ApDC|mouse|4|ApDC|2.7|mg/kg||| \n",
      "DM1 low|mouse| |DM1|12|μg/kg||| \n",
      "DM1 medium|mouse| |DM1|60|μg/kg||| \n",
      "DM1 high|mouse| |DM1|300|μg/kg|||\n",
      "[['control', 'mouse', '4', '', '', '', '', '', '', ''], ['PBS', 'mouse', '4', '', '', '', '', '', '', ''], ['free DM1', 'mouse', '4', 'DM1', '60', 'μg/kg', '', '', ''], ['ApDC', 'mouse', '4', 'ApDC', '2.7', 'mg/kg', '', '', ''], ['DM1 low', 'mouse', ' ', 'DM1', '12', 'μg/kg', '', '', ''], ['DM1 medium', 'mouse', ' ', 'DM1', '60', 'μg/kg', '', '', ''], ['DM1 high', 'mouse', ' ', 'DM1', '300', 'μg/kg', '', '', '']]\n"
     ]
    }
   ],
   "source": [
    "text_groups = []\n",
    "group_names = []\n",
    "for row in utils.groups_from_text(PMC, abstract, methods, results):\n",
    "    if len(row) not in [4, 6, 7, 9] and len(row) < 10:\n",
    "        print(\"ERROR: Invalid text_groups label\")\n",
    "        continue\n",
    "    text_group = {}\n",
    "    text_group[\"group\"] = row[0]\n",
    "    text_group[\"animal_model\"] = row[1] if row[1] else None\n",
    "    text_group[\"sample_size\"] = row[2] if row[2] else None\n",
    "    if len(row) == 4:\n",
    "        text_group[\"treatment1\"] = None\n",
    "        text_group[\"dose1\"] = None\n",
    "        text_group[\"units1\"] = None\n",
    "        text_group[\"treatment2\"] = None\n",
    "        text_group[\"dose2\"] = None\n",
    "        text_group[\"units2\"] = None\n",
    "    elif len(row) == 6:\n",
    "        text_group[\"treatment1\"] = row[3]\n",
    "        text_group[\"dose1\"] = row[4]\n",
    "        text_group[\"units1\"] = row[5]\n",
    "        text_group[\"treatment2\"] = None\n",
    "        text_group[\"dose2\"] = None\n",
    "        text_group[\"units2\"] = None\n",
    "    else:\n",
    "        text_group[\"treatment1\"] = row[3]\n",
    "        text_group[\"dose1\"] = row[4]\n",
    "        text_group[\"units1\"] = row[5]\n",
    "        text_group[\"treatment2\"] = row[6]\n",
    "        text_group[\"dose2\"] = row[7]\n",
    "        text_group[\"units2\"] = row[8]\n",
    "    text_groups.append(text_group)\n",
    "    group_names.append(row[0].strip().lower())\n",
    "text_groups_df = pd.DataFrame(data=text_groups, index=group_names)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "id": "68b3d31a",
   "metadata": {},
   "outputs": [],
   "source": [
    "BATCH_SIZE = 5\n",
    "paragraph_list = []\n",
    "last_title = \"Results\"\n",
    "for paragraph in results_text_list:\n",
    "    if paragraph in results_title_list:\n",
    "        last_title = paragraph\n",
    "    else:\n",
    "        paragraph_list.append({\"title\": last_title, \"text\": paragraph})\n",
    "\n",
    "batch_list = []\n",
    "for index in range(0, len(paragraph_list), BATCH_SIZE):\n",
    "    batch = []\n",
    "    for i in range(5):\n",
    "        if index+i >= len(paragraph_list): break\n",
    "        batch.append(paragraph_list[index+i])\n",
    "    utils.label_batch(PMC, methods, text_groups, batch)\n",
    "    batch_list.append(batch)\n",
    "        "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "id": "7250c987",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Label: False\n",
      "---\n",
      "It is known that RNA molecules are generally unstable and easily degraded by RNases in various environments. However, the stability of RNA can be secured by various conventional modification methods, and the RNA aptamer used in the present study (Figure 1A) was also modified at the pyrimidine by 2'-fluorinated pyrimidine, thereby achieving much higher stability than the unmodified RNA aptamer. The gel retardation analysis of aptamer molecules (Figure 1B) shows the relative stabilities of unmodified RNA aptamer and 2'-pyrimidine-modified RNA aptamer in 10% FBS-containing medium. The results of gel retardation analysis of the unmodified RNA aptamer revealed that the unmodified molecules degraded more than 50% in 1 h. On the other hand, 2'-pyrimidine-modified RNA aptamers were able to maintain their structure over 50% even 48 h later.\n",
      "------------------------------------------------\n",
      "Label: False\n",
      "---\n",
      "The 2'-pyrimidine-modified RNA aptamers utilized in this study exhibited a strong binding affinity to HER2 receptors. According to the qPCR measurement, the dissociation constant of the HER2 aptamer was 16.5 +- 4.4 nM (Figure 1C). Their HER2-specific binding was also confirmed by microscopic assessment (Figure 1D). Fluorescent HER2 aptamers effectively bind to BT-474 and A549 cancer cells, both over-expressing HER2 receptors. Meanwhile, the same molecules little bind MCF7 low-expressing HER2 and MDA-MB-231 cells deficient of HER2. These results suggest that the 2'-pyrimidine-modified HER2 RNA aptamer prepared for this study can be utilized as a ligand for HER2-specific delivery of anticancer drugs in vivo.\n",
      "------------------------------------------------\n",
      "Label: False\n",
      "---\n",
      "For the synthesis of HER2 aptamer-drug conjugate (ApDC), a 20 kDa PEG molecule was coupled to the 5' end of HER2 RNA aptamer and a molecule of DM1 was then attached to the other side 3' end (Figure 2A). DM1 is a widely utilized anticancer drug and has a structural advantage at the coupling to carrier molecules since it is a thiol form of mertansine. The disulfide bond between DM1 and the 3' end of the aptamer is expected to be broken in the reducing environment of early and late endosomes. The linkage fragility of ApDC could facilitate the release of DM1 from the endocytic pathway, elevating the anticancer therapeutic efficacy of the drug. Also, PEGylation of the aptamer at 5' end could enhance the half-life of ApDC in circulation by avoiding renal clearance. The prolonged circulation of the ApDC is expected to increase the availability and targetability of ApDC molecules to target cancer cells, presumably resulting in increased anticancer therapeutic efficacy.\n",
      "------------------------------------------------\n",
      "Label: False\n",
      "---\n",
      "According to the HPLC analysis (Figure 2B) and mass spectrometry (Figure 2C), the final product of ApDC exhibited 98% of conjugation yield and ~33,000 g/mole of molecular weight. The strict and stable procedure of ApDC synthesis provide strong advantages over biological ligand conjugates in terms of structural modification and quality control. Because of the stability of aptamer chemistry, there was little variation from batch to batch and their derivatives such as aptamer-drug conjugates were able to maintain targetability as well as therapeutic efficacy.\n",
      "------------------------------------------------\n",
      "Label: False\n",
      "---\n",
      "The target-specific cytotoxicity of HER2 aptamer-DM1 conjugate was analyzed by the WST-1 assay (n = 8) on HER2-positive BT-474 and HER2-negative MDA-MB-231 cancer cells. In the target BT-474 cancer cells, free DM1 and the HER2 ApDC exhibited similar levels of IC50 value, 230.4 nM and 201.7 nM, respectively (Figure 3A). However, the MDA-MB-231 cells deficient of HER2 did not respond to the HER2 ApDC while the same cells showed cytotoxic response to the free DM 1 (Figure 3B). The ApDC hardly affect the viability of MDA-MB-231 even at the highest concentration tested (2 muM). This implies that the cytotoxicity of the ApDC is dependent upon the HER2 expressed on the surface of cancer cells.\n",
      "------------------------------------------------\n",
      "Label: False\n",
      "---\n",
      "The free form of DM1 was able to penetrate the plasma membranes regardless HER2 expression. Therefore, it can be stated that the ApDC has cell-specific cytotoxicity. Presumably, the aptamer-drug conjugates were too big and too hydrophilic to directly penetrate through the plasma membranes via a certain transport and appeared to be internalized via HER2-mediated endocytosis. The disulfide bond was easily cleaved in the in the reducing environment of endocytic vesicles, resulting in facilitated escapes of the freed DM1 from endosomes. Free HER aptamers did not show any cell toxicity even to BT-474 cells at the highest concentrations. This implies the cytotoxicity of the ApDC solely comes from the drug DM1, not from the HER2 aptamer.\n",
      "------------------------------------------------\n",
      "Label: False\n",
      "---\n",
      "In order to examine the antitumoral activity of the HER2 aptamer-DM1 conjugate, the HER2 ApDC was intravenously administered to mice carrying BT-474 tumors three times at intervals of 3 days. Tumor volumes and body weights were measured for a total of 36 days. Compared with the saline-treated group of mice, both the free DM1 and the ApDC exhibited a reduction of tumor size just after treatment (Figure 4A) and then more effectively suppressed the growth of the tumor. When the tumor-carrying mice were treated with the same amount of DM1 (60 mug/kg), the tumors of mice injected with the ApDC were statically smaller than those with free DM1 after the 19th-day post-treatment. These results tell that the chemically modified DM1 is still effective in inhibition of tumor growth and the HER2 aptamers conjugated to the drugs presumably enhance the anti-cancer therapeutic efficacy of DM1.\n",
      "------------------------------------------------\n",
      "Label: False\n",
      "---\n",
      "DM1 is known to be a strong inhibitor for microtubule assembly, yielding inherent systemic toxicity. However, the ApDC and free DM1 appeared to be safe at the treated dose, 60 mug DM1/kg. The mice administered with the suggested dose of ApDC or free DM1 did not show any significant body weight loss during treatment (Figure 4B). Aberrant side-effects of DM1 under in vivo conditions were already reported and based on the analyses, preclinical and clinical trials of DM1 conjugates have been allowed and then proceeded. Since any systemic cytotoxicity of RNA aptamers has not been reported and only some allergic reaction of PEG molecules has been reported, the enhanced anti-cancer therapeutic efficacy would certainly result from the inherent toxicity of DM1 which were more efficiently delivered to the cancer cells. However, at this moment it is uncertain whether providing HER2 targetability and PEG conjugation to DM1 may reduce the systemic toxicity of the drug. Upscaled animal studies would further verify the enhanced safety of the ApDC in vivo.\n",
      "------------------------------------------------\n",
      "Label: True\n",
      "---\n",
      "At the end point of tumor growth analysis, major internal organs such as tumor, heart, liver, lungs, and spleen were resected, and any forms of cell death were then analyzed through hematoxylin and eosin (H&E) staining and terminal deoxynucleotidyl transferase dUTP nick end labeling (TUNEL) assay. The TUNEL assay verified some apoptotic cells in the superficial area of the tumor tissue of the ApDC-administered mice (Figure 5). In the case of mice treated with free DM1, apoptotic cells were found in tumor tissues and spleen as well.\n",
      "------------------------------------------------\n",
      "Label: True\n",
      "---\n",
      "According to the microscopic analysis, no significant infiltration of immune cells was found in the tissues of major organs of mice treated with the ApDC or free DM1 at 60 mug DM1/kg (Figure S1). These results imply that the intravenous administration of the target specific ApDC appeared to be rather a safer procedure to treat cancer, compared to the direct administration of free DM1. The tumor-targeting aptamer conjugated to DM1 molecule presumably enhanced DM1 accumulation in the tumor tissue. Meanwhile, the free DM1 happened to be more off-targeted to spleen resulting in aberrant tissue damages, due to a lack of tumor-targeting capability.\n",
      "------------------------------------------------\n",
      "Label: True\n",
      "---\n",
      "In order to further verify the in vivo toxicity of HER2 aptamer-DM1 conjugate, mice were intravenously injected with varied concentrations of free DM1 and the ApDC. Two days later, the hematological and biochemical parameters were analyzed using the blood infraorbital taken. According to the hematological analysis, the counts of platelets, RBC, and hematocrit were not significantly changed at the test dose, 60 mug DM1/kg and the equivalent dose of ApDC (2.7 mg/kg). Any significant changes in the numbers of white blood cells including monocytes, eosinophils, and basophils were not monitored either. These imply that the administration of ApDC and free DM1 at the controlled dose does not elicit serious side-effects such as aberrant immune stimulation and anemia, reported major side-effects of DM1. The biochemical analysis (Table 1) also showed that the administration of ApDC and free DM1 at the dose of 60 mug DM1/kg did not seriously affect the hepatic functions and hematologic characteristics of a mouse. These results suggest that the ApDC and DM1 can be systemically utilized for cancer treatment at the controlled doses.\n",
      "------------------------------------------------\n",
      "Label: False\n",
      "---\n",
      "A reducing environment of the body may readily break the disulfide bonding between the aptamer and free DM1 and the premature exposure of free DM1 in the blood circulation may cause the aberrant side-effects of DM1. Therefore, when combining a delivery carrier with a drug, it is important to consider the stability of the chemical linkage in the body. The disulfide linkage for the combination of aptamer molecules and drugs can enhance the efficiency of the drug but may not guarantee the stability of the conjugates in the body. In order to make a further analysis regarding the in vivo side-effects of ApDC, the distribution and pharmacokinetics of the drug in the body should be analyzed. A more systematic analysis on a larger scale would tell us a genuine benefit of ApDC in terms of efficacy and toxicity, differentiating from those of free DM1. At the same time, to expand the utilization of aptamer in drug targeting, it is necessary to prove beneficial merits of aptamer-mediated targeting over antibody-mediated drug delivery.\n",
      "------------------------------------------------\n"
     ]
    }
   ],
   "source": [
    "for batch in batch_list:\n",
    "    for paragraph in batch:\n",
    "        print(f\"Label: {paragraph[\"label\"]}\")\n",
    "        print(\"---\")\n",
    "        print(paragraph[\"text\"])\n",
    "        print(\"------------------------------------------------\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "id": "d8986bb4",
   "metadata": {},
   "outputs": [],
   "source": [
    "num_output = []\n",
    "freq_output = []\n",
    "desc_output = []\n",
    "for batch in batch_list:\n",
    "    for paragraph in batch:\n",
    "        if paragraph[\"label\"]:\n",
    "            paragraph_data = utils.data_from_text(PMC, methods, text_groups_df, paragraph)\n",
    "            num_output = num_output + paragraph_data[\"num\"]\n",
    "            freq_output = freq_output + paragraph_data[\"freq\"]\n",
    "            desc_output = desc_output + paragraph_data[\"desc\"]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "4501d48e",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "env",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.12.10"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
