{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "50c9e7d2",
   "metadata": {},
   "outputs": [],
   "source": [
    "import table_utils\n",
    "import process_paper_utils as utils\n",
    "import os\n",
    "from bs4 import BeautifulSoup\n",
    "import html\n",
    "from datetime import datetime\n",
    "import pandas as pd"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "e44ed7db",
   "metadata": {},
   "outputs": [],
   "source": [
    "#Enter PMC here:\n",
    "PMC = \"PMC9322224\""
   ]
  },
  {
   "cell_type": "markdown",
   "id": "5bb5afc3",
   "metadata": {},
   "source": [
    "Imports paper from PMC"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "64c1b7b3",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "<?xml version=\"1.0\" encoding=\"UTF-8\"?><!DOCTYPE collection SYSTEM \"BioC.dtd\"><collection><source>BioC-API</source><date>20250617</date><key>collection.key</key><document><id>PMC9322224</id><infon key=\"license\">CC BY-NC-ND</infon><passage><infon key=\"article-id_doi\">10.1016/j.jfda.2017.11.001</infon><infon key=\"article-id_pmc\">PMC9322224</infon><infon key=\"article-id_pmid\">29567262</infon><infon key=\"article-id_publisher-id\">jfda-26-02-903</infon><infon key=\"fpage\">903</infon><infon key=\"issue\">2</infon><infon key=\"kwd\">Aspartame Memory Folate deficient rat model Oxidative stress Free radical</infon><infon key=\"license\">This is an open access article under the CC-BY-NC-ND license (http://creativecommons.org/licenses/by-nc-nd/4.0/).</infon><infon key=\"lpage\">916</infon><infon key=\"name_0\">surname:Iyaswamy;given-names:Ashok</infon><infon key=\"name_1\">surname:Kammella;given-names:Ananth Kumar</infon><infon key=\"name_2\">surname:Thavasimuthu;given-names:Citarasu</infon><infon key=\"name_3\">surname:Wankupar;given-names:Wankhar</infon><infon key=\"name_4\">surname:Dapkupar;given-names:Wankhar</infon><infon key=\"name_5\">surname:Shanmugam;given-names:Sambantham</infon><infon key=\"name_6\">surname:Rajan;given-names:Ravindran</infon><infon key=\"name_7\">surname:Rathinasamy;given-names:Sheeladevi</infon><infon key=\"section_type\">TITLE</infon><infon key=\"type\">front</infon><infon key=\"volume\">26</infon><infon key=\"year\">2017</infon><offset>0</offset><text>Oxidative stress evoked damages leading to attenuated memory and inhibition of NMDAR-CaMKII-ERK/CREB signalling on consumption of aspartame in rat model</text></passage><passage><infon key=\"section_type\">ABSTRACT</infon><infon key=\"type\">abstract</infon><offset>157</offset><text>Many controversial reports are available on the use of aspartame as it releases methanol as one of its metabolite during metabolism. The present study proposes to investigate whether long term (90 days) aspartame (40 mg/kg b.wt) administration could induce oxidative stress and alter the memory in Wistar strain male albino rats. To mimic the human methanol metabolism, methotrexate (MTX)-treated rats were included as a model to study the effects of aspartame. Wistar strain albino rats were administered with aspartame (40 mg/kg b.wt) orally and studied along with controls and MTX-treated controls. Aspartame interfered in the body weight and corticosterone levels in the rats. A marked increase in the mRNA and protein expression of neuronal nitric oxide synthase (nNOS) and induced nitric oxide synthase (iNOS) which resulted in the increased nitric oxide radical's level indicating that aspartame is a stressor. These reactive nitrogen species could be responsible for the altered cell membrane integrity and even cause death of neurons by necrosis or apoptosis. The animals showed a marked decrease in learning, spatial working and spatial recognition memory deficit in the Morris water maze and Y-maze performance task which could have resulted due to reduced hippocampal acetylcholine esterase (AChE) activity. The animal brain homogenate also revealed the decrease in the phosphorylation of NMDAR1-CaMKII-ERK/CREB signalling pathway, which well documents the inhibition of phosphorylation leads to the excitotoxicity of the neurons and memory decline. This effect may be due to methanol which may also activate the NOS levels, microglia and astrocytes, inducing neurodegeneration in brain. Neuronal shrinkage of hippocampal layer due to degeneration of pyramidal cells revealed the abnormal neuronal morphology of pyramidal cell layers in the aspartame treated animals. These findings demonstrate that aspartame metabolites could be a contributing factor for the development of oxidative stress in the brain.</text></passage><passage><infon key=\"section_type\">INTRO</infon><infon key=\"type\">title_1</infon><offset>2182</offset><text>1. Introduction</text></passage><passage><infon key=\"section_type\">INTRO</infon><infon key=\"type\">paragraph</infon><offset>2198</offset><text>Aspartame, a low-calorie sweetener obtained by the synthesis from two amino acids, l-phenylalanine and l-aspartic acid, is widely used in beverages, soft drinks, food and dietary products. Aspartame's attractiveness as a sweetener is 180 times sweeter than sugar in typical concentrations without the high energy of sugar. Aspartame on ingestion is metabolised to 40% of aspartic acid, 50% is phenylalanine and 10% is methanol. Being that 10% of the metabolized by product, methanol is oxidized in the liver to formaldehyde which is further oxidized to formic acid in many tissues. In our previous finding it has been reported that aspartame appears to be a potential stressor as aspartame could elevate the corticosterone level and such elevated level may be the reason for the deviations in the free-radical-scavenging system. Further, this elevation of corticosteroid may be documented by the methanol that has been released from aspartame during its metabolism. The severity of clinical outcomes in methanol intoxication interrelated better with formate levels. Also, methanol intoxication is concomitant with mitochondrial damage, augmented microsomal proliferation resulting in increased production of oxygen radicals and altered free-radical-scavenging system in the brain (40 mg/kg b.wt).</text></passage><passage><infon key=\"section_type\">INTRO</infon><infon key=\"type\">paragraph</infon><offset>3497</offset><text>All isoforms of nitric oxide synthase (NOS) can be expressed in the brain; endothelial nitric oxide synthase (eNOS) in endothelial cells and neurons, neuronal nitric oxide synthase (nNOS) in neurons (presumably in only 1-2% of the total neuronal population in many brain regions) and induced nitric oxide synthase (iNOS) in activated microglia and astrocytes.When overproduced, much of the newly synthesized nitric oxide (NO) will be converted into peroxynitrite, is an enormously potent free radical. Nitric oxide is also likely to be involved in axonal and neuronal injury in demyelinating conditions.</text></passage><passage><infon key=\"section_type\">INTRO</infon><infon key=\"type\">paragraph</infon><offset>4103</offset><text>Ionic involvement has been suggested to be involved in memory formation and the Na+/K+-ATPase enzyme is decisive for sustaining ionic gradients in neurons and tissues. Cholinergic transmission is mainly terminated by Acetylcholine (Ach) hydrolysis by the enzyme AChE, this enzyme is widely expressed in tissues that receive cholinergic innervations, in neurons and muscle cells. Hippocampus plays a crucial role in memory and damage to it will cause memory dysfunction and behavioural disturbances. Important molecular feature of learning and memory depends on the NMDAR-CaMKII-ERK/CREB signalling pathway. The synaptic plasticity in the brain is maintained by this signalling pathway. When there is inhibition in this signalling pathway it leads to the memory decline and neurodegeneration. Reduced levels of p-NMDAR1, p-CaMKII, and p-CREB may inhibit the normal function of neuron and it may result in neurodegeneration and memory decline. This may also activate the microglia and astrocytes to clear the degenerated neurons. Heat shock protein70 (Hsp70) is the most important proteins induced by stress in the brain, and its role in neuroprotection has been demonstrated both in vivo and in vitro.</text></passage><passage><infon key=\"section_type\">INTRO</infon><infon key=\"type\">paragraph</infon><offset>5308</offset><text>Normally formate is metabolized twice as fast in the rat as in the monkey. The rodents do not develop metabolic acidosis during methanol poisoning, due to their high liver folate content and to create similar results in human beings, merely folate deficient rodents are required to accumulate formate in order to develop acidosis. Hence, in this study to mimic the human situation, a folate deficiency status is induced. Over the years researchers have expressed concern over methanol toxicity resulting from aspartame consumption and the chemical has been tested in both nonhuman primates and humans. The central nervous system (CNS) is extremely sensitive to free radical damage due to high polyunsaturated fatty acid content in cell membrane and relatively low total antioxidant capacity which are accountable to neuronal damage. Therefore, in this present study, an attempt has been made to investigate expression of aspartame in NMDAR-CaMKII-ERK/CREB signalling pathway in the central nervous system and the role of NMDAR-CaMKII-ERK/CREB signalling pathway and its inhibition in memory loss.</text></passage><passage><infon key=\"section_type\">METHODS</infon><infon key=\"type\">title_1</infon><offset>6413</offset><text>2. Materials and methods</text></passage><passage><infon key=\"section_type\">METHODS</infon><infon key=\"type\">title_2</infon><offset>6438</offset><text>2.1. Animals</text></passage><passage><infon key=\"section_type\">METHODS</infon><infon key=\"type\">paragraph</infon><offset>6451</offset><text>Wistar strain male albino rats (200-220 g) were maintained under standard laboratory conditions with water and food. For the folate-deficient group, folate-deficient diet was provided for 45 days prior to the experiment and methotrexate (MTX) was administered for a week before the experiment. The animals were handled according to the principles of laboratory care framed by the committee for the purpose of Control and Supervision of Experiments on Animals (CPCSEA), Government of India. Prior to the experimentation, proper approval was obtained from the Institutional Animal Ethical Committee (No: 01/032/2010/Aug-11).</text></passage><passage><infon key=\"section_type\">METHODS</infon><infon key=\"type\">title_2</infon><offset>7076</offset><text>2.2. Chemicals</text></passage><passage><infon key=\"section_type\">METHODS</infon><infon key=\"type\">paragraph</infon><offset>7091</offset><text>Aspartame and Methotrexate were purchased from Sigma chemical company, St. Louis, MO, USA. Nitric oxide assay kit was purchased from Biovision, USA. Taq-Polymerase, DNTPs from Genet Bio, China, Reverse Transcriptase enzyme kit from Thermo scientific, USA, primary antibody was purchased from Sigma and Cell signalling, USA, secondary antibody was purchased from Merck and other molecular grade chemicals were from Merck Bangalore, India. All other chemicals were of analytical grade obtained from the Sisco research laboratory, Bombay, India.</text></passage><passage><infon key=\"section_type\">METHODS</infon><infon key=\"type\">title_2</infon><offset>7634</offset><text>2.3. Experimental design</text></passage><passage><infon key=\"section_type\">METHODS</infon><infon key=\"type\">paragraph</infon><offset>7659</offset><text>The European Food Safety Authority recently confirmed its daily acceptable intake (ADI) for aspartame as 40 mg/kg b.wt./day. In order to confine within the human permitted exposure limit, this dose was selected. Aspartame (Sigma Aldrich, USA) was administered orally; (40 mg/kg b.wt) mixed 1:1 with 0.9% saline and this dosage based on the food and drug administration (FDA) approved ADI limit.</text></passage><passage><infon key=\"section_type\">METHODS</infon><infon key=\"type\">title_2</infon><offset>8054</offset><text>2.4. Groups</text></passage><passage><infon key=\"section_type\">METHODS</infon><infon key=\"type\">paragraph</infon><offset>8066</offset><text>The rats were randomly divided into three groups, namely, saline control, folate deficient (MTX treated) control, and MTX-treated (folate deficient) + aspartame administered groups. Each group consisted of six animals. Methotrexate (MTX) in sterile saline was administered (0.2 mg/kg/day) subcutaneously for 7 days to folate-deficient groups. After 7 days of administration with MTX, folate deficiency was confirmed by estimating the urinary excretion of for-maminoglutamic acid (FIGLU). From the eighth day, the MTX-treated (folate deficient) (third) groups were administered orally with aspartame for 90 days, whereas the saline control or vehicle control and (folate deficient) MTX treated control groups received equivalent volumes of saline as an oral dose and all animals were handled similarly.</text></passage><passage><infon key=\"section_type\">METHODS</infon><infon key=\"type\">title_2</infon><offset>8868</offset><text>2.5. Brain collection and dissection</text></passage><passage><infon key=\"section_type\">METHODS</infon><infon key=\"type\">paragraph</infon><offset>8905</offset><text>The animals were sacrificed using higher dose of long acting pentothal sodium (100 mg/kg.b.wt). Collection of the blood samples and isolation of brain was performed between 8 and 10 a.m. to avoid circadian rhythm-induced changes. The brain was immediately removed and washed with ice-cold phosphate buffered saline (PBS). Further dissection was made on ice cold glass plate. The discrete regions of the brain (cerebral cortex, cerebellum, midbrain, Pons medulla, hippocampus and hypothalamus) were dissected according the method given in our previous publication. The homogenate (10% w/v) of the individual regions was prepared in a Teflon-glass tissue homogenizer, using ice-cold PBS (100 mM, pH 7.4) buffer and centrifuged separately in refrigerated centrifuge 4  C at 3000 rpm for 15 min. The supernatant was used for analyzing the biochemical estimations in this study. The body weight was recorded every week to monitor the influence of the aspartame treatment period.</text></passage><passage><infon key=\"section_type\">METHODS</infon><infon key=\"type\">title_2</infon><offset>9880</offset><text>2.6. Estimation of plasma corticosterone and LD50 of aspartame by Karber's method</text></passage><passage><infon key=\"section_type\">METHODS</infon><infon key=\"type\">paragraph</infon><offset>9964</offset><text>Plasma corticosterone was estimated by the method given by Clark. The method is based on the oxidation of corticosteroids with iron (III) in acidic medium and subsequent complex of iron (II) with potassium hexacynoferrate (III). The method offers the advantage of sensitivity and stability. The absorbance was measured at 780 nm in spectrophotometer (Shimadzu UV-1800). The level of corticosterone in the plasma was expressed as mumolar (muM). The calculation of LD50 by the method of Karber was used for the determination of LD50 of aspartame in Wistar albino rats. LD50 = LD100 -  (a x b)n. n = number of rats in a group. a = difference between two successive doses of aspartame administered. b = average number of dead animals in two successive aspartame doses. LD100 = Lethal dose causing the 100% death of all rats.</text></passage><passage><infon key=\"section_type\">METHODS</infon><infon key=\"type\">title_2</infon><offset>10790</offset><text>2.7. Estimation of membrane bound enzymes</text></passage><passage><infon key=\"section_type\">METHODS</infon><infon key=\"type\">paragraph</infon><offset>10832</offset><text>The activity of Na+/K+ ATPase (ATP: Phosphohydrolase - EC. 3.6.1.3.) in the tissue was estimated by the method of Bernabe et al.. The activity of Ca2+-ATPase (ATP: Phosphohydrolase - EC. 3.6.1.3.) in the tissues was estimated as described by Hjerten and Pan and the activity of Mg2+-ATPase (ATP: Phosphohydrolase - EC. 3.6.1.3.) in the tissues was estimated by the methods of Ohnishi et al..</text></passage><passage><infon key=\"section_type\">METHODS</infon><infon key=\"type\">title_2</infon><offset>11230</offset><text>2.8. Estimation of acetylcholine esterase (AChE)</text></passage><passage><infon key=\"section_type\">METHODS</infon><infon key=\"type\">paragraph</infon><offset>11279</offset><text>The activity of Acetylcholine esterase in the tissues was estimated by the method of Ellman et al.. The Principle of the method being the measurement of the rate of production of thiocholine as acetylthiocholine is hydrolysed. This is accomplished by the continuous reaction of the thiol with 5:5-dithiobis-2-nitrobenzoate iron (I) 5 to produce the yellow anion of 5-thio-2-nitro-benzoic acid (II). The rate of colour production is measured at 412 nm.</text></passage><passage><infon key=\"section_type\">METHODS</infon><infon key=\"type\">title_2</infon><offset>11735</offset><text>2.9. Determination of nitrite using nitric oxide assay kit</text></passage><passage><infon key=\"section_type\">METHODS</infon><infon key=\"type\">paragraph</infon><offset>11794</offset><text>To investigate nitric oxide formation it is essential to measure nitrite (NO2) which is one of two primary stable and nonvolatile breakdown products of NO. This assay relies on a diazotization reaction that was originally described by Griess. The original reaction has been described as a Griess reagent system is based on the chemical reaction, which uses sulfanilamide and N-1-napthylethylenediamine dihydrochloride (NED) under acidic (phosphoric acid) conditions.</text></passage><passage><infon key=\"section_type\">METHODS</infon><infon key=\"type\">title_2</infon><offset>12261</offset><text>2.10. Measurement of memory in Y maze and Morris water maze</text></passage><passage><infon key=\"section_type\">METHODS</infon><infon key=\"type\">paragraph</infon><offset>12321</offset><text>The Y maze measures spatial working and recognition memory by making use of a rodent's natural exploratory instincts. In Y-maze the three arms were marked as A, B, C and &quot;C&quot; is marked as the novel arm (blocked arm) to test the spatial working memory. The first session measures spatial working memory in the rat by scoring the number of alternations the rat does in Y-maze when the animal visits all three arms without going into the same arm twice in a row. If working memory is intact, the rat should remember the first two arms and be more inquisitive about the third arm (originally blocked arm). In the second session open the blocked arm and the rats were left 5 min of time to evaluate the spatial recognition memory. A rat with a normal memory should spend more time exploring the &quot;new&quot; unblocked arm. The alternations for the first 2 min were calculated. The overall percentage for 5 min and the percentage for the first 2 min were also calculated for spatial working and recognition memory. The Morris water maze (MWM) test consists of a circular pool made of non-toxic white metal, filled with water and maintained at 21 +- 1  C. A platform was placed 1 cm below the water surface at a fixed position. Animals were acclimatized and trained. Animals were monitored using an automated tracking system (Ethovision software). Visible platform trials were performed to evaluate sensorimotor and/or motivational deficits that could influence performance during the learning task. Hidden-platform training consisted of 6 days with each day comprised four trials with a 60min interval. The platform location remained constant in the hidden platform trials, and the entry point was changed randomly in all days. Memory retention was assessed by a probe trial after the 6 days hidden-platform training. The time spent in the target quadrant without the platform was taken to measure level of memory retention in the probe trial.</text></passage><passage><infon key=\"section_type\">METHODS</infon><infon key=\"type\">title_2</infon><offset>14262</offset><text>2.11. Isolation of total RNA &amp; reverse transcription-polymerase chain reaction (RT-PCR)</text></passage><passage><infon key=\"section_type\">METHODS</infon><infon key=\"type\">paragraph</infon><offset>14350</offset><text>Total RNA was isolated from cells using Trizol reagent following the method given in our previous publication. The total RNA obtained was free from protein and DNA contamination. The reverse transcription step was performed by using the reverse transcriptase (RT) enzyme kit. Each 20 muL reaction mixture contained 5 muL OligodT (10 muM), 1 muL dNTP (10 muM), 4 muL First Strand buffer (5x), 0.2 muL super script III reverse transcriptase (200 U/muL) varied quantity of RNA template (dependent on RNA concentration) and RNase free water to make up the volume. Thermal cycling conditions for the first strand reaction consisted of 25  C for 5 min, 50  C for 45 min, 70  C for 15 min and finally maintained at 4  C for 5 min. PCR amplification was performed using Taq DNA Polymerase. Each 20 muL of sample contained 10 muL Master mix (2 muM), 1 muL forward primer, 1 muL reverse primer (Table 3) for both gene of interest and internal control consecutively, 2 muL RT sample, 4 muL sterile water. The mixture was kept at thermocycler and amplified for 35 cycles. Each thermocycling consisted of 94  C for 30 s, varied annealing temperature for each gene of interest for 30 s, 72  C for 30 s. beta-actin was co amplified with c-fos, iNOS, nNOS, Hsp70 genes using the same procedures. The sense and antisense primer for the study is tabulated in table. Ten microliters of each PCR product was analyzed by gel electrophoresis on 2% agarose gel. Agarose gel electrophoresis is an effective method for the identification of purified DNA molecules. Amplified product was analyzed by agarose gel electrophoresis with ethidium bromide staining. Then the gel containing cDNA was visualized with the help of the fluorescent imager (Bio-Rad, USA). The Band intensity was quantified by Quantity One Software. The band intensification for each enzyme mRNA was normalized with that of the internal control beta-actin using Quantity One Software.</text></passage><passage><infon key=\"section_type\">METHODS</infon><infon key=\"type\">title_2</infon><offset>16282</offset><text>2.12. Histopathology</text></passage><passage><infon key=\"section_type\">METHODS</infon><infon key=\"type\">paragraph</infon><offset>16303</offset><text>Animals were deeply anesthetized with ketamine hydrochloride. Rats were then perfused transcardially with phosphate-buffered saline, followed by buffered 10% formalin. The brain was removed, and preserved in formalin until processed for histology. Then kept on running water to remove formalin pigments and dehydrated with ascending grades of alcohol. After impregnation with paraffin wax, the paraffin blocks were made. They were processed and sections were cut with 10 nm in thickness using &quot;Spencer Lens, rotatory microtome (No 820, New York, USA) and then stained with Cresyl fast violet as follows for the brain.</text></passage><passage><infon key=\"section_type\">METHODS</infon><infon key=\"type\">title_2</infon><offset>16923</offset><text>2.13. Immunohistochemical analysis</text></passage><passage><infon key=\"section_type\">METHODS</infon><infon key=\"type\">paragraph</infon><offset>16958</offset><text>Immunohistochemical analysis was carried out using the DAB universal staining kit (Merck Genie, Bengaluru, India). The sections were deparaffinized in xylene and dehydrated in ethanol. After washing with PBS, slides were incubated with 3% H2O2 in at room temperature for 15 min to quench endogenous peroxidase activity. After antigen retrieval (15 min of autoclaving in 10 mM citrate buffer, pH 6.0), the slides were incubated with blocking solution (10% normal goat serum) for 5 min at room temperature. Then, the sections were incubated overnight with primary antibody. Subsequently, the sections were incubated with horseradish peroxidase (HRP) secondary link antibody for 30 min at room temperature, washed with PBS. Then, the sections were treated with 3,3'-diaminobenzidine (DAB) chromogen for 15 min. Finally, the sections were washed with deionised water, counter stained with haematoxylin and mounted. Photographs were taken using Nikon microscope (Nikon 1.25X Japan DS-Fi1).</text></passage><passage><infon key=\"section_type\">METHODS</infon><infon key=\"type\">title_2</infon><offset>17945</offset><text>2.14. Western blot</text></passage><passage><infon key=\"section_type\">METHODS</infon><infon key=\"type\">paragraph</infon><offset>17964</offset><text>The brains were washed with ice-cold PBS and powdered in liquid nitrogen, was suspended in the cytobuster buffer (Invitrogen) was used to detect the NMDAR, CaMKII, ERK, CREB, PSD95 and Synaptophysin. Protein concentrations of brain lysates were assayed using BCA reagent (BioRad, USA). Equal amounts of protein were subjected to 8% and 10% SDS polyacrylamide gels at 70 V, and were transferred onto polyvinylidene fluoride (PVDF) membranes (Merck Millipore) at 300 mA for 90 min, and then blocked with 5% fat-free milk or BSA for 1 h at room temperature. The membranes were then incubated overnight with the primary antibodies at 4  C. Membranes were washed and probed with secondary antibody: goat anti-rat IgG, and goat anti-rabbit IgG (Thermo-scientific). Protein signals were visualized using an ECL chemiluminescent HRP substrate detection system. The band intensity was quantified by ImageJ and Quantity One software (Bio-Rad, USA). The membranes were stripped and reprobed for B-actin (Sigma) (1:10,000) as an internal control.</text></passage><passage><infon key=\"section_type\">METHODS</infon><infon key=\"type\">title_2</infon><offset>19000</offset><text>2.15. Statistical analysis</text></passage><passage><infon key=\"section_type\">METHODS</infon><infon key=\"type\">paragraph</infon><offset>19027</offset><text>Statistical analysis was carried out using the SPSS statistical package version 18.0. The results are expressed as Mean +- SD and the data were analyzed by the one-way analysis of variance (ANOVA) followed by Turkey's multiple comparison tests when there is a significant 'F' test ratio. The level of significance was fixed at p &lt;= 0.05.</text></passage><passage><infon key=\"section_type\">RESULTS</infon><infon key=\"type\">title_1</infon><offset>19372</offset><text>3. Results and discussion</text></passage><passage><infon key=\"section_type\">RESULTS</infon><infon key=\"type\">paragraph</infon><offset>19398</offset><text>The present study support the fatal effect of aspartame when consumed repeatedly for a prolonged period and it may be probably due to the toxic metabolite methanol. It has been reported that following aspartame consumption, the concentrations of its metabolites are increased in the blood stream. A small amount of aspartame significantly increases the plasma methanol level. Further, it was reported that not only the metabolites of methanol, but methanol per se as well, is toxic to the brain. In addition, this increase in the blood methanol level was associated with a marked increase in the free radical generation in brain regions of aspartame treated animals. The cells can be injured or killed when the ROS generation overwhelms the cellular antioxidant capacity indicate its vulnerability to free radicals. In this study the LD50 of aspartame was analyzed by karber's method, the LD50 of aspartame (Table 2) in rats is 10.95 g/kg of body weight. Aspartame interfered in the body weight of the treated animals (Fig. 1B) when compared to the control rats throughout the study duration.</text></passage><passage><infon key=\"section_type\">RESULTS</infon><infon key=\"type\">title_2</infon><offset>20493</offset><text>3.1. Aspartame induces the free radical generation</text></passage><passage><infon key=\"section_type\">RESULTS</infon><infon key=\"type\">paragraph</infon><offset>20544</offset><text>The nitric oxide ( NO), a nitrogen free radical, is produced by a number of different cell types. The elevation of nitric oxide (Fig. 1C) observed in this study again support the generation of free radical by aspartame. However, this increase in nitric oxide after aspartame consumption could not be overlooked because according to McCord, the prolonged exposure to free radicals, even at a low concentration, may result in the damage of biologically important molecules. In this finding the aspartame treated animals, did showed an elevation of nitric oxide level but not the nitrite and this is supported by the western blot study in which nNOS protein expression was enhanced along with increase mRNA expression of nitric oxide synthesizing enzyme nNOS and iNOS (Figs. 3 and 4). Studies demonstrated that NO produced by iNOS causes damage to brain cells, a process which has been demonstrated in many neurodegenerative diseases and exert numerous toxic effects in a wide variety of mammalian cell targets.</text></passage><passage><infon key=\"section_type\">RESULTS</infon><infon key=\"type\">title_2</infon><offset>21555</offset><text>3.2. Aspartame induces stress and decreases membrane homeostasis</text></passage><passage><infon key=\"section_type\">RESULTS</infon><infon key=\"type\">paragraph</infon><offset>21620</offset><text>In the present study, there was an increased plasma corticosterone level (Fig. 1A) in the aspartame treated animals and it is relevant to point out that, in our earlier report also there was a marked increase in the corticosterone level in the plasma on (75 mg/kg) aspartame administered rats, indicating that the dietary sweetener aspartame could act as a chemical stressor. Decrease in membrane bound ATPases activity (Table 1) in this study may be due to free radical generation and high NO level, which is supported by few studies. Ashok et al. reported that methanol induced increased production of free radicals and increased oxidative damage to distinct brain regions, retina and optic nerve has also been reported. McIntosh et al. reported a decreased activity of the antioxidant enzymes in the brain of rats treated with glucocorticoids. Indicating that elevated free radical generation overwhelming the antioxidant defence system. High levels of NO have been associated with membrane lipid peroxidation which suggest that the aspartame mediated increase in free radical may be the cause behind the altered membrane integrity observed in this study which eventually lead to decrease in membrane bound ATPases activity.</text></passage><passage><infon key=\"section_type\">RESULTS</infon><infon key=\"type\">paragraph</infon><offset>22848</offset><text>Na+/K+-ATPase, the enzyme that maintains Na+ and K+ gradients across the plasma membrane, was reported to be inhibited by ROS in the brain which strengthen our argument. According to Polizzi et al., the Na+/K+ ATPase is very sensitive to the plasma membrane structure changes and therefore measuring its activity represents a valuable indicator of the early and late stages of tissue injury. Further, the decrease in the activity of these membrane bound enzymes could not be ignored as Na+/K+-ATPase is responsible for the generation of the membrane potential through the active transport of sodium and potassium ions in the neurons in CNS necessary and to maintain neuronal excitability. Moreover, the role of Mg2+-ATPase is to maintain high brain intracellular Mg2+ which can control rates of protein synthesis and cell growth. Na+/K+, Mg2+ and Ca2+ ATPase in the plasma membrane also keeps the intracellular sodium low but intracellular magnesium and potassium high when compared with the levels in extracellular fluids. Hence, the increase in nitric oxide, free radical and corticosteroids in this study after aspartame consumption may be a contributing factor for the alteration observed in the membrane bound enzymes and finally altering the neuronal integrity.</text></passage><passage><infon key=\"section_type\">RESULTS</infon><infon key=\"type\">title_2</infon><offset>24116</offset><text>3.3. Aspartame attenuates the learning and memory</text></passage><passage><infon key=\"section_type\">RESULTS</infon><infon key=\"type\">paragraph</infon><offset>24166</offset><text>The Y-maze task as used in the present study, with one novel arm throughout the experiment, is so-called &quot;reference memory task.&quot; This type of test is considered to be hippocampus dependent. Various reports have shown that bilateral cytotoxic lesions of the hippocampal formation result in severe impairments in the Y-maze reference memory task and NO also contributes to hippocampal dependent learning and memory tasks including spatial memory. AChE is a key enzyme of the muscarinic cholinergic system, involved in learning and memory. However, NO formed as a result of iNOS up regulation during hypoxia has been found to interrupt the memory process by inhibiting acetylcholinergic activity. A marked decrease in the AChE activity (Fig. 1D) in the aspartame treated animals of present study lends a support to alteration in the learning and memory process (Fig. 2A-E) of these aspartame fed animals in both Morris water maze and Y maze task. It is also in agreement with the Irene et al., who reported that aspartame metabolites inhibit the activity of hippocampal AChE. Rico et al., reported that methanol exposure remarkably decreased zebrafish brain AChE activity, suggesting a neurodegenerative event promoted possibly by methanol and suggest that aspartame induced alterations may be mediated by methanol by affecting the hippocampus functioning. Moreover our previous study in 75 mg/kg aspartame consumption also showed alteration in the locomotor, anxiety and emotional behaviour of the treated rats. Thus supporting our study that aspartame attenuates learning, memory (Fig. 2A-E) and declines the memory retention in the Morris water maze task.</text></passage><passage><infon key=\"section_type\">RESULTS</infon><infon key=\"type\">title_2</infon><offset>25831</offset><text>3.4. Aspartame inhibits the NMDAR-CaMKII-ERK/CREB signalling pathway</text></passage><passage><infon key=\"section_type\">RESULTS</infon><infon key=\"type\">paragraph</infon><offset>25904</offset><text>As aspartame attenuated memory, we were curious in finding out the signalling pathway involved in memory decline and neurodegeneration. We further investigated the NMDAR-CaMKII-ERK/CREB signalling pathway. This is an important molecular feature of learning and memory in the neuroplasticity of brain which involves in phosphorylation of receptor and calcium influx leading to activation of several proteins by phosphorylating the proteins. In our study we found that the phosphorylation of NMDAR1, CamKII, ERK and CREB was inhibited (Fig. 4C) which leads to the memory decline and neuronal damage. As this signalling pathway is inhibited the protein levels of presynaptic and postsynaptic markers namely PSD95, Synaptophysin (Fig. 4B) were decreased. The NOS expression levels activated the microglia and astrocytes, there activated protein expression was illustrated by IHC of Iba1 and GFAP (Fig. 4A). These activated proteins illustrates that neuronal degeneration may also have been through gliosis, which clearly confirms that neurodegeneration may be due to inhibition of NMDAR-CaMKII-ERK/CREB signalling pathway activation of glial cells which might have initiated the inhibitory synaptic excitotoxicity in neurons. When methanol from aspartame inhibits the phosphorylation of NMDAR1 it leads to the decrease in the Ca2+ influx, which inhibits the phosphorylation of Calmodulin kinase II further inhibits the phosphorylation of ERK and CREB leading to decrease in memory proteins and other neuronal proteins which damages the neurons and accelerating the generation of free radicals by increasing the expression levels of NOS. Interestingly we also found that NOS also activated the astrocytes and glial cells as neurons were at the verge of cell death. Further the neuroprotective effects of NO may be overcome by the nitrosative stress which leads to prolonged, excessive, and persistent production of NO radical, ultimately leading to neurotoxicity and cell death. Aspartame administration attenuated memory by inhibiting NMDAR-CaMKII-ERK/CREB signalling pathway (Fig. 6), the observed changes may owe to the methanol from aspartame which inhibited the phosphorylation of NMDAR-CaMKII-ERK/CREB signalling pathway.</text></passage><passage><infon key=\"section_type\">RESULTS</infon><infon key=\"type\">title_2</infon><offset>28143</offset><text>3.5. Aspartame is a chemical stressor and causes oxidative damage to brain</text></passage><passage><infon key=\"section_type\">RESULTS</infon><infon key=\"type\">paragraph</infon><offset>28218</offset><text>In the present study a significant increase mRNA expression of c-Fos and increase in the expression of Hsp70 in aspartame treated animals (Figs. 3 and 5B) when compared to controls. Gil et al. reported that Fos protein plays an important role in the cell growth, differentiation and in the recovery from reversible injuries. In various studies c-Fos gene expression has been associated with delayed neuronal cell death, and it has been shown that prolonged c-Fos induction precedes neuronal death. Hsp70 gene encodes a major stress inducible heat shock protein, which plays an important role in protecting cells from deleterious stresses. It was reported that gene expression leading to the synthesis of HSP exerts cytoprotective functions. Moreover, it has been reported that the heat shock system play a vital role in the protective mechanism of brain and the Hsp70 is considered as a critical determinant of brain's stress tolerance. Therefore, the possible cause for the aspartame consumption induced increased expression of these Hsp70 and c-Fos may be to prevent cellular injury, probably caused by the elevated levels of free radicals.</text></passage><passage><infon key=\"section_type\">RESULTS</infon><infon key=\"type\">paragraph</infon><offset>29363</offset><text>The aspartame treated animals showed a neuronal shrinkage of hippocampal layer (HL) due to degeneration of hippocampal neurons (Fig. 5A), revealed the abnormal neuronal morphology of pyramidal cell layers of Cornu Ammonis and the morphology of pyramidal cell layers appeared disorganized. Higher levels of ROS in the GSH-depleted brain may produce tissue damage and ROS generation may cause deleterious effect to other organs namely liver, testis and kidney. This is in agreement with previous reports which suggest that aspartame administration at a dose level of 40 mg/kg may induce an oxidative stress. In the present study aspartame treated animals showed a marked significant increase in the stress specific c-fos, Hsp70 and nNOS protein expression (Figs. 3 and 5B) in brain regions and numerous brown coloured immuno-reaction positive cells were observed. These results indicate that the stress reaction in the brain tissue may have resulted due to methanol from aspartame. The observed changes may be due to the methanol or its metabolite. Since aspartame is consumed more than the recommended dosage by common people, it is essential to do more work on aspartame and to create awareness regarding the usage of this artificial sweetener.</text></passage><passage><infon key=\"section_type\">CONCL</infon><infon key=\"type\">title_1</infon><offset>30608</offset><text>4. Conclusion</text></passage><passage><infon key=\"section_type\">CONCL</infon><infon key=\"type\">paragraph</infon><offset>30622</offset><text>The observed results support the fatal effect of aspartame when consumed repeatedly and aspartame administration attenuated memory by inhibiting NMDAR-CaMKII-ERK/CREB signalling pathway and decreased the membrane bound ATPase enzymes activity, Ach E, increased the NO radical generation and iNOS expression. However the cellular damage was much prevented by c-fos and Hsp70 expression in the brain. The observed changes may owe to the methanol from aspartame which inhibited the phosphorylation of NMDAR-CaMKII-ERK/CREB signalling pathway (Fig. 6). Since aspartame consumption is on the rise among common people, it is essential to create awareness regarding the usage of this artificial sweetener. Further studies are required to evaluate the effect of aspartame in the future.</text></passage><passage><infon key=\"section_type\">CONCL</infon><infon key=\"type\">footnote</infon><offset>31409</offset><text> Conflicts of interest </text></passage><passage><infon key=\"section_type\">CONCL</infon><infon key=\"type\">footnote</infon><offset>31433</offset><text>The authors report no conflicts of interest. The authors alone are responsible for the content and writing of this paper. The authors declare that they have no competing interests.</text></passage><passage><infon key=\"section_type\">REF</infon><infon key=\"type\">title</infon><offset>31614</offset><text>REFERENCES</text></passage><passage><infon key=\"name_0\">surname:Newsome;given-names:RL</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">Sweeteners: nutritive and non-nutritive</infon><infon key=\"type\">ref</infon><infon key=\"year\">1986</infon><offset>31625</offset></passage><passage><infon key=\"fpage\">679</infon><infon key=\"lpage\">91</infon><infon key=\"name_0\">surname:Iyaswamy;given-names:Ashok</infon><infon key=\"name_1\">surname:Rathinasamy;given-names:Sheeladevi</infon><infon key=\"pub-id_pmid\">28911484</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">J Food Drug Anal</infon><infon key=\"type\">ref</infon><infon key=\"volume\">23</infon><infon key=\"year\">2015</infon><offset>31626</offset><text>Oxidant stress evoked damages in rat hepatocyte leading to triggered NOS levels on long term consumption of aspartame</text></passage><passage><infon key=\"fpage\">390</infon><infon key=\"issue\">5</infon><infon key=\"lpage\">6</infon><infon key=\"name_0\">surname:Ashok;given-names:I</infon><infon key=\"name_1\">surname:Sheeladevi;given-names:R</infon><infon key=\"name_2\">surname:Dapkupar;given-names:W</infon><infon key=\"pub-id_pmid\">26445572</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">J Biomed Res</infon><infon key=\"type\">ref</infon><infon key=\"volume\">29</infon><infon key=\"year\">2015</infon><offset>31744</offset><text>Acute effect of aspartame (artificial sweetener) induced oxidative stress in the brain regions of Wistar albino rats</text></passage><passage><infon key=\"fpage\">76</infon><infon key=\"lpage\">85</infon><infon key=\"name_0\">surname:Ashok;given-names:I</infon><infon key=\"name_1\">surname:Dapkupar;given-names:W</infon><infon key=\"name_2\">surname:Wankupar;given-names:W</infon><infon key=\"name_3\">surname:Sheeladevi;given-names:R</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">J Nutr Intermed Metab</infon><infon key=\"type\">ref</infon><infon key=\"volume\">2</infon><infon key=\"year\">2015</infon><offset>31861</offset><text>Neurobehavioral changes and activation of neurodegenerative apoptosis on long-term consumption of aspartame in the rat brain</text></passage><passage><infon key=\"fpage\">1</infon><infon key=\"lpage\">7</infon><infon key=\"name_0\">surname:Ashok;given-names:I</infon><infon key=\"name_1\">surname:Poornima;given-names:PS</infon><infon key=\"name_2\">surname:Wankhar;given-names:D</infon><infon key=\"name_3\">surname:Ravindran;given-names:R</infon><infon key=\"name_4\">surname:Sheeladevi;given-names:R</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">Int J Impot Res</infon><infon key=\"type\">ref</infon><infon key=\"year\">2017</infon><offset>31986</offset><text>Oxidative stress evoked damages on rat sperm and attenuated antioxidant status on consumption of aspartame</text></passage><passage><infon key=\"fpage\">2000</infon><infon key=\"lpage\">3</infon><infon key=\"name_0\">surname:Osterloh;given-names:JD</infon><infon key=\"name_1\">surname:Pond;given-names:SM</infon><infon key=\"name_2\">surname:Grady;given-names:S</infon><infon key=\"name_3\">surname:Becker;given-names:CE</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">Ann Intern Med</infon><infon key=\"type\">ref</infon><infon key=\"volume\">104</infon><infon key=\"year\">1986</infon><offset>32093</offset><text>Serum formate concentrations in methanol intoxication as a criterion for hemodialysis</text></passage><passage><infon key=\"fpage\">227</infon><infon key=\"issue\">3</infon><infon key=\"lpage\">36</infon><infon key=\"name_0\">surname:Castro;given-names:GD</infon><infon key=\"name_1\">surname:Costantini;given-names:MH</infon><infon key=\"name_2\">surname:Delgado de Layno;given-names:AM</infon><infon key=\"name_3\">surname:Castro;given-names:A</infon><infon key=\"pub-id_pmid\">11888706</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">Toxicol Lett</infon><infon key=\"type\">ref</infon><infon key=\"volume\">129</infon><infon key=\"year\">2002</infon><offset>32179</offset><text>Rat liver microsomal and nuclear activation of methanol to hydroxyl methyl free radicals</text></passage><passage><infon key=\"fpage\">41</infon><infon key=\"lpage\">50</infon><infon key=\"name_0\">surname:Iyaswamy;given-names:Ashok</infon><infon key=\"name_1\">surname:Wankhar;given-names:Dapkupar</infon><infon key=\"name_2\">surname:Loganathan;given-names:Sundareswaran</infon><infon key=\"name_3\">surname:Shanmugam;given-names:Sambantham</infon><infon key=\"name_4\">surname:Rajan;given-names:Ravindran</infon><infon key=\"name_5\">surname:Rathinasamy;given-names:Sheeladevi</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">J Nutr Intermed Metab</infon><infon key=\"type\">ref</infon><infon key=\"volume\">8</infon><infon key=\"year\">2017</infon><offset>32268</offset><text>Disruption of redox homeostasis in liver function and activation of apoptosis on consumption of aspartame in folate deficient rat model</text></passage><passage><infon key=\"fpage\">593</infon><infon key=\"lpage\">615</infon><infon key=\"name_0\">surname:Alderton;given-names:WK</infon><infon key=\"name_1\">surname:Cooper;given-names:CE</infon><infon key=\"name_2\">surname:Knowles;given-names:RG</infon><infon key=\"pub-id_pmid\">11463332</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">BiochemJ</infon><infon key=\"type\">ref</infon><infon key=\"volume\">357</infon><infon key=\"year\">2001</infon><offset>32404</offset><text>Nitric oxide synthases: structure, function and inhibition</text></passage><passage><infon key=\"fpage\">33</infon><infon key=\"lpage\">40</infon><infon key=\"name_0\">surname:Lipton;given-names:SA</infon><infon key=\"name_1\">surname:Choi;given-names:YB</infon><infon key=\"name_2\">surname:Sucher;given-names:NJ</infon><infon key=\"name_3\">surname:Chen;given-names:HS</infon><infon key=\"pub-id_pmid\">9699006</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">Biofactors</infon><infon key=\"type\">ref</infon><infon key=\"volume\">8</infon><infon key=\"year\">1998</infon><offset>32463</offset><text>Neuroprotective versus neurodestructive effects of NO-related species</text></passage><passage><infon key=\"fpage\">145</infon><infon key=\"lpage\">55</infon><infon key=\"name_0\">surname:Garthwaite;given-names:G</infon><infon key=\"name_1\">surname:Goodwin;given-names:DA</infon><infon key=\"name_2\">surname:Batchelor;given-names:AM</infon><infon key=\"name_3\">surname:Leeming;given-names:K</infon><infon key=\"name_4\">surname:Garthwaite;given-names:J</infon><infon key=\"pub-id_pmid\">11784706</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">Neuroscience</infon><infon key=\"type\">ref</infon><infon key=\"volume\">109</infon><infon key=\"year\">2002</infon><offset>32533</offset><text>Nitric oxide toxicity in CNS white matter: an in vitro study using rat optic nerve</text></passage><passage><infon key=\"fpage\">2582</infon><infon key=\"issue\">6</infon><infon key=\"lpage\">90</infon><infon key=\"name_0\">surname:Conrad;given-names:CD</infon><infon key=\"name_1\">surname:Roy;given-names:EJ</infon><infon key=\"pub-id_pmid\">8501524</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">J Neurosci</infon><infon key=\"type\">ref</infon><infon key=\"volume\">13</infon><infon key=\"year\">1993</infon><offset>32616</offset><text>Selective loss of hippocampal granule cells following adrenalectomy: implications for spatial memory</text></passage><passage><infon key=\"fpage\">31</infon><infon key=\"lpage\">91</infon><infon key=\"name_0\">surname:Massoulie;given-names:J</infon><infon key=\"name_1\">surname:Pezzementi;given-names:L</infon><infon key=\"name_2\">surname:Bon;given-names:S</infon><infon key=\"name_3\">surname:Krejci;given-names:E</infon><infon key=\"name_4\">surname:Vallette;given-names:FM</infon><infon key=\"pub-id_pmid\">8321908</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">Prog Neurobiol</infon><infon key=\"type\">ref</infon><infon key=\"volume\">41</infon><infon key=\"year\">1993</infon><offset>32717</offset><text>Molecular and cellular biology of cholinesterases</text></passage><passage><infon key=\"fpage\">147</infon><infon key=\"lpage\">58</infon><infon key=\"name_0\">surname:Brown;given-names:IR</infon><infon key=\"pub-id_pmid\">17656567</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">Ann N Y Acad Sci</infon><infon key=\"type\">ref</infon><infon key=\"volume\">1113</infon><infon key=\"year\">2007</infon><offset>32767</offset><text>Heat shock proteins and protection of the nervous system</text></passage><passage><infon key=\"fpage\">645</infon><infon key=\"lpage\">9</infon><infon key=\"name_0\">surname:McMartin;given-names:KE</infon><infon key=\"name_1\">surname:Martin-Amat;given-names:G</infon><infon key=\"name_2\">surname:Noker;given-names:PE</infon><infon key=\"name_3\">surname:Tephly;given-names:TR</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">Biochem Pharm</infon><infon key=\"type\">ref</infon><infon key=\"volume\">28</infon><infon key=\"year\">1978</infon><offset>32824</offset><text>Lack of a role for formaldehyde in methanol poisoning in the monkey</text></passage><passage><infon key=\"fpage\">321</infon><infon key=\"lpage\">30</infon><infon key=\"name_0\">surname:Eells;given-names:JT</infon><infon key=\"name_1\">surname:Henry;given-names:MM</infon><infon key=\"name_2\">surname:Lewandowski;given-names:MF</infon><infon key=\"name_3\">surname:Seme;given-names:MT</infon><infon key=\"name_4\">surname:Murray;given-names:TG</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">Neurotoxicol Rev</infon><infon key=\"type\">ref</infon><infon key=\"volume\">21</infon><infon key=\"year\">2000</infon><offset>32892</offset><text>Development and characterization of a rodent model of methanol-induced retinal and optic nerve toxicity</text></passage><passage><infon key=\"fpage\">123</infon><infon key=\"lpage\">4</infon><infon key=\"name_0\">surname:Clark;given-names:I</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">Nature</infon><infon key=\"type\">ref</infon><infon key=\"volume\">75</infon><infon key=\"year\">1955</infon><offset>32996</offset><text>A colorimetric reaction for the estimation of cortisone, hydrocortisone, aldosterone and related steroids</text></passage><passage><infon key=\"fpage\">67</infon><infon key=\"lpage\">79</infon><infon key=\"name_0\">surname:Bernabe;given-names:Bloj</infon><infon key=\"name_1\">surname:Morero Roberto;given-names:D</infon><infon key=\"name_2\">surname:Farias Ricardo;given-names:N</infon><infon key=\"name_3\">surname:Trucco Raul;given-names:E</infon><infon key=\"pub-id_pmid\">4268761</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">Biochim Biophys Acta</infon><infon key=\"type\">ref</infon><infon key=\"volume\">311</infon><infon key=\"year\">1973</infon><offset>33102</offset><text>Membrane lipid fatty acids and regulation of membrane bound enzymes. Allosteric behaviour of erythrocyte Mg2+ ATPase (Na+/K+)-ATPase and acetylcholine esterase from rats fed different fat supplemental diets</text></passage><passage><infon key=\"fpage\">281</infon><infon key=\"lpage\">8</infon><infon key=\"name_0\">surname:Hjerten;given-names:S</infon><infon key=\"name_1\">surname:Pan;given-names:H</infon><infon key=\"pub-id_pmid\">6219703</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">Biochim Biophys Acta</infon><infon key=\"type\">ref</infon><infon key=\"volume\">728</infon><infon key=\"year\">1983</infon><offset>33309</offset><text>Purification and characterization of two forms of a low-affinity Ca2+-ATPase from erythrocyte membranes</text></passage><passage><infon key=\"fpage\">67</infon><infon key=\"lpage\">74</infon><infon key=\"name_0\">surname:Ohnishi;given-names:T</infon><infon key=\"name_1\">surname:Suzuki;given-names:T</infon><infon key=\"name_2\">surname:Suzuki;given-names:Y</infon><infon key=\"name_3\">surname:Ozawa;given-names:K</infon><infon key=\"pub-id_pmid\">6120003</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">Biochim Biophys Acta</infon><infon key=\"type\">ref</infon><infon key=\"volume\">684</infon><infon key=\"year\">1982</infon><offset>33413</offset><text>A comparative study of plasma membrane Mg2+ ATPase activities in normal, regenerating and malignant cells</text></passage><passage><infon key=\"fpage\">88</infon><infon key=\"lpage\">95</infon><infon key=\"name_0\">surname:Ellman;given-names:GL</infon><infon key=\"name_1\">surname:Courtney;given-names:KD</infon><infon key=\"name_2\">surname:Andres;given-names:V;suffix:Jr</infon><infon key=\"name_3\">surname:Featherstone;given-names:RM</infon><infon key=\"pub-id_pmid\">13726518</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">Biochem Pharmacol</infon><infon key=\"type\">ref</infon><infon key=\"volume\">7</infon><infon key=\"year\">1961</infon><offset>33519</offset><text>A new and rapid colorimetric determination of acetylcholine esterase activity</text></passage><passage><infon key=\"fpage\">426</infon><infon key=\"name_0\">surname:Griess;given-names:P</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">Ber Dtsch Chem Ges</infon><infon key=\"type\">ref</infon><infon key=\"volume\">12</infon><infon key=\"year\">1879</infon><offset>33597</offset></passage><passage><infon key=\"fpage\">246</infon><infon key=\"lpage\">51</infon><infon key=\"name_0\">surname:Stegink;given-names:LD</infon><infon key=\"name_1\">surname:Filter;given-names:LJ;suffix:Jr</infon><infon key=\"name_2\">surname:Baker;given-names:GL</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">Metabolism</infon><infon key=\"type\">ref</infon><infon key=\"volume\">37</infon><infon key=\"year\">1983</infon><offset>33598</offset><text>Repeated ingestion of aspartame sweetened beverage: effect on plasma amino acid concentrations in normal adults</text></passage><passage><infon key=\"fpage\">187</infon><infon key=\"lpage\">9</infon><infon key=\"name_0\">surname:Davoli;given-names:E</infon><infon key=\"pub-id_pmid\">3957170</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">Food Chem Toxicol</infon><infon key=\"type\">ref</infon><infon key=\"volume\">24</infon><infon key=\"year\">1986</infon><offset>33710</offset><text>Serum methanol concentrations in rats and in men after a single dose of aspartame</text></passage><passage><infon key=\"fpage\">325</infon><infon key=\"lpage\">71</infon><infon key=\"name_0\">surname:Oberly;given-names:LW</infon><infon key=\"name_1\">surname:Oberly;given-names:TD</infon><infon key=\"name_2\">surname:Johnson;given-names:JE;suffix:Jr</infon><infon key=\"name_3\">surname:Miquel;given-names:J</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">Free radicals, aging and degenerative diseases</infon><infon key=\"type\">ref</infon><infon key=\"year\">1986</infon><offset>33792</offset><text>Free radicals, cancer and aging</text></passage><passage><infon key=\"fpage\">664</infon><infon key=\"lpage\">6</infon><infon key=\"name_0\">surname:Palmer;given-names:RMJ</infon><infon key=\"name_1\">surname:Ashton;given-names:DS</infon><infon key=\"name_2\">surname:Moncada;given-names:S</infon><infon key=\"pub-id_pmid\">3131684</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">Nature</infon><infon key=\"type\">ref</infon><infon key=\"volume\">333</infon><infon key=\"year\">1988</infon><offset>33824</offset><text>Vascular endothelial cells synthesize nitric oxide from L-arginine</text></passage><passage><infon key=\"fpage\">65218</infon><infon key=\"lpage\">22</infon><infon key=\"name_0\">surname:McCord;given-names:JM</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">Am J Med</infon><infon key=\"type\">ref</infon><infon key=\"volume\">108</infon><infon key=\"year\">2000</infon><offset>33891</offset><text>The evolution of free radicals and oxidative stress</text></passage><passage><infon key=\"fpage\">3024</infon><infon key=\"lpage\">7</infon><infon key=\"name_0\">surname:Korproski;given-names:H</infon><infon key=\"name_1\">surname:Yong;given-names:MZ</infon><infon key=\"name_2\">surname:Ellen;given-names:HK</infon><infon key=\"name_3\">surname:Nigel;given-names:F</infon><infon key=\"name_4\">surname:Lucy;given-names:R</infon><infon key=\"name_5\">surname:Zhen;given-names:FF</infon><infon key=\"pub-id_pmid\">7681993</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">Proc Natl Acad Sci U S A</infon><infon key=\"type\">ref</infon><infon key=\"volume\">90</infon><infon key=\"year\">1993</infon><offset>33943</offset><text>In vivo expression of inducible nitric oxide synthase in experimentally induced neurologic diseases</text></passage><passage><infon key=\"fpage\">215</infon><infon key=\"lpage\">22</infon><infon key=\"name_0\">surname:McIntosh;given-names:LJ</infon><infon key=\"name_1\">surname:Cortopassi;given-names:KM</infon><infon key=\"name_2\">surname:Sapolsky;given-names:RM</infon><infon key=\"pub-id_pmid\">9593900</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">Brain Res</infon><infon key=\"type\">ref</infon><infon key=\"volume\">791</infon><infon key=\"year\">1998</infon><offset>34043</offset><text>Glucocorticoids may alter antioxidant enzyme capacity in the brain: kainic acid studies</text></passage><passage><infon key=\"fpage\">481</infon><infon key=\"lpage\">7</infon><infon key=\"name_0\">surname:Radi;given-names:R</infon><infon key=\"name_1\">surname:Beckman;given-names:JS</infon><infon key=\"name_2\">surname:Bush;given-names:KM</infon><infon key=\"pub-id_pmid\">1654835</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">Arch Biochem Biophys</infon><infon key=\"type\">ref</infon><infon key=\"volume\">288</infon><infon key=\"year\">1991</infon><offset>34131</offset><text>Peroxynitrite-induced membrane lipid peroxidation: the cytotoxic potential of superoxide and nitric oxide</text></passage><passage><infon key=\"fpage\">471</infon><infon key=\"lpage\">6</infon><infon key=\"name_0\">surname:Rohn;given-names:TT</infon><infon key=\"name_1\">surname:Hinds;given-names:TR</infon><infon key=\"name_2\">surname:Vincenzi;given-names:FF</infon><infon key=\"pub-id_pmid\">8619892</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">Biochem Pharmacol</infon><infon key=\"type\">ref</infon><infon key=\"volume\">51</infon><infon key=\"year\">1996</infon><offset>34237</offset><text>ROS and membrane bound ATPases</text></passage><passage><infon key=\"fpage\">761</infon><infon key=\"lpage\">74</infon><infon key=\"name_0\">surname:Polizzi;given-names:S</infon><infon key=\"name_1\">surname:Pira;given-names:E</infon><infon key=\"name_2\">surname:Ferrara;given-names:M</infon><infon key=\"name_3\">surname:Bugiani;given-names:M</infon><infon key=\"name_4\">surname:Papaleo;given-names:A</infon><infon key=\"name_5\">surname:Albera;given-names:R</infon><infon key=\"pub-id_pmid\">12520766</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">Neurotoxicology</infon><infon key=\"type\">ref</infon><infon key=\"volume\">23</infon><infon key=\"year\">2002</infon><offset>34268</offset><text>Neurotoxic effects of aluminium among foundry workers and Alzheimer's disease</text></passage><passage><infon key=\"fpage\">159</infon><infon key=\"issue\">1-2</infon><infon key=\"lpage\">66</infon><infon key=\"name_0\">surname:Pragasam;given-names:V</infon><infon key=\"name_1\">surname:Kalaiselvi;given-names:P</infon><infon key=\"name_2\">surname:Sumitra;given-names:K</infon><infon key=\"name_3\">surname:Srinivasan;given-names:S</infon><infon key=\"name_4\">surname:Varalakshmi;given-names:P</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">Clin Chem Acta</infon><infon key=\"type\">ref</infon><infon key=\"volume\">354</infon><infon key=\"year\">2005</infon><offset>34348</offset><text>Counteraction of oxalate induced nitrosative stress by supplementation of L-arginine, a potent antilithic agent</text></passage><passage><infon key=\"fpage\">866</infon><infon key=\"lpage\">70</infon><infon key=\"name_0\">surname:Bannerman;given-names:DM</infon><infon key=\"name_1\">surname:Deacon;given-names:RM</infon><infon key=\"name_2\">surname:Seeburg;given-names:PH</infon><infon key=\"name_3\">surname:Rawlins;given-names:J</infon><infon key=\"pub-id_pmid\">12931971</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">Behav Neurosci</infon><infon key=\"type\">ref</infon><infon key=\"volume\">117</infon><infon key=\"year\">2003</infon><offset>34460</offset><text>GluR-A-deficient mice display normal acquisition of a hippocampus-dependent spatial reference memory task but are impaired during spatial reversal</text></passage><passage><infon key=\"fpage\">673</infon><infon key=\"lpage\">9</infon><infon key=\"name_0\">surname:Azizi-Malekabadi;given-names:H</infon><infon key=\"name_1\">surname:Hosseini;given-names:M</infon><infon key=\"name_2\">surname:Saffarzadeh;given-names:F</infon><infon key=\"name_3\">surname:Karami;given-names:R</infon><infon key=\"name_4\">surname:Khodabandehloo;given-names:F</infon><infon key=\"pub-id_pmid\">21655764</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">Clinics (Sao Paulo)</infon><infon key=\"type\">ref</infon><infon key=\"volume\">66</infon><infon key=\"year\">2011</infon><offset>34607</offset><text>Chronic treatment with the nitric oxide synthase inhibitor, L-NAME, attenuates estradiol mediated improvement of learning and memory in ovariectomized rats</text></passage><passage><infon key=\"fpage\">138</infon><infon key=\"lpage\">49</infon><infon key=\"name_0\">surname:Udayabanu;given-names:M</infon><infon key=\"name_1\">surname:Kumaran;given-names:D</infon><infon key=\"name_2\">surname:Nair;given-names:RU</infon><infon key=\"name_3\">surname:Srinivas;given-names:P</infon><infon key=\"name_4\">surname:Bhagat;given-names:N</infon><infon key=\"name_5\">surname:Aneja;given-names:R</infon><infon key=\"pub-id_pmid\">18639532</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">Brain Res</infon><infon key=\"type\">ref</infon><infon key=\"volume\">1230</infon><infon key=\"year\">2008</infon><offset>34763</offset><text>Nitric oxide associated with iNOS expression inhibits acetylcholine esterase activity and induces memory impairment during acute hypobaric hypoxia</text></passage><passage><infon key=\"fpage\">155</infon><infon key=\"lpage\">9</infon><infon key=\"name_0\">surname:Irene;given-names:Simintzi</infon><infon key=\"name_1\">surname:Schulpis Kleopatra;given-names:H</infon><infon key=\"name_2\">surname:Panagoula;given-names:Angelogianni</infon><infon key=\"name_3\">surname:Charis;given-names:Liapi</infon><infon key=\"name_4\">surname:Stylianos;given-names:Tsakiris</infon><infon key=\"pub-id_pmid\">17580119</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">Pharmacol Res</infon><infon key=\"type\">ref</infon><infon key=\"volume\">56</infon><infon key=\"year\">2007</infon><offset>34910</offset><text>The effect of aspartame on acetylcholineesterase activity in hippocampal homogenates of suckling rats</text></passage><passage><infon key=\"fpage\">489</infon><infon key=\"lpage\">96</infon><infon key=\"name_0\">surname:Rico;given-names:EP</infon><infon key=\"name_1\">surname:Rosemberg;given-names:DB</infon><infon key=\"name_2\">surname:Senger;given-names:MR</infon><infon key=\"name_3\">surname:Arizi;given-names:M</infon><infon key=\"name_4\">surname:de;given-names:B</infon><infon key=\"name_5\">surname:Bernardi;given-names:GF</infon><infon key=\"pub-id_pmid\">16806813</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">Neurotoxicol Teratol</infon><infon key=\"type\">ref</infon><infon key=\"volume\">28</infon><infon key=\"year\">2006</infon><offset>35012</offset><text>Methanol alters ecto-nucleotidases and acetylcholinesterase in zebrafish brain</text></passage><passage><infon key=\"fpage\">1881</infon><infon key=\"lpage\">94</infon><infon key=\"name_0\">surname:Gil;given-names:GA</infon><infon key=\"name_1\">surname:Bussolino;given-names:DF</infon><infon key=\"name_2\">surname:Portal;given-names:MM</infon><infon key=\"name_3\">surname:Pecchio;given-names:AA</infon><infon key=\"name_4\">surname:Renner;given-names:ML</infon><infon key=\"name_5\">surname:Borioli;given-names:GA</infon><infon key=\"pub-id_pmid\">14767061</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">Mol Biol Cell</infon><infon key=\"type\">ref</infon><infon key=\"volume\">15</infon><infon key=\"year\">2004</infon><offset>35091</offset><text>C-fos activated phospholipid synthesis is required for neurite elongation in differentiating PC12 cells</text></passage><passage><infon key=\"fpage\">116</infon><infon key=\"lpage\">69</infon><infon key=\"name_0\">surname:Smeyne;given-names:RJ</infon><infon key=\"name_1\">surname:Vendrell;given-names:M</infon><infon key=\"name_2\">surname:Hayward;given-names:M</infon><infon key=\"name_3\">surname:Baker;given-names:SJ</infon><infon key=\"name_4\">surname:Miao;given-names:G</infon><infon key=\"name_5\">surname:Schilling;given-names:K</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">Nature</infon><infon key=\"type\">ref</infon><infon key=\"volume\">363</infon><infon key=\"year\">1993</infon><offset>35195</offset><text>Continuous c-fos expression precedes programmed cell death in vivo</text></passage><passage><infon key=\"fpage\">363</infon><infon key=\"lpage\">6</infon><infon key=\"name_0\">surname:Sorger;given-names:PK</infon><infon key=\"pub-id_pmid\">2018972</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">Cell</infon><infon key=\"type\">ref</infon><infon key=\"volume\">65</infon><infon key=\"year\">1991</infon><offset>35262</offset><text>Heat shock factor and the heat shock response</text></passage><passage><infon key=\"fpage\">404</infon><infon key=\"lpage\">16</infon><infon key=\"name_0\">surname:Calabrese;given-names:V</infon><infon key=\"name_1\">surname:Colombrita;given-names:C</infon><infon key=\"name_2\">surname:Sultana;given-names:R</infon><infon key=\"name_3\">surname:Scapagnini;given-names:G</infon><infon key=\"name_4\">surname:Calvani;given-names:M</infon><infon key=\"name_5\">surname:Butterfield;given-names:DA</infon><infon key=\"pub-id_pmid\">16677087</infon><infon key=\"section_type\">REF</infon><infon key=\"source\">Antioxid Redox Signal</infon><infon key=\"type\">ref</infon><infon key=\"volume\">8</infon><infon key=\"year\">2006</infon><offset>35308</offset><text>Redox modulation of heat shock protein expression by acetlcarnitine in aging brain relationship to antioxidant status and mitochondrial function</text></passage><passage><infon key=\"file\">jfda-26-02-903f1.jpg</infon><infon key=\"id\">f1-jfda-26-02-903</infon><infon key=\"section_type\">FIG</infon><infon key=\"type\">fig_caption</infon><offset>35453</offset><text>Long term aspartame consumption interferes the body weight, plasma corticosterone and brain NO and AcHE levels. A, Effect of aspartame (40 mg/kg b.w) on plasma corticosterone level (mM) in rat. B, Long term consumption of aspartame interferes on body weight of the rats. C, Effect of aspartame (40 mg/kg b.w) on nitric oxide levels (mumoles/mg tissue) in rat brain discrete regions. D, Effect of aspartame (40 mg/kg b.w) on acetylcholine esterase levels (mumoles/min/mg protein) in rat brain discrete regions. Comparison and analysis were done by the one-way analysis of variance (ANOVA) (n = 6). The data from various groups for the individual parameters are presented as bar diagram with mean +- SD. Significance fixed at P &lt; 0.05, CC - cerebral cortex, CB - cerebellum, MB - midbrain, PM - pons medulla, HP - hippocampus, HY - hypothalamus. Aspartame treated group when compared to control significance is marked as * and MTX treated groups significance is marked as #.</text></passage><passage><infon key=\"file\">jfda-26-02-903f2.jpg</infon><infon key=\"id\">f2-jfda-26-02-903</infon><infon key=\"section_type\">FIG</infon><infon key=\"type\">fig_caption</infon><offset>36438</offset><text>Long term aspartame consumption (40 mg/kg b.wt) attenuates the learning and memory in Wistar albino rats in both Morris water maze and Y maze. A, Effect of aspartame (40 mg/kg b.wt) during the training period of learning and attenuation of memory. B and C, Long term consumption of aspartame (40 mg/kg b.wt) attenuates the memory retention in probe trial. D, Evaluation of aspartame effect (40 mg/kg b.wt) on spatial working memory by Y maze. E, Evaluation of aspartame effect (40 mg/kg b.wt) on spatial recognition memory by Y maze. Comparison and analysis were done by the one-way analysis of variance (ANOVA) (n = 6) control group was compared with MTX control group and aspartame MTX group, MTX control group was compared with Aspartame MTX group. Data are expressed as mean +- SD, n = 6. *P &lt;= 0.05. Control, MTX control - methotrexate treated group, Asp + MTX - Aspartame + Methotrexate treated group. Aspartame treated group when compared to control significance is marked as * and MTX treated groups significance is marked as #.</text></passage><passage><infon key=\"file\">jfda-26-02-903f3.jpg</infon><infon key=\"id\">f3-jfda-26-02-903</infon><infon key=\"section_type\">FIG</infon><infon key=\"type\">fig_caption</infon><offset>37480</offset><text>Effect of long term aspartame on c-fos, Hsp70, iNOS and nNOS mRNA expression in brain regions of Wistar albino rats. Lanes 1,4,7,10 - Control, Lanes 2,5,8,11 - MTX control, Lanes 3,6,9,12 - MTX + ASP, M-100 bp marker. Lanes 1,2,3 - cerebral cortex, Lanes 4,5,6 - cerebellum, Lanes 7,8,9 - hippocampus, Lanes 10,11,12 - hypothalamus. CC - cerebral cortex, CB - cerebellum, HI - hippocampus, HY - hypothalamus. Comparison and analysis were done by the one-way analysis of variance (ANOVA), control group was compared with MTX control group and aspartame MTX group, MTX control group was compared with Aspartame MTX group. Data are expressed as mean +- SD, n = 6. *P &lt;= 0.05. Control, MTX control - methotrexate treated group, Asp + MTX - Aspartame + Methotrexate treated group. Aspartame treated group when compared to control significance is marked as * and MTX treated groups significance is marked as #.</text></passage><passage><infon key=\"file\">jfda-26-02-903f4.jpg</infon><infon key=\"id\">f4-jfda-26-02-903</infon><infon key=\"section_type\">FIG</infon><infon key=\"type\">fig_caption</infon><offset>38412</offset><text>Effect of long term aspartame on [A] glial cells and astrocytes activation in brain of Wistar albino rats by imunohistochemistry and [B] synaptic markers by western blot [C] mechanism of action on the neuronal memory loss and degeneration. [A] Immunohistomicrograph staining shows the effect of aspartame (40 mg/kg b.w) on iba1 and GFAP in brain of Control, Methotrexate (MTX) control and Aspartame treated animals. Comparison and analysis were done by the one-way analysis of variance (ANOVA), control group was compared with MTX control group and aspartame MTX group, MTX control group was compared with Aspartame MTX group. Data are expressed as mean +- SD, n = 6. *P &lt;= 0.05. [B] Effect of Aspartame (40 mg/kg b.w) on synaptic markers namely PSD-95, Synaptophysin, p-ERK and total ERK. [C] Effect of Aspartame (40 mg/kg b.w) on the mechanism of action in memory processing and neuronal degeneration, the proteins involved namely NMDAR1, CaMKII, CREB and its phosphorylated proteins.</text></passage><passage><infon key=\"file\">jfda-26-02-903f5.jpg</infon><infon key=\"id\">f5-jfda-26-02-903</infon><infon key=\"section_type\">FIG</infon><infon key=\"type\">fig_caption</infon><offset>39400</offset><text>Effect of long term aspartame (40 mg/kg b.wt) on brain in Wistar albino rat, [A] the histomicrograph of brain stained by cresyl fast violet. [B] Imunohistochemistry of Hsp70, nNOS, c-fos and iNOS protein expression in brain of Wistar albino rats. [A] Histomicrograph Cresyl fast violet (CFV) staining of brain in Control, Methotrexate (MTX) control and Aspartame + Methotrexate treated animals. CA1 - Cornus Ammonis 1, CA2 - Cornus Ammonis 2, CA3 - Cornus Ammonis 3, DG - Dentate Gyrus, DNC - Degenerative neuronal cells (DNC), HL - Hippocampal Layer. [B] Immunohistomicrograph staining of brain in Control, Methotrexate (MTX) control and Aspartame + Methotrexate treated animals. Comparison and analysis were done by the one-way analysis of variance (ANOVA), control group was compared with MTX control group and aspartame MTX group, MTX control group was compared with Aspartame MTX group. Data are expressed as mean +- SD, n = 6. *P &lt;= 0.05. Aspartame treated group when compared to control significance is marked as * and MTX treated groups significance is marked as #.</text></passage><passage><infon key=\"file\">jfda-26-02-903f6.jpg</infon><infon key=\"id\">f6-jfda-26-02-903</infon><infon key=\"section_type\">FIG</infon><infon key=\"type\">fig_caption</infon><offset>40487</offset><text>Schematic model for the effect of long term aspartame consumption illustrating the mechanism of action on the neuronal memory loss and neuronal degeneration in the brain.</text></passage><passage><infon key=\"file\">t1-jfda-26-02-903.xml</infon><infon key=\"id\">t1-jfda-26-02-903</infon><infon key=\"section_type\">TABLE</infon><infon key=\"type\">table_caption</infon><offset>40658</offset><text>Effect of Aspartame (40 mg/kg b.wt) on Na+/K+ ATPase, Ca+ ATPase and Mg2+ ATPase (mumoles of phosphorous liberated/min/mg protein) in brain regions.</text></passage><passage><infon key=\"file\">t1-jfda-26-02-903.xml</infon><infon key=\"id\">t1-jfda-26-02-903</infon><infon key=\"section_type\">TABLE</infon><infon key=\"type\">table</infon><infon key=\"xml\">&lt;?xml version=&quot;1.0&quot; encoding=&quot;UTF-8&quot;?&gt;\n",
      "&lt;table frame=&quot;box&quot; rules=&quot;groups&quot;&gt;&lt;thead&gt;&lt;tr&gt;&lt;th valign=&quot;top&quot; align=&quot;left&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;Parameter&lt;/th&gt;&lt;th valign=&quot;top&quot; align=&quot;center&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;Control&lt;/th&gt;&lt;th valign=&quot;top&quot; align=&quot;center&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;MTX treated&lt;/th&gt;&lt;th valign=&quot;top&quot; align=&quot;center&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;Asp + MTX treated&lt;/th&gt;&lt;/tr&gt;&lt;/thead&gt;&lt;tbody&gt;&lt;tr&gt;&lt;td colspan=&quot;4&quot; valign=&quot;top&quot; align=&quot;left&quot; rowspan=&quot;1&quot;&gt;Cerebralcortex&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td valign=&quot;top&quot; align=&quot;left&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt; Na&lt;sup&gt;+&lt;/sup&gt;/K&lt;sup&gt;+&lt;/sup&gt; ATPase&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;center&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;0.73 +- 0.04&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;center&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;0.71 +- 0.04&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;center&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;0.29 +- 0.04*#&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td valign=&quot;top&quot; align=&quot;left&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt; Ca&lt;sup&gt;+&lt;/sup&gt; ATPase&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;center&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;0.44 +- 0.04&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;center&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;0.45 +- 0.04&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;center&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;0.28 +- 0.03*#&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td valign=&quot;top&quot; align=&quot;left&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt; Mg&lt;sup&gt;2+&lt;/sup&gt;&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;center&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;0.70 +- 0.03&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;center&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;0.69 +- 0.02&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;center&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;0.38 +- 0.02*#&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td colspan=&quot;4&quot; valign=&quot;top&quot; align=&quot;left&quot; rowspan=&quot;1&quot;&gt;Cerebellum&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td valign=&quot;top&quot; align=&quot;left&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt; Na&lt;sup&gt;+&lt;/sup&gt;/K&lt;sup&gt;+&lt;/sup&gt; ATPase&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;center&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;0.53 +- 0.04&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;center&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;0.52 +- 0.03&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;center&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;0.35 +- 0.03*#&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td valign=&quot;top&quot; align=&quot;left&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt; Ca&lt;sup&gt;+&lt;/sup&gt; ATPase&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;center&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;0.34 +- 0.04&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;center&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;0.34 +- 0.03&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;center&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;0.20 +- 0.04*#&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td valign=&quot;top&quot; align=&quot;left&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt; Mg&lt;sup&gt;2+&lt;/sup&gt;&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;center&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;0.49 +- 0.03&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;center&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;0.51 +- 0.03&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;center&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;0.22 +- 0.04*#&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td colspan=&quot;4&quot; valign=&quot;top&quot; align=&quot;left&quot; rowspan=&quot;1&quot;&gt;Midbrain&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td valign=&quot;top&quot; align=&quot;left&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt; Na&lt;sup&gt;+&lt;/sup&gt;/K&lt;sup&gt;+&lt;/sup&gt; ATPase&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;center&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;0.70 +- 0.05&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;center&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;0.69 +- 0.03&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;center&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;0.28 +- 0.03*#&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td valign=&quot;top&quot; align=&quot;left&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt; Ca&lt;sup&gt;+&lt;/sup&gt; ATPase&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;center&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;0.45 +- 0.04&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;center&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;0.47 +- 0.04&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;center&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;0.25 +- 0.01*#&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td valign=&quot;top&quot; align=&quot;left&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt; Mg&lt;sup&gt;2+&lt;/sup&gt;&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;center&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;0.53 +- 0.04&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;center&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;0.53 +- 0.04&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;center&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;0.26 +- 0.03*#&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td colspan=&quot;4&quot; valign=&quot;top&quot; align=&quot;left&quot; rowspan=&quot;1&quot;&gt;Pons medulla&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td valign=&quot;top&quot; align=&quot;left&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt; Na&lt;sup&gt;+&lt;/sup&gt;/K&lt;sup&gt;+&lt;/sup&gt; ATPase&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;center&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;0.63 +- 0.03&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;center&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;0.63 +- 0.03&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;center&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;0.41 +- 0.03*#&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td valign=&quot;top&quot; align=&quot;left&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt; Ca&lt;sup&gt;+&lt;/sup&gt; ATPase&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;center&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;0.39 +- 0.04&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;center&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;0.39 +- 0.04&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;center&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;0.27 +- 0.02*#&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td valign=&quot;top&quot; align=&quot;left&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt; Mg&lt;sup&gt;2+&lt;/sup&gt;&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;center&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;0.70 +- 0.02&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;center&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;0.72 +- 0.03&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;center&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;0.38 +- 0.03*#&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td colspan=&quot;4&quot; valign=&quot;top&quot; align=&quot;left&quot; rowspan=&quot;1&quot;&gt;Hippocampus&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td valign=&quot;top&quot; align=&quot;left&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt; Na&lt;sup&gt;+&lt;/sup&gt;/K&lt;sup&gt;+&lt;/sup&gt; ATPase&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;center&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;0.86 +- 0.04&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;center&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;0.85 +- 0.05&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;center&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;0.49 +- 0.03*#&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td valign=&quot;top&quot; align=&quot;left&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt; Ca&lt;sup&gt;+&lt;/sup&gt; ATPase&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;center&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;0.43 +- 0.05&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;center&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;0.41 +- 0.03&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;center&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;0.27 +- 0.02*#&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td valign=&quot;top&quot; align=&quot;left&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt; Mg&lt;sup&gt;2+&lt;/sup&gt;&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;center&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;0.53 +- 0.03&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;center&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;0.51 +- 0.04&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;center&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;0.29 +- 0.04*#&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td colspan=&quot;4&quot; valign=&quot;top&quot; align=&quot;left&quot; rowspan=&quot;1&quot;&gt;Hypothalamus&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td valign=&quot;top&quot; align=&quot;left&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt; Na&lt;sup&gt;+&lt;/sup&gt;/K&lt;sup&gt;+&lt;/sup&gt; ATPase&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;center&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;0.53 +- 0.05&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;center&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;0.50 +- 0.05&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;center&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;0.34 +- 0.03*#&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td valign=&quot;top&quot; align=&quot;left&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt; Ca&lt;sup&gt;+&lt;/sup&gt; ATPase&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;center&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;0.45 +- 0.04&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;center&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;0.44 +- 0.02&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;center&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;0.36 +- 0.02*#&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td valign=&quot;top&quot; align=&quot;left&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt; Mg&lt;sup&gt;2+&lt;/sup&gt;&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;center&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;0.48 +- 0.02&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;center&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;0.47 +- 0.03&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;center&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;0.27 +- 0.03*#&lt;/td&gt;&lt;/tr&gt;&lt;/tbody&gt;&lt;/table&gt;\n",
      "</infon><offset>40807</offset><text>Parameter\tControl\tMTX treated\tAsp + MTX treated\t \tCerebralcortex\t \t Na+/K+ ATPase\t0.73 +- 0.04\t0.71 +- 0.04\t0.29 +- 0.04*#\t \t Ca+ ATPase\t0.44 +- 0.04\t0.45 +- 0.04\t0.28 +- 0.03*#\t \t Mg2+\t0.70 +- 0.03\t0.69 +- 0.02\t0.38 +- 0.02*#\t \tCerebellum\t \t Na+/K+ ATPase\t0.53 +- 0.04\t0.52 +- 0.03\t0.35 +- 0.03*#\t \t Ca+ ATPase\t0.34 +- 0.04\t0.34 +- 0.03\t0.20 +- 0.04*#\t \t Mg2+\t0.49 +- 0.03\t0.51 +- 0.03\t0.22 +- 0.04*#\t \tMidbrain\t \t Na+/K+ ATPase\t0.70 +- 0.05\t0.69 +- 0.03\t0.28 +- 0.03*#\t \t Ca+ ATPase\t0.45 +- 0.04\t0.47 +- 0.04\t0.25 +- 0.01*#\t \t Mg2+\t0.53 +- 0.04\t0.53 +- 0.04\t0.26 +- 0.03*#\t \tPons medulla\t \t Na+/K+ ATPase\t0.63 +- 0.03\t0.63 +- 0.03\t0.41 +- 0.03*#\t \t Ca+ ATPase\t0.39 +- 0.04\t0.39 +- 0.04\t0.27 +- 0.02*#\t \t Mg2+\t0.70 +- 0.02\t0.72 +- 0.03\t0.38 +- 0.03*#\t \tHippocampus\t \t Na+/K+ ATPase\t0.86 +- 0.04\t0.85 +- 0.05\t0.49 +- 0.03*#\t \t Ca+ ATPase\t0.43 +- 0.05\t0.41 +- 0.03\t0.27 +- 0.02*#\t \t Mg2+\t0.53 +- 0.03\t0.51 +- 0.04\t0.29 +- 0.04*#\t \tHypothalamus\t \t Na+/K+ ATPase\t0.53 +- 0.05\t0.50 +- 0.05\t0.34 +- 0.03*#\t \t Ca+ ATPase\t0.45 +- 0.04\t0.44 +- 0.02\t0.36 +- 0.02*#\t \t Mg2+\t0.48 +- 0.02\t0.47 +- 0.03\t0.27 +- 0.03*#\t \t</text></passage><passage><infon key=\"file\">t1-jfda-26-02-903.xml</infon><infon key=\"id\">t1-jfda-26-02-903</infon><infon key=\"section_type\">TABLE</infon><infon key=\"type\">table_footnote</infon><offset>41951</offset><text>The data from various groups for the individual parameters are presented as table with mean +- SD. Significance fixed at P &lt;= 0.05. Aspartame treated group when compared to control*, MTX treated groups #.</text></passage><passage><infon key=\"file\">t1-jfda-26-02-903.xml</infon><infon key=\"id\">t1-jfda-26-02-903</infon><infon key=\"section_type\">TABLE</infon><infon key=\"type\">table_footnote</infon><offset>42157</offset><text>Comparison and analysis were done by the one-way analysis of variance (ANOVA) (n = 6) control group was compared with MTX control group and aspartame MTX group, MTX control group was compared with Aspartame MTX group.</text></passage><passage><infon key=\"file\">t1-jfda-26-02-903.xml</infon><infon key=\"id\">t1-jfda-26-02-903</infon><infon key=\"section_type\">TABLE</infon><infon key=\"type\">table_footnote</infon><offset>42375</offset><text>Control, MTX control - Methotrexate treated group, Asp + MTX - Aspartame + Methotrexate treated group.</text></passage><passage><infon key=\"file\">t2-jfda-26-02-903.xml</infon><infon key=\"id\">t2-jfda-26-02-903</infon><infon key=\"section_type\">TABLE</infon><infon key=\"type\">table_caption</infon><offset>42482</offset><text>LD50 of Aspartame in Wistar albino rats as per Karber's method. The LD50 of aspartame is 10.95 g/kg body weight.</text></passage><passage><infon key=\"file\">t2-jfda-26-02-903.xml</infon><infon key=\"id\">t2-jfda-26-02-903</infon><infon key=\"section_type\">TABLE</infon><infon key=\"type\">table</infon><infon key=\"xml\">&lt;?xml version=&quot;1.0&quot; encoding=&quot;UTF-8&quot;?&gt;\n",
      "&lt;table frame=&quot;box&quot; rules=&quot;groups&quot;&gt;&lt;thead&gt;&lt;tr&gt;&lt;th colspan=&quot;6&quot; valign=&quot;top&quot; align=&quot;left&quot; rowspan=&quot;1&quot;&gt;Karber's method to LD&lt;sub&gt;50&lt;/sub&gt; for Aspartame in Wistar female rat&lt;/th&gt;&lt;/tr&gt;&lt;tr&gt;&lt;th colspan=&quot;6&quot; valign=&quot;top&quot; align=&quot;left&quot; rowspan=&quot;1&quot;&gt;\n",
      "&lt;hr/&gt;&lt;/th&gt;&lt;/tr&gt;&lt;tr&gt;&lt;th valign=&quot;top&quot; align=&quot;left&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;Group&lt;/th&gt;&lt;th valign=&quot;top&quot; align=&quot;center&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;Dose (g/kg)&lt;/th&gt;&lt;th valign=&quot;top&quot; align=&quot;center&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;Dose difference (g/kg) = &lt;italic toggle=&quot;yes&quot;&gt;A&lt;/italic&gt;&lt;/th&gt;&lt;th valign=&quot;top&quot; align=&quot;center&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;Dead&lt;/th&gt;&lt;th valign=&quot;top&quot; align=&quot;center&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;Mean &lt;italic toggle=&quot;yes&quot;&gt;B&lt;/italic&gt;&lt;/th&gt;&lt;th valign=&quot;top&quot; align=&quot;center&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;Product &lt;italic toggle=&quot;yes&quot;&gt;A&lt;/italic&gt;\n",
      "&lt;sup&gt;*&lt;/sup&gt;\n",
      "&lt;italic toggle=&quot;yes&quot;&gt;B&lt;/italic&gt;&lt;/th&gt;&lt;/tr&gt;&lt;/thead&gt;&lt;tbody&gt;&lt;tr&gt;&lt;td valign=&quot;top&quot; align=&quot;left&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;1&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;left&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;12&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;left&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;-&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;left&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;5&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;left&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;5&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;left&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;/&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td valign=&quot;top&quot; align=&quot;left&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;2&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;left&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;9&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;left&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;3&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;left&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;5&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;left&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;5&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;left&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;15&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td valign=&quot;top&quot; align=&quot;left&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;3&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;left&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;6&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;left&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;3&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;left&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;3&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;left&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;1.5&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;left&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;4.5&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td valign=&quot;top&quot; align=&quot;left&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;4&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;left&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;3&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;left&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;1.5&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;left&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;1&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;left&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;0.5&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;left&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;0.75&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td valign=&quot;top&quot; align=&quot;left&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;5&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;left&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;1.5&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;left&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;1.5&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;left&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;0&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;left&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;0&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;left&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;0&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td valign=&quot;top&quot; align=&quot;left&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;/&gt;&lt;td valign=&quot;top&quot; align=&quot;left&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;LD&lt;sub&gt;50&lt;/sub&gt; = 15 - (20.25/5) = 10.95 g/kg&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;left&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;/&gt;&lt;td valign=&quot;top&quot; align=&quot;left&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;/&gt;&lt;td valign=&quot;top&quot; align=&quot;left&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;/&gt;&lt;td valign=&quot;top&quot; align=&quot;left&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;Total = 20.25&lt;/td&gt;&lt;/tr&gt;&lt;/tbody&gt;&lt;/table&gt;\n",
      "</infon><offset>42597</offset><text>Karber's method to LD50 for Aspartame in Wistar female rat\t \t\t \tGroup\tDose (g/kg)\tDose difference (g/kg) = A\tDead\tMean B\tProduct A*B\t \t1\t12\t-\t5\t5\t\t \t2\t9\t3\t5\t5\t15\t \t3\t6\t3\t3\t1.5\t4.5\t \t4\t3\t1.5\t1\t0.5\t0.75\t \t5\t1.5\t1.5\t0\t0\t0\t \t\tLD50 = 15 - (20.25/5) = 10.95 g/kg\t\t\t\tTotal = 20.25\t \t</text></passage><passage><infon key=\"file\">t3-jfda-26-02-903.xml</infon><infon key=\"id\">t3-jfda-26-02-903</infon><infon key=\"section_type\">TABLE</infon><infon key=\"type\">table_caption</infon><offset>42880</offset><text>The sense and antisense primer sequences of the gene of interest for PCR amplification.</text></passage><passage><infon key=\"file\">t3-jfda-26-02-903.xml</infon><infon key=\"id\">t3-jfda-26-02-903</infon><infon key=\"section_type\">TABLE</infon><infon key=\"type\">table</infon><infon key=\"xml\">&lt;?xml version=&quot;1.0&quot; encoding=&quot;UTF-8&quot;?&gt;\n",
      "&lt;table frame=&quot;box&quot; rules=&quot;groups&quot;&gt;&lt;thead&gt;&lt;tr&gt;&lt;th valign=&quot;bottom&quot; align=&quot;left&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;Gene&lt;/th&gt;&lt;th valign=&quot;bottom&quot; align=&quot;center&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;Sequence&lt;/th&gt;&lt;th valign=&quot;bottom&quot; align=&quot;center&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;Amplified product (bp)&lt;/th&gt;&lt;th valign=&quot;bottom&quot; align=&quot;center&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;Annealing temp/cycles&lt;/th&gt;&lt;/tr&gt;&lt;/thead&gt;&lt;tbody&gt;&lt;tr&gt;&lt;td valign=&quot;top&quot; align=&quot;left&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;beta-actin&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;left&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;Sense: TCATGCCATCCTGCGTCTGGACCT&lt;break/&gt;Antisense: CGGACTCATCGTACTCCTGCTTG&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;center&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;598&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;center&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;55  C/35&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td valign=&quot;top&quot; align=&quot;left&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;Hsp70&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;left&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;Sense: GAGTCCTACGCCTTCAATATGAAG&lt;break/&gt;Antisense: CATCAAGAGTCTGTCTCTAGCCAA&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;center&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;347&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;center&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;55  C/35&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td valign=&quot;top&quot; align=&quot;left&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;iNOS&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;left&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;Sense: TCTGTGCCTTTGCTCATGAC&lt;break/&gt;Antisense: CATGGTGAACACGTTCTTGG&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;center&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;305&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;center&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;55  C/35&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td valign=&quot;top&quot; align=&quot;left&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;nNOS&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;left&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;Sense: CCTTCCGAAGCTTCTGGCAACAGC&lt;break/&gt;Antisense: TGGACTCAGATCTAAGGCGGTTGG&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;center&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;474&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;center&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;66  C/35&lt;/td&gt;&lt;/tr&gt;&lt;tr&gt;&lt;td valign=&quot;top&quot; align=&quot;left&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;c-fos&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;left&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;Sense: AGTGGTGAAGACCATGTCAGG&lt;break/&gt;Antisense: CATTGGGGATCTTGCAGG&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;center&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;296&lt;/td&gt;&lt;td valign=&quot;top&quot; align=&quot;center&quot; rowspan=&quot;1&quot; colspan=&quot;1&quot;&gt;55  C/35&lt;/td&gt;&lt;/tr&gt;&lt;/tbody&gt;&lt;/table&gt;\n",
      "</infon><offset>42968</offset><text>Gene\tSequence\tAmplified product (bp)\tAnnealing temp/cycles\t \tbeta-actin\tSense: TCATGCCATCCTGCGTCTGGACCTAntisense: CGGACTCATCGTACTCCTGCTTG\t598\t55  C/35\t \tHsp70\tSense: GAGTCCTACGCCTTCAATATGAAGAntisense: CATCAAGAGTCTGTCTCTAGCCAA\t347\t55  C/35\t \tiNOS\tSense: TCTGTGCCTTTGCTCATGACAntisense: CATGGTGAACACGTTCTTGG\t305\t55  C/35\t \tnNOS\tSense: CCTTCCGAAGCTTCTGGCAACAGCAntisense: TGGACTCAGATCTAAGGCGGTTGG\t474\t66  C/35\t \tc-fos\tSense: AGTGGTGAAGACCATGTCAGGAntisense: CATTGGGGATCTTGCAGG\t296\t55  C/35\t \t</text></passage></document>\n",
      "</collection>\n"
     ]
    }
   ],
   "source": [
    "os.makedirs(\"papers\", exist_ok=True)\n",
    "if os.path.isfile(f\"papers/{PMC}.xml\"):\n",
    "    with open(f\"papers/{PMC}.xml\", 'r', encoding='utf-8') as f:\n",
    "        paper_xml = f.read()\n",
    "else:\n",
    "    paper_xml = utils.fetch_xml(PMC)\n",
    "    with open(f\"papers/{PMC}.xml\", \"w\", encoding=\"utf-8\") as f:\n",
    "        f.write(paper_xml)\n",
    "print(paper_xml)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "027b38a0",
   "metadata": {},
   "source": [
    "Extracts abstract and tables from paper"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "a0d34193",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "c:\\Users\\jelli\\Desktop\\Animal Toxicity Trials\\process_table\\table_utils.py:410: XMLParsedAsHTMLWarning: It looks like you're using an HTML parser to parse an XML document.\n",
      "\n",
      "Assuming this really is an XML document, what you're doing might work, but you should know that using an XML parser will be more reliable. To parse this document as XML, make sure you have the Python package 'lxml' installed, and pass the keyword argument `features=\"xml\"` into the BeautifulSoup constructor.\n",
      "\n",
      "If you want or need to use an HTML parser on this document, you can make this warning go away by filtering it. To do that, run this code before calling the BeautifulSoup constructor:\n",
      "\n",
      "    from bs4 import XMLParsedAsHTMLWarning\n",
      "    import warnings\n",
      "\n",
      "    warnings.filterwarnings(\"ignore\", category=XMLParsedAsHTMLWarning)\n",
      "\n",
      "  soup = BeautifulSoup(html_content, \"html.parser\")\n",
      "c:\\Users\\jelli\\Desktop\\Animal Toxicity Trials\\process_table\\table_utils.py:18: XMLParsedAsHTMLWarning: It looks like you're using an HTML parser to parse an XML document.\n",
      "\n",
      "Assuming this really is an XML document, what you're doing might work, but you should know that using an XML parser will be more reliable. To parse this document as XML, make sure you have the Python package 'lxml' installed, and pass the keyword argument `features=\"xml\"` into the BeautifulSoup constructor.\n",
      "\n",
      "If you want or need to use an HTML parser on this document, you can make this warning go away by filtering it. To do that, run this code before calling the BeautifulSoup constructor:\n",
      "\n",
      "    from bs4 import XMLParsedAsHTMLWarning\n",
      "    import warnings\n",
      "\n",
      "    warnings.filterwarnings(\"ignore\", category=XMLParsedAsHTMLWarning)\n",
      "\n",
      "  soup = BeautifulSoup(html, \"html.parser\")\n"
     ]
    }
   ],
   "source": [
    "from collections import defaultdict\n",
    "\n",
    "soup = BeautifulSoup(paper_xml, \"lxml-xml\")\n",
    "\n",
    "#abstract\n",
    "abstract_passages = [p for p in soup.find_all(\"passage\") if p.find(\"infon\", {\"key\": \"section_type\"}) and \n",
    "        p.find(\"infon\", {\"key\": \"section_type\"}).text.strip().upper() == \"ABSTRACT\"]\n",
    "abstract_text_list = [p.find(\"text\").get_text() for p in abstract_passages if p.find(\"text\")]\n",
    "abstract = \"\\n\".join(abstract_text_list)\n",
    "\n",
    "#methods\n",
    "methods_passages = [p for p in soup.find_all(\"passage\") if p.find(\"infon\", {\"key\": \"section_type\"}) and \n",
    "        p.find(\"infon\", {\"key\": \"section_type\"}).text.strip().upper() == \"METHODS\"]\n",
    "methods_text_list = [p.find(\"text\").get_text() for p in methods_passages if p.find(\"text\")]\n",
    "methods = \"\\n\".join(methods_text_list)\n",
    "\n",
    "#results\n",
    "results_passages = [p for p in soup.find_all(\"passage\") if p.find(\"infon\", {\"key\": \"section_type\"}) and \n",
    "        p.find(\"infon\", {\"key\": \"section_type\"}).text.strip().upper() == \"RESULTS\"]\n",
    "results_text_list = [p.find(\"text\").get_text() for p in results_passages if p.find(\"text\")]\n",
    "results = \"\\n\".join(results_text_list)\n",
    "\n",
    "#tables\n",
    "table_passages = [p for p in soup.find_all(\"passage\") if (sect := p.find(\"infon\", {\"key\": \"section_type\"})) and sect.text.strip() == \"TABLE\"]\n",
    "tables = defaultdict(lambda: {\"label\": \"o\", \"caption\": None, \"markdown\": None, \"footnotes\": [], \"col_labels\": None, \"group_cols\": [], \"biomarker_cols\": []})\n",
    "for p in table_passages:\n",
    "    table_id = p.find(\"infon\", {\"key\": \"id\"}).text.strip()\n",
    "    passage_type = p.find(\"infon\", {\"key\": \"type\"}).text.strip()\n",
    "    if passage_type == \"table_caption\":\n",
    "        tables[table_id][\"caption\"] = p.find(\"text\").text.strip()\n",
    "    elif passage_type == \"table\":\n",
    "        table_xml = p.find(\"infon\", {\"key\": \"xml\"}).text\n",
    "        if \"<table\" not in table_xml:\n",
    "            continue\n",
    "        table = table_utils.single_html_table_to_markdown(html.unescape(table_xml))\n",
    "        tables[table_id][\"markdown\"] = table_utils.transpose_markdown_table(table_utils.transpose_markdown_table(table))\n",
    "    elif passage_type in (\"table_foot\", \"table_footnote\"):\n",
    "        tables[table_id][\"footnotes\"].append(p.find(\"text\").text.strip())"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "13e88191",
   "metadata": {},
   "source": [
    "Generate prompt for identifying each table"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "id": "fd6970b7",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "# Instructions\n",
      "You will be given the abstract of a research paper which includes at least one animal toxicity study then a list of all the tables included in that paper, each one labeled with its title. For each table identify it as one of three categories: treatment group table (G), biomarker table (B), or other (O). Treatment group tables should contain information on the specific treatment groups, such as the medications, the dosages, the sample size, etc. Tables that only give information on the chemicals used (such as the sourcing) should not be labeled G and should instead be labeled O. Even if the table meets these criteria, only label the table G if it describes the groups of specifically an animal toxicity study within the paper. Otherwise, label it O. Biomarker tables may contain treatment group dosage information, but the main difference between is that biomarker tables should contain biomarker data observed from the treatment groups. This can frequency data (such as survival or number occurences of a condition) or metric data (such as ALT levels or compound concentration), and the table must describe the results of specifically an animal toxicity study within the paper to be labeled B. Otherwise, label it O. Any other irrelevant papers should be labeled O, such as results of in vitro assays or gene expression tables. For each table on a new line, include the label in the following format in the order that the tables are given: <table name|label>. Do not include any extra text in your answer or extra whitespace inside the labels. Here is an example input and output:\n",
      "# Example Input\n",
      "<abstract>Abstract\n",
      "Background: Compound ABC is a novel small molecule with potential therapeutic applications in metabolic disorders. To evaluate its safety profile, we conducted two complementary toxicity studies: an in vitro cytotoxicity assay and an in vivo rodent toxicity assessment.\n",
      "Methods:\n",
      "In vitro assay: Human hepatocellular carcinoma (HepG2) and rat cardiomyoblast (H9c2) cell lines were exposed to ABC at concentrations of 1-200 µM for 24, 48, and 72 hours. Cell viability was measured by MTT assay, and apoptosis was assessed by flow cytometry using Annexin V/PI staining and caspase-3 activation.\n",
      "In vivo study: Male and female Sprague-Dawley rats (n = 10 per sex per group) received daily oral doses of ABC at 0 (vehicle control), 10, 50, or 200 mg/kg for 28 days. Clinical observations, body weight, and food consumption were recorded. At study termination, hematology, serum biochemistry (ALT, AST, BUN, creatinine), and organ histopathology (liver, kidney, heart) were evaluated.\n",
      "Results:\n",
      "In vitro, ABC induced a concentration- and time-dependent decrease in cell viability, with IC₅₀ values of 45 µM (HepG2) and 120 µM (H9c2) at 48 h. Apoptotic indices increased significantly at ≥50 µM, accompanied by a 3-fold rise in caspase-3 activity. In vivo, no mortality or adverse clinical signs were observed at ≤50 mg/kg. Rats dosed at 200 mg/kg exhibited transient body weight suppression (5% lower vs. control), mild elevations in ALT and creatinine (1.4- and 1.3-fold, respectively), and minimal hepatocellular vacuolation on histology. No significant changes were seen in hematological parameters or cardiac histopathology.\n",
      "Conclusions: ABC demonstrates moderate cytotoxicity in vitro, primarily via apoptosis, and is well tolerated in rats at doses up to 50 mg/kg/day. The no-observed-adverse-effect level (NOAEL) in this 28-day study is 50 mg/kg. These findings support further preclinical development of ABC, with emphasis on mechanistic studies of hepatic metabolism and long-term safety.</abstract>\n",
      "<example table 1>Caption: Overview of dosing regimens for both in vitro and in vivo studies.\n",
      "| Study | Model | Group | Dose / Concentration | Frequency / Exposure | Route | Replicates (n) |\n",
      "| --- | --- | --- | --- | --- | --- | --- |\n",
      "| In vitro | HepG2 & H9c2 cell lines | Control | 0 µM | 48 h single exposure | N/A | 3 per timepoint |\n",
      "| In vitro | HepG2 & H9c2 cell lines | Low | 10 µM | 48 h single exposure | N/A | 3 per timepoint |\n",
      "| In vitro | HepG2 & H9c2 cell lines | Medium | 50 µM | 48 h single exposure | N/A | 3 per timepoint |\n",
      "| In vitro | HepG2 & H9c2 cell lines | High | 100 µM | 48 h single exposure | N/A | 3 per timepoint |\n",
      "| In vivo | Sprague-Dawley rat | Control | 0 mg/kg/day | Once daily 28 days | Oral gavage | 10 rats/sex/group |\n",
      "| In vivo | Sprague-Dawley rat | Low | 10 mg/kg/day | Once daily 28 days | Oral gavage | 10 rats/sex/group |\n",
      "| In vivo | Sprague-Dawley rat | Medium | 50 mg/kg/day | Once daily 28 days | Oral gavage | 10 rats/sex/group |\n",
      "| In vivo | Sprague-Dawley rat | High | 200 mg/kg/day | Once daily 28 days | Oral gavage | 10 rats/sex/group |\n",
      "Footnotes:\n",
      "1 In vitro replicates refer to independent wells measured per concentration and timepoint.\n",
      "2 In vivo 'n' indicates number of animals per sex in each treatment group.\n",
      "</example table 1>\n",
      "<example table 2>Caption: Summary of cell-based toxicity endpoints following 48 h exposure to ABC.\n",
      "| Group | Viability (% of control) | Apoptotic Cells (%) | Caspase-3 Activity (fold of control) |\n",
      "| --- | --- | --- | --- |\n",
      "| Control (0 µM) | 100 +- 5 | 5 +- 1 | 1.0 +- 0.1 |\n",
      "| Low (10 µM) | 95 +- 4 |  8 +- 2 | 1.2 +- 0.1 |\n",
      "| Medium (50 µM) | 60 +- 6 | 25 +- 3 | 3.0 +- 0.2 |\n",
      "| High (100 µM) | 30 +- 5 | 50 +- 4 | 5.5 +- 0.4 |\n",
      "Footnotes:\n",
      "3 Viability measured by MTT assay, expressed as percent of untreated control.\n",
      "4 Apoptotic fraction determined by Annexin V/PI staining via flow cytometry.\n",
      "5 Caspase-3 activity normalized to control, measured enzymatically.\n",
      "</example table 2>\n",
      "<example table 3>Caption: Key serum biomarker changes in Sprague-Dawley rats after 28 days of ABC dosing.\n",
      "| Group | ALT (U/L) | AST (U/L) | Creatinine (mg/dL) | BUN (mg/dL) |\n",
      "| --- | --- | --- | --- | --- |\n",
      "| Control (0 mg/kg) | 35 +- 5 | 42 +- 6 | 0.6 +- 0.1 | 14 +- 2 |\n",
      "| Low (10 mg/kg) | 38 +- 6 | 45 +- 7 | 0.7 +- 0.1 | 15 +- 2 |\n",
      "| Medium (50 mg/kg) | 45 +- 7 | 52 +- 8 | 0.8 +- 0.1 | 18 +- 3 |\n",
      "| High (200 mg/kg) | 50 +- 8 | 60 +- 10 | 0.9 +- 0.1 | 20 +- 3 |\n",
      "Footnotes:\n",
      "6 Alanine aminotransferase (ALT) and aspartate aminotransferase (AST) reflect hepatic function.\n",
      "7 BUN: blood urea nitrogen, indicates renal function.\n",
      "</example table 3>\n",
      "# Example output\n",
      "<example table 1|G>\n",
      "<example table 2|O>\n",
      "<example table 3|B>\n",
      "# Input\n",
      "<abstract>Many controversial reports are available on the use of aspartame as it releases methanol as one of its metabolite during metabolism. The present study proposes to investigate whether long term (90 days) aspartame (40 mg/kg b.wt) administration could induce oxidative stress and alter the memory in Wistar strain male albino rats. To mimic the human methanol metabolism, methotrexate (MTX)-treated rats were included as a model to study the effects of aspartame. Wistar strain albino rats were administered with aspartame (40 mg/kg b.wt) orally and studied along with controls and MTX-treated controls. Aspartame interfered in the body weight and corticosterone levels in the rats. A marked increase in the mRNA and protein expression of neuronal nitric oxide synthase (nNOS) and induced nitric oxide synthase (iNOS) which resulted in the increased nitric oxide radical's level indicating that aspartame is a stressor. These reactive nitrogen species could be responsible for the altered cell membrane integrity and even cause death of neurons by necrosis or apoptosis. The animals showed a marked decrease in learning, spatial working and spatial recognition memory deficit in the Morris water maze and Y-maze performance task which could have resulted due to reduced hippocampal acetylcholine esterase (AChE) activity. The animal brain homogenate also revealed the decrease in the phosphorylation of NMDAR1-CaMKII-ERK/CREB signalling pathway, which well documents the inhibition of phosphorylation leads to the excitotoxicity of the neurons and memory decline. This effect may be due to methanol which may also activate the NOS levels, microglia and astrocytes, inducing neurodegeneration in brain. Neuronal shrinkage of hippocampal layer due to degeneration of pyramidal cells revealed the abnormal neuronal morphology of pyramidal cell layers in the aspartame treated animals. These findings demonstrate that aspartame metabolites could be a contributing factor for the development of oxidative stress in the brain.</abstract>\n",
      "<t1-jfda-26-02-903>Caption: Effect of Aspartame (40 mg/kg b.wt) on Na+/K+ ATPase, Ca+ ATPase and Mg2+ ATPase (mumoles of phosphorous liberated/min/mg protein) in brain regions.\n",
      "| Parameter | Control | MTX treated | Asp + MTX treated |\n",
      "| --- | --- | --- | --- |\n",
      "| Cerebralcortex | Cerebralcortex | Cerebralcortex | Cerebralcortex |\n",
      "| Na/KATPase | 0.73 +- 0.04 | 0.71 +- 0.04 | 0.29 +- 0.04*# |\n",
      "| CaATPase | 0.44 +- 0.04 | 0.45 +- 0.04 | 0.28 +- 0.03*# |\n",
      "| Mg | 0.70 +- 0.03 | 0.69 +- 0.02 | 0.38 +- 0.02*# |\n",
      "| Cerebellum | Cerebellum | Cerebellum | Cerebellum |\n",
      "| Na/KATPase | 0.53 +- 0.04 | 0.52 +- 0.03 | 0.35 +- 0.03*# |\n",
      "| CaATPase | 0.34 +- 0.04 | 0.34 +- 0.03 | 0.20 +- 0.04*# |\n",
      "| Mg | 0.49 +- 0.03 | 0.51 +- 0.03 | 0.22 +- 0.04*# |\n",
      "| Midbrain | Midbrain | Midbrain | Midbrain |\n",
      "| Na/KATPase | 0.70 +- 0.05 | 0.69 +- 0.03 | 0.28 +- 0.03*# |\n",
      "| CaATPase | 0.45 +- 0.04 | 0.47 +- 0.04 | 0.25 +- 0.01*# |\n",
      "| Mg | 0.53 +- 0.04 | 0.53 +- 0.04 | 0.26 +- 0.03*# |\n",
      "| Pons medulla | Pons medulla | Pons medulla | Pons medulla |\n",
      "| Na/KATPase | 0.63 +- 0.03 | 0.63 +- 0.03 | 0.41 +- 0.03*# |\n",
      "| CaATPase | 0.39 +- 0.04 | 0.39 +- 0.04 | 0.27 +- 0.02*# |\n",
      "| Mg | 0.70 +- 0.02 | 0.72 +- 0.03 | 0.38 +- 0.03*# |\n",
      "| Hippocampus | Hippocampus | Hippocampus | Hippocampus |\n",
      "| Na/KATPase | 0.86 +- 0.04 | 0.85 +- 0.05 | 0.49 +- 0.03*# |\n",
      "| CaATPase | 0.43 +- 0.05 | 0.41 +- 0.03 | 0.27 +- 0.02*# |\n",
      "| Mg | 0.53 +- 0.03 | 0.51 +- 0.04 | 0.29 +- 0.04*# |\n",
      "| Hypothalamus | Hypothalamus | Hypothalamus | Hypothalamus |\n",
      "| Na/KATPase | 0.53 +- 0.05 | 0.50 +- 0.05 | 0.34 +- 0.03*# |\n",
      "| CaATPase | 0.45 +- 0.04 | 0.44 +- 0.02 | 0.36 +- 0.02*# |\n",
      "| Mg | 0.48 +- 0.02 | 0.47 +- 0.03 | 0.27 +- 0.03*# |\n",
      "Footnotes:\n",
      "The data from various groups for the individual parameters are presented as table with mean +- SD. Significance fixed at P <= 0.05. Aspartame treated group when compared to control*, MTX treated groups #.\n",
      "Comparison and analysis were done by the one-way analysis of variance (ANOVA) (n = 6) control group was compared with MTX control group and aspartame MTX group, MTX control group was compared with Aspartame MTX group.\n",
      "Control, MTX control - Methotrexate treated group, Asp + MTX - Aspartame + Methotrexate treated group.\n",
      "</t1-jfda-26-02-903>\n",
      "<t2-jfda-26-02-903>Caption: LD50 of Aspartame in Wistar albino rats as per Karber's method. The LD50 of aspartame is 10.95 g/kg body weight.\n",
      "| Karber's method to LD50for Aspartame in Wistar female rat. Group | Karber's method to LD50for Aspartame in Wistar female rat.Dose (g/kg) | Karber's method to LD50for Aspartame in Wistar female rat.Dose difference (g/kg) =A | Karber's method to LD50for Aspartame in Wistar female rat.Dead | Karber's method to LD50for Aspartame in Wistar female rat.MeanB | Karber's method to LD50for Aspartame in Wistar female rat .ProductAB |\n",
      "| --- | --- | --- | --- | --- | --- |\n",
      "| 1 | 12 | - | 5 | 5 |  |\n",
      "| 2 | 9 | 3 | 5 | 5 | 15 |\n",
      "| 3 | 6 | 3 | 3 | 1.5 | 4.5 |\n",
      "| 4 | 3 | 1.5 | 1 | 0.5 | 0.75 |\n",
      "| 5 | 1.5 | 1.5 | 0 | 0 | 0 |\n",
      "|  | LD50= 15 - (20.25/5) = 10.95 g/kg |  |  |  | Total = 20.25 |\n",
      "</t2-jfda-26-02-903>\n",
      "<t3-jfda-26-02-903>Caption: The sense and antisense primer sequences of the gene of interest for PCR amplification.\n",
      "| Gene | Sequence | Amplified product (bp) | Annealing temp/cycles |\n",
      "| --- | --- | --- | --- |\n",
      "| beta-actin | Sense: TCATGCCATCCTGCGTCTGGACCTAntisense: CGGACTCATCGTACTCCTGCTTG | 598 | 55  C/35 |\n",
      "| Hsp70 | Sense: GAGTCCTACGCCTTCAATATGAAGAntisense: CATCAAGAGTCTGTCTCTAGCCAA | 347 | 55  C/35 |\n",
      "| iNOS | Sense: TCTGTGCCTTTGCTCATGACAntisense: CATGGTGAACACGTTCTTGG | 305 | 55  C/35 |\n",
      "| nNOS | Sense: CCTTCCGAAGCTTCTGGCAACAGCAntisense: TGGACTCAGATCTAAGGCGGTTGG | 474 | 66  C/35 |\n",
      "| c-fos | Sense: AGTGGTGAAGACCATGTCAGGAntisense: CATTGGGGATCTTGCAGG | 296 | 55  C/35 |\n",
      "</t3-jfda-26-02-903>\n",
      "\n"
     ]
    }
   ],
   "source": [
    "id_tables_prompt = utils.id_tables_prompt(abstract, tables)\n",
    "print(id_tables_prompt)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "2ed29d5c",
   "metadata": {},
   "source": [
    "API call to identify tables"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "id": "d8e3bc12",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "https://pharmacoinfo-openai-5.openai.azure.com/openai/deployments/gpt-4o/chat/completions?api-version=2024-10-21\n",
      "ChatGPT Request Status: 200\n"
     ]
    }
   ],
   "source": [
    "id_tables_output = utils.chatgpt_request(id_tables_prompt, 100, 0)\n",
    "\n",
    "time = datetime.now().strftime(\"%Y-%m-%d_%H-%M-%S\")\n",
    "with open(f\"responses/table_id/{time}.txt\", \"w\", encoding=\"utf-8\") as file:\n",
    "    file.write(f\"{PMC}\\n---\\n{id_tables_prompt}\\n---\\n{id_tables_output}\")"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "ca600b6e",
   "metadata": {},
   "source": [
    "Assigning labels to table dict"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "id": "04681700",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "defaultdict(<function <lambda> at 0x000001F826675440>, {'t1-jfda-26-02-903': {'label': 'b', 'caption': 'Effect of Aspartame (40 mg/kg b.wt) on Na+/K+ ATPase, Ca+ ATPase and Mg2+ ATPase (mumoles of phosphorous liberated/min/mg protein) in brain regions.', 'markdown': '| Parameter | Control | MTX treated | Asp + MTX treated |\\n| --- | --- | --- | --- |\\n| Cerebralcortex | Cerebralcortex | Cerebralcortex | Cerebralcortex |\\n| Na/KATPase | 0.73 +- 0.04 | 0.71 +- 0.04 | 0.29 +- 0.04*# |\\n| CaATPase | 0.44 +- 0.04 | 0.45 +- 0.04 | 0.28 +- 0.03*# |\\n| Mg | 0.70 +- 0.03 | 0.69 +- 0.02 | 0.38 +- 0.02*# |\\n| Cerebellum | Cerebellum | Cerebellum | Cerebellum |\\n| Na/KATPase | 0.53 +- 0.04 | 0.52 +- 0.03 | 0.35 +- 0.03*# |\\n| CaATPase | 0.34 +- 0.04 | 0.34 +- 0.03 | 0.20 +- 0.04*# |\\n| Mg | 0.49 +- 0.03 | 0.51 +- 0.03 | 0.22 +- 0.04*# |\\n| Midbrain | Midbrain | Midbrain | Midbrain |\\n| Na/KATPase | 0.70 +- 0.05 | 0.69 +- 0.03 | 0.28 +- 0.03*# |\\n| CaATPase | 0.45 +- 0.04 | 0.47 +- 0.04 | 0.25 +- 0.01*# |\\n| Mg | 0.53 +- 0.04 | 0.53 +- 0.04 | 0.26 +- 0.03*# |\\n| Pons medulla | Pons medulla | Pons medulla | Pons medulla |\\n| Na/KATPase | 0.63 +- 0.03 | 0.63 +- 0.03 | 0.41 +- 0.03*# |\\n| CaATPase | 0.39 +- 0.04 | 0.39 +- 0.04 | 0.27 +- 0.02*# |\\n| Mg | 0.70 +- 0.02 | 0.72 +- 0.03 | 0.38 +- 0.03*# |\\n| Hippocampus | Hippocampus | Hippocampus | Hippocampus |\\n| Na/KATPase | 0.86 +- 0.04 | 0.85 +- 0.05 | 0.49 +- 0.03*# |\\n| CaATPase | 0.43 +- 0.05 | 0.41 +- 0.03 | 0.27 +- 0.02*# |\\n| Mg | 0.53 +- 0.03 | 0.51 +- 0.04 | 0.29 +- 0.04*# |\\n| Hypothalamus | Hypothalamus | Hypothalamus | Hypothalamus |\\n| Na/KATPase | 0.53 +- 0.05 | 0.50 +- 0.05 | 0.34 +- 0.03*# |\\n| CaATPase | 0.45 +- 0.04 | 0.44 +- 0.02 | 0.36 +- 0.02*# |\\n| Mg | 0.48 +- 0.02 | 0.47 +- 0.03 | 0.27 +- 0.03*# |', 'footnotes': ['The data from various groups for the individual parameters are presented as table with mean +- SD. Significance fixed at P <= 0.05. Aspartame treated group when compared to control*, MTX treated groups #.', 'Comparison and analysis were done by the one-way analysis of variance (ANOVA) (n = 6) control group was compared with MTX control group and aspartame MTX group, MTX control group was compared with Aspartame MTX group.', 'Control, MTX control - Methotrexate treated group, Asp + MTX - Aspartame + Methotrexate treated group.'], 'col_labels': None, 'group_cols': [], 'biomarker_cols': []}, 't2-jfda-26-02-903': {'label': 'g', 'caption': \"LD50 of Aspartame in Wistar albino rats as per Karber's method. The LD50 of aspartame is 10.95 g/kg body weight.\", 'markdown': \"| Karber's method to LD50for Aspartame in Wistar female rat. Group | Karber's method to LD50for Aspartame in Wistar female rat.Dose (g/kg) | Karber's method to LD50for Aspartame in Wistar female rat.Dose difference (g/kg) =A | Karber's method to LD50for Aspartame in Wistar female rat.Dead | Karber's method to LD50for Aspartame in Wistar female rat.MeanB | Karber's method to LD50for Aspartame in Wistar female rat .ProductAB |\\n| --- | --- | --- | --- | --- | --- |\\n| 1 | 12 | - | 5 | 5 |  |\\n| 2 | 9 | 3 | 5 | 5 | 15 |\\n| 3 | 6 | 3 | 3 | 1.5 | 4.5 |\\n| 4 | 3 | 1.5 | 1 | 0.5 | 0.75 |\\n| 5 | 1.5 | 1.5 | 0 | 0 | 0 |\\n|  | LD50= 15 - (20.25/5) = 10.95 g/kg |  |  |  | Total = 20.25 |\", 'footnotes': [], 'col_labels': None, 'group_cols': [], 'biomarker_cols': []}, 't3-jfda-26-02-903': {'label': 'o', 'caption': 'The sense and antisense primer sequences of the gene of interest for PCR amplification.', 'markdown': '| Gene | Sequence | Amplified product (bp) | Annealing temp/cycles |\\n| --- | --- | --- | --- |\\n| beta-actin | Sense: TCATGCCATCCTGCGTCTGGACCTAntisense: CGGACTCATCGTACTCCTGCTTG | 598 | 55  C/35 |\\n| Hsp70 | Sense: GAGTCCTACGCCTTCAATATGAAGAntisense: CATCAAGAGTCTGTCTCTAGCCAA | 347 | 55  C/35 |\\n| iNOS | Sense: TCTGTGCCTTTGCTCATGACAntisense: CATGGTGAACACGTTCTTGG | 305 | 55  C/35 |\\n| nNOS | Sense: CCTTCCGAAGCTTCTGGCAACAGCAntisense: TGGACTCAGATCTAAGGCGGTTGG | 474 | 66  C/35 |\\n| c-fos | Sense: AGTGGTGAAGACCATGTCAGGAntisense: CATTGGGGATCTTGCAGG | 296 | 55  C/35 |', 'footnotes': [], 'col_labels': None, 'group_cols': [], 'biomarker_cols': []}})\n"
     ]
    }
   ],
   "source": [
    "for label in id_tables_output.split(\"\\n\"):\n",
    "    label = label.strip()\n",
    "    if not label: continue\n",
    "\n",
    "    content = label[1:-1]\n",
    "    tid, label = content.split(\"|\")\n",
    "    tables[tid.strip()][\"label\"] = label.strip().lower()\n",
    "\n",
    "print(tables)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "1ba414a4",
   "metadata": {},
   "source": [
    "Formatting Tables and converting to df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "id": "18736fce",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "['Cerebralcortex', 'Cerebralcortex', 'Cerebralcortex', 'Cerebellum', 'Cerebellum', 'Cerebellum', 'Midbrain', 'Midbrain', 'Midbrain', 'Pons medulla', 'Pons medulla', 'Pons medulla', 'Hippocampus', 'Hippocampus', 'Hippocampus', 'Hypothalamus', 'Hypothalamus', 'Hypothalamus']\n",
      "https://pharmacoinfo-openai-5.openai.azure.com/openai/deployments/gpt-4o/chat/completions?api-version=2024-10-21\n",
      "ChatGPT Request Status: 200\n",
      "###t1-jfda-26-02-903###\n",
      "| Header.Parameter | Cerebralcortex.Na/KATPase.mean | Cerebralcortex.Na/KATPase.SD | Cerebralcortex.CaATPase.mean | Cerebralcortex.CaATPase.SD | Cerebralcortex.Mg.mean | Cerebralcortex.Mg.SD | Cerebellum.Na/KATPase.mean | Cerebellum.Na/KATPase.SD | Cerebellum.CaATPase.mean | Cerebellum.CaATPase.SD | Cerebellum.Mg.mean | Cerebellum.Mg.SD | Midbrain.Na/KATPase.mean | Midbrain.Na/KATPase.SD | Midbrain.CaATPase.mean | Midbrain.CaATPase.SD | Midbrain.Mg.mean | Midbrain.Mg.SD | Pons medulla.Na/KATPase.mean | Pons medulla.Na/KATPase.SD | Pons medulla.CaATPase.mean | Pons medulla.CaATPase.SD | Pons medulla.Mg.mean | Pons medulla.Mg.SD | Hippocampus.Na/KATPase.mean | Hippocampus.Na/KATPase.SD | Hippocampus.CaATPase.mean | Hippocampus.CaATPase.SD | Hippocampus.Mg.mean | Hippocampus.Mg.SD | Hypothalamus.Na/KATPase.mean | Hypothalamus.Na/KATPase.SD | Hypothalamus.CaATPase.mean | Hypothalamus.CaATPase.SD |\n",
      "| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |\n",
      "| Control | 0.73 | 0.04 | 0.44 | 0.04 | 0.70 | 0.03 | 0.53 | 0.04 | 0.34 | 0.04 | 0.49 | 0.03 | 0.70 | 0.05 | 0.45 | 0.04 | 0.53 | 0.04 | 0.63 | 0.03 | 0.39 | 0.04 | 0.70 | 0.02 | 0.86 | 0.04 | 0.43 | 0.05 | 0.53 | 0.03 | 0.53 | 0.05 | 0.45 | 0.04 |\n",
      "| MTX treated | 0.71 | 0.04 | 0.45 | 0.04 | 0.69 | 0.02 | 0.52 | 0.03 | 0.34 | 0.03 | 0.51 | 0.03 | 0.69 | 0.03 | 0.47 | 0.04 | 0.53 | 0.04 | 0.63 | 0.03 | 0.39 | 0.04 | 0.72 | 0.03 | 0.85 | 0.05 | 0.41 | 0.03 | 0.51 | 0.04 | 0.50 | 0.05 | 0.44 | 0.02 |\n",
      "| Asp + MTX treated | 0.29 | 0.04*# | 0.28 | 0.03*# | 0.38 | 0.02*# | 0.35 | 0.03*# | 0.20 | 0.04*# | 0.22 | 0.04*# | 0.28 | 0.03*# | 0.25 | 0.01*# | 0.26 | 0.03*# | 0.41 | 0.03*# | 0.27 | 0.02*# | 0.38 | 0.03*# | 0.49 | 0.03*# | 0.27 | 0.02*# | 0.29 | 0.04*# | 0.34 | 0.03*# | 0.36 | 0.02*# |\n",
      "###t2-jfda-26-02-903###\n",
      "| Karber's method to LD50for Aspartame in Wistar female rat. Group | Karber's method to LD50for Aspartame in Wistar female rat.Dose (g/kg) | Karber's method to LD50for Aspartame in Wistar female rat.Dose difference (g/kg) =A | Karber's method to LD50for Aspartame in Wistar female rat.Dead | Karber's method to LD50for Aspartame in Wistar female rat.MeanB | Karber's method to LD50for Aspartame in Wistar female rat .ProductAB |\n",
      "| --- | --- | --- | --- | --- | --- |\n",
      "| 1 | 12 | - | 5 | 5 |  |\n",
      "| 2 | 9 | 3 | 5 | 5 | 15 |\n",
      "| 3 | 6 | 3 | 3 | 1.5 | 4.5 |\n",
      "| 4 | 3 | 1.5 | 1 | 0.5 | 0.75 |\n",
      "| 5 | 1.5 | 1.5 | 0 | 0 | 0 |\n",
      "|  | LD50= 15 - (20.25/5) = 10.95 g/kg |  |  |  | Total = 20.25 |\n",
      "###t3-jfda-26-02-903###\n",
      "| Gene | Sequence | Amplified product (bp) | Annealing temp/cycles |\n",
      "| --- | --- | --- | --- |\n",
      "| beta-actin | Sense: TCATGCCATCCTGCGTCTGGACCTAntisense: CGGACTCATCGTACTCCTGCTTG | 598 | 55  C/35 |\n",
      "| Hsp70 | Sense: GAGTCCTACGCCTTCAATATGAAGAntisense: CATCAAGAGTCTGTCTCTAGCCAA | 347 | 55  C/35 |\n",
      "| iNOS | Sense: TCTGTGCCTTTGCTCATGACAntisense: CATGGTGAACACGTTCTTGG | 305 | 55  C/35 |\n",
      "| nNOS | Sense: CCTTCCGAAGCTTCTGGCAACAGCAntisense: TGGACTCAGATCTAAGGCGGTTGG | 474 | 66  C/35 |\n",
      "| c-fos | Sense: AGTGGTGAAGACCATGTCAGGAntisense: CATTGGGGATCTTGCAGG | 296 | 55  C/35 |\n"
     ]
    }
   ],
   "source": [
    "#Note: 1 LLM API call per table\n",
    "formatted_tables = utils.format_tables(PMC, abstract, methods, tables)\n",
    "\n",
    "for tid, info in formatted_tables.items():\n",
    "    print(f\"###{tid}###\")\n",
    "    if info[\"markdown\"]: print(info[\"markdown\"])\n",
    "    info[\"df\"] = table_utils.markdown_to_dataframe(info[\"markdown\"])"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "936f7f0f",
   "metadata": {},
   "source": [
    "IDing columns via LLM"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "id": "b863bdda",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "https://pharmacoinfo-openai-5.openai.azure.com/openai/deployments/gpt-4o/chat/completions?api-version=2024-10-21\n",
      "ChatGPT Request Status: 200\n"
     ]
    }
   ],
   "source": [
    "for tid, info in formatted_tables.items():\n",
    "    if info[\"label\"] != \"b\": continue\n",
    "    info_combined = \"\"\n",
    "    if not info[\"markdown\"]: continue\n",
    "    if info[\"caption\"]:\n",
    "        info_combined += f\"Caption: {info[\"caption\"]}\\n\"\n",
    "    info_combined += info[\"markdown\"] + \"\\n\"\n",
    "    if len(info[\"footnotes\"]) > 0:\n",
    "        info_combined += \"Footnotes:\\n\"\n",
    "        for fn in info[\"footnotes\"]:\n",
    "            info_combined += fn + \"\\n\"\n",
    "    prompt = utils.id_columns_prompt(info_combined)\n",
    "    answer = utils.chatgpt_request(prompt, 5000, 0)\n",
    "    info[\"col_labels\"] = [[elt.strip() for elt in label.strip().lstrip('<').rstrip('>').split(\"|\")] for label in answer.split(\"\\n\")]\n",
    "    time = datetime.now().strftime(\"%Y-%m-%d_%H-%M-%S\")\n",
    "    with open(f\"responses/column_id/{time}.txt\", \"w\", encoding=\"utf-8\") as file:\n",
    "        file.write(f\"{PMC}\\n---\\n{tid}\\n---\\n{prompt}\\n---\\n{answer}\")"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "906282bd",
   "metadata": {},
   "source": [
    "Processing column IDs"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "id": "53ca598a",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "group\n",
      "biomarker\n",
      "biomarker\n",
      "biomarker\n",
      "biomarker\n",
      "biomarker\n",
      "biomarker\n",
      "biomarker\n",
      "biomarker\n",
      "biomarker\n",
      "biomarker\n",
      "biomarker\n",
      "biomarker\n",
      "biomarker\n",
      "biomarker\n",
      "biomarker\n",
      "biomarker\n",
      "biomarker\n",
      "biomarker\n",
      "biomarker\n",
      "biomarker\n",
      "biomarker\n",
      "biomarker\n",
      "biomarker\n",
      "biomarker\n",
      "biomarker\n",
      "biomarker\n",
      "biomarker\n",
      "biomarker\n",
      "biomarker\n",
      "biomarker\n",
      "biomarker\n",
      "biomarker\n",
      "biomarker\n",
      "biomarker\n"
     ]
    }
   ],
   "source": [
    "for tid, info in formatted_tables.items():\n",
    "    if info[\"label\"] != \"b\":\n",
    "        continue\n",
    "    info[\"biomarker_cols\"] = []\n",
    "    info[\"group_cols\"] = []\n",
    "    column_info = {}\n",
    "    for label in info[\"col_labels\"]:\n",
    "        if label[1].lower() not in [\"biomarker\", \"group\", \"dose\", \"size\", \"animal\", \"time\"]: continue\n",
    "        print(label[1].lower())\n",
    "        if label[1].lower() == \"biomarker\":\n",
    "            if label[3].lower() not in [\"mean\", \"variation\", \"frequency\", \"severity\"]: continue\n",
    "            biomarker_label = {\"column\": label[0], \"label\": \"biomarker\", \"type\": label[3].lower(), \"units\": None, \"name\": label[2]}\n",
    "            if label[3].lower() == \"mean\" and len(label) > 4:\n",
    "                biomarker_label[\"units\"] = label[4]\n",
    "            elif label[3].lower() == \"variation\" and len(label) > 4:\n",
    "                biomarker_label[\"units\"] = label[4]\n",
    "            elif label[3].lower() == \"frequency\":\n",
    "                if len(label) > 4 and label[4].lower() in [\"percent\", \"percents\", \"percentage\", \"percentages\", \"%\", \"count\", \"counts\", \"decimal\", \"decimals\"]:\n",
    "                    biomarker_label[\"units\"] = label[4]\n",
    "                else:\n",
    "                    print(\"ERROR: Invalid type for count label -> skipping\")\n",
    "                    continue\n",
    "            info[\"biomarker_cols\"].append(biomarker_label)\n",
    "        else:\n",
    "            group_label = {\"column\": label[0], \"label\": label[1].strip().lower(), \"units\": [], \"link\": None, \"name\": None}\n",
    "            if label[1].lower() == \"dose\":\n",
    "                for i in range(2,len(label)):\n",
    "                    if label[i].strip()[:5].lower() == \"name:\":\n",
    "                        group_label[\"name\"] = label[i].strip()[5:].strip()\n",
    "                    elif label[i].strip()[:6].lower() == \"units:\":\n",
    "                        group_label[\"units\"].append(label[i].strip()[6:].strip())\n",
    "                    elif label[i].strip()[:5].lower() == \"link:\":\n",
    "                        group_label[\"link\"] = label[i].strip()[5:].strip()\n",
    "            elif label[1].lower() == \"animal\":\n",
    "                if len(label) > 2:\n",
    "                    group_label[\"name\"] = label[2]\n",
    "            info[\"group_cols\"].append(group_label)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "id": "1fd62af1",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[['Header.Parameter', 'group'], ['Cerebralcortex.Na/KATPase.mean', 'biomarker', 'Cerebral cortex Na+/K+ ATPase', 'mean', 'mumoles/min/mg protein'], ['Cerebralcortex.Na/KATPase.SD', 'biomarker', 'Cerebral cortex Na+/K+ ATPase', 'variation', 'mumoles/min/mg protein'], ['Cerebralcortex.CaATPase.mean', 'biomarker', 'Cerebral cortex Ca+ ATPase', 'mean', 'mumoles/min/mg protein'], ['Cerebralcortex.CaATPase.SD', 'biomarker', 'Cerebral cortex Ca+ ATPase', 'variation', 'mumoles/min/mg protein'], ['Cerebralcortex.Mg.mean', 'biomarker', 'Cerebral cortex Mg2+ ATPase', 'mean', 'mumoles/min/mg protein'], ['Cerebralcortex.Mg.SD', 'biomarker', 'Cerebral cortex Mg2+ ATPase', 'variation', 'mumoles/min/mg protein'], ['Cerebellum.Na/KATPase.mean', 'biomarker', 'Cerebellum Na+/K+ ATPase', 'mean', 'mumoles/min/mg protein'], ['Cerebellum.Na/KATPase.SD', 'biomarker', 'Cerebellum Na+/K+ ATPase', 'variation', 'mumoles/min/mg protein'], ['Cerebellum.CaATPase.mean', 'biomarker', 'Cerebellum Ca+ ATPase', 'mean', 'mumoles/min/mg protein'], ['Cerebellum.CaATPase.SD', 'biomarker', 'Cerebellum Ca+ ATPase', 'variation', 'mumoles/min/mg protein'], ['Cerebellum.Mg.mean', 'biomarker', 'Cerebellum Mg2+ ATPase', 'mean', 'mumoles/min/mg protein'], ['Cerebellum.Mg.SD', 'biomarker', 'Cerebellum Mg2+ ATPase', 'variation', 'mumoles/min/mg protein'], ['Midbrain.Na/KATPase.mean', 'biomarker', 'Midbrain Na+/K+ ATPase', 'mean', 'mumoles/min/mg protein'], ['Midbrain.Na/KATPase.SD', 'biomarker', 'Midbrain Na+/K+ ATPase', 'variation', 'mumoles/min/mg protein'], ['Midbrain.CaATPase.mean', 'biomarker', 'Midbrain Ca+ ATPase', 'mean', 'mumoles/min/mg protein'], ['Midbrain.CaATPase.SD', 'biomarker', 'Midbrain Ca+ ATPase', 'variation', 'mumoles/min/mg protein'], ['Midbrain.Mg.mean', 'biomarker', 'Midbrain Mg2+ ATPase', 'mean', 'mumoles/min/mg protein'], ['Midbrain.Mg.SD', 'biomarker', 'Midbrain Mg2+ ATPase', 'variation', 'mumoles/min/mg protein'], ['Pons medulla.Na/KATPase.mean', 'biomarker', 'Pons medulla Na+/K+ ATPase', 'mean', 'mumoles/min/mg protein'], ['Pons medulla.Na/KATPase.SD', 'biomarker', 'Pons medulla Na+/K+ ATPase', 'variation', 'mumoles/min/mg protein'], ['Pons medulla.CaATPase.mean', 'biomarker', 'Pons medulla Ca+ ATPase', 'mean', 'mumoles/min/mg protein'], ['Pons medulla.CaATPase.SD', 'biomarker', 'Pons medulla Ca+ ATPase', 'variation', 'mumoles/min/mg protein'], ['Pons medulla.Mg.mean', 'biomarker', 'Pons medulla Mg2+ ATPase', 'mean', 'mumoles/min/mg protein'], ['Pons medulla.Mg.SD', 'biomarker', 'Pons medulla Mg2+ ATPase', 'variation', 'mumoles/min/mg protein'], ['Hippocampus.Na/KATPase.mean', 'biomarker', 'Hippocampus Na+/K+ ATPase', 'mean', 'mumoles/min/mg protein'], ['Hippocampus.Na/KATPase.SD', 'biomarker', 'Hippocampus Na+/K+ ATPase', 'variation', 'mumoles/min/mg protein'], ['Hippocampus.CaATPase.mean', 'biomarker', 'Hippocampus Ca+ ATPase', 'mean', 'mumoles/min/mg protein'], ['Hippocampus.CaATPase.SD', 'biomarker', 'Hippocampus Ca+ ATPase', 'variation', 'mumoles/min/mg protein'], ['Hippocampus.Mg.mean', 'biomarker', 'Hippocampus Mg2+ ATPase', 'mean', 'mumoles/min/mg protein'], ['Hippocampus.Mg.SD', 'biomarker', 'Hippocampus Mg2+ ATPase', 'variation', 'mumoles/min/mg protein'], ['Hypothalamus.Na/KATPase.mean', 'biomarker', 'Hypothalamus Na+/K+ ATPase', 'mean', 'mumoles/min/mg protein'], ['Hypothalamus.Na/KATPase.SD', 'biomarker', 'Hypothalamus Na+/K+ ATPase', 'variation', 'mumoles/min/mg protein'], ['Hypothalamus.CaATPase.mean', 'biomarker', 'Hypothalamus Ca+ ATPase', 'mean', 'mumoles/min/mg protein'], ['Hypothalamus.CaATPase.SD', 'biomarker', 'Hypothalamus Ca+ ATPase', 'variation', 'mumoles/min/mg protein']]\n",
      "GROUP COLUMNS:\n",
      "[{'column': 'Header.Parameter', 'label': 'group', 'units': [], 'link': None, 'name': None}]\n",
      "BIOMARKER COLUMNS:\n",
      "[{'column': 'Cerebralcortex.Na/KATPase.mean', 'label': 'biomarker', 'type': 'mean', 'units': 'mumoles/min/mg protein', 'name': 'Cerebral cortex Na+/K+ ATPase'}, {'column': 'Cerebralcortex.Na/KATPase.SD', 'label': 'biomarker', 'type': 'variation', 'units': 'mumoles/min/mg protein', 'name': 'Cerebral cortex Na+/K+ ATPase'}, {'column': 'Cerebralcortex.CaATPase.mean', 'label': 'biomarker', 'type': 'mean', 'units': 'mumoles/min/mg protein', 'name': 'Cerebral cortex Ca+ ATPase'}, {'column': 'Cerebralcortex.CaATPase.SD', 'label': 'biomarker', 'type': 'variation', 'units': 'mumoles/min/mg protein', 'name': 'Cerebral cortex Ca+ ATPase'}, {'column': 'Cerebralcortex.Mg.mean', 'label': 'biomarker', 'type': 'mean', 'units': 'mumoles/min/mg protein', 'name': 'Cerebral cortex Mg2+ ATPase'}, {'column': 'Cerebralcortex.Mg.SD', 'label': 'biomarker', 'type': 'variation', 'units': 'mumoles/min/mg protein', 'name': 'Cerebral cortex Mg2+ ATPase'}, {'column': 'Cerebellum.Na/KATPase.mean', 'label': 'biomarker', 'type': 'mean', 'units': 'mumoles/min/mg protein', 'name': 'Cerebellum Na+/K+ ATPase'}, {'column': 'Cerebellum.Na/KATPase.SD', 'label': 'biomarker', 'type': 'variation', 'units': 'mumoles/min/mg protein', 'name': 'Cerebellum Na+/K+ ATPase'}, {'column': 'Cerebellum.CaATPase.mean', 'label': 'biomarker', 'type': 'mean', 'units': 'mumoles/min/mg protein', 'name': 'Cerebellum Ca+ ATPase'}, {'column': 'Cerebellum.CaATPase.SD', 'label': 'biomarker', 'type': 'variation', 'units': 'mumoles/min/mg protein', 'name': 'Cerebellum Ca+ ATPase'}, {'column': 'Cerebellum.Mg.mean', 'label': 'biomarker', 'type': 'mean', 'units': 'mumoles/min/mg protein', 'name': 'Cerebellum Mg2+ ATPase'}, {'column': 'Cerebellum.Mg.SD', 'label': 'biomarker', 'type': 'variation', 'units': 'mumoles/min/mg protein', 'name': 'Cerebellum Mg2+ ATPase'}, {'column': 'Midbrain.Na/KATPase.mean', 'label': 'biomarker', 'type': 'mean', 'units': 'mumoles/min/mg protein', 'name': 'Midbrain Na+/K+ ATPase'}, {'column': 'Midbrain.Na/KATPase.SD', 'label': 'biomarker', 'type': 'variation', 'units': 'mumoles/min/mg protein', 'name': 'Midbrain Na+/K+ ATPase'}, {'column': 'Midbrain.CaATPase.mean', 'label': 'biomarker', 'type': 'mean', 'units': 'mumoles/min/mg protein', 'name': 'Midbrain Ca+ ATPase'}, {'column': 'Midbrain.CaATPase.SD', 'label': 'biomarker', 'type': 'variation', 'units': 'mumoles/min/mg protein', 'name': 'Midbrain Ca+ ATPase'}, {'column': 'Midbrain.Mg.mean', 'label': 'biomarker', 'type': 'mean', 'units': 'mumoles/min/mg protein', 'name': 'Midbrain Mg2+ ATPase'}, {'column': 'Midbrain.Mg.SD', 'label': 'biomarker', 'type': 'variation', 'units': 'mumoles/min/mg protein', 'name': 'Midbrain Mg2+ ATPase'}, {'column': 'Pons medulla.Na/KATPase.mean', 'label': 'biomarker', 'type': 'mean', 'units': 'mumoles/min/mg protein', 'name': 'Pons medulla Na+/K+ ATPase'}, {'column': 'Pons medulla.Na/KATPase.SD', 'label': 'biomarker', 'type': 'variation', 'units': 'mumoles/min/mg protein', 'name': 'Pons medulla Na+/K+ ATPase'}, {'column': 'Pons medulla.CaATPase.mean', 'label': 'biomarker', 'type': 'mean', 'units': 'mumoles/min/mg protein', 'name': 'Pons medulla Ca+ ATPase'}, {'column': 'Pons medulla.CaATPase.SD', 'label': 'biomarker', 'type': 'variation', 'units': 'mumoles/min/mg protein', 'name': 'Pons medulla Ca+ ATPase'}, {'column': 'Pons medulla.Mg.mean', 'label': 'biomarker', 'type': 'mean', 'units': 'mumoles/min/mg protein', 'name': 'Pons medulla Mg2+ ATPase'}, {'column': 'Pons medulla.Mg.SD', 'label': 'biomarker', 'type': 'variation', 'units': 'mumoles/min/mg protein', 'name': 'Pons medulla Mg2+ ATPase'}, {'column': 'Hippocampus.Na/KATPase.mean', 'label': 'biomarker', 'type': 'mean', 'units': 'mumoles/min/mg protein', 'name': 'Hippocampus Na+/K+ ATPase'}, {'column': 'Hippocampus.Na/KATPase.SD', 'label': 'biomarker', 'type': 'variation', 'units': 'mumoles/min/mg protein', 'name': 'Hippocampus Na+/K+ ATPase'}, {'column': 'Hippocampus.CaATPase.mean', 'label': 'biomarker', 'type': 'mean', 'units': 'mumoles/min/mg protein', 'name': 'Hippocampus Ca+ ATPase'}, {'column': 'Hippocampus.CaATPase.SD', 'label': 'biomarker', 'type': 'variation', 'units': 'mumoles/min/mg protein', 'name': 'Hippocampus Ca+ ATPase'}, {'column': 'Hippocampus.Mg.mean', 'label': 'biomarker', 'type': 'mean', 'units': 'mumoles/min/mg protein', 'name': 'Hippocampus Mg2+ ATPase'}, {'column': 'Hippocampus.Mg.SD', 'label': 'biomarker', 'type': 'variation', 'units': 'mumoles/min/mg protein', 'name': 'Hippocampus Mg2+ ATPase'}, {'column': 'Hypothalamus.Na/KATPase.mean', 'label': 'biomarker', 'type': 'mean', 'units': 'mumoles/min/mg protein', 'name': 'Hypothalamus Na+/K+ ATPase'}, {'column': 'Hypothalamus.Na/KATPase.SD', 'label': 'biomarker', 'type': 'variation', 'units': 'mumoles/min/mg protein', 'name': 'Hypothalamus Na+/K+ ATPase'}, {'column': 'Hypothalamus.CaATPase.mean', 'label': 'biomarker', 'type': 'mean', 'units': 'mumoles/min/mg protein', 'name': 'Hypothalamus Ca+ ATPase'}, {'column': 'Hypothalamus.CaATPase.SD', 'label': 'biomarker', 'type': 'variation', 'units': 'mumoles/min/mg protein', 'name': 'Hypothalamus Ca+ ATPase'}]\n"
     ]
    }
   ],
   "source": [
    "for tid, info in formatted_tables.items():\n",
    "    if info[\"label\"] != \"b\": continue\n",
    "    print(info[\"col_labels\"])\n",
    "    print(f\"GROUP COLUMNS:\\n{info[\"group_cols\"]}\")\n",
    "    print(f\"BIOMARKER COLUMNS:\\n{info[\"biomarker_cols\"]}\")"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "240cf8d6",
   "metadata": {},
   "source": [
    "Deriving Groups via Tables"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "id": "01466c29",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'column': 'Header.Parameter', 'label': 'group', 'units': [], 'link': None, 'name': None}\n",
      "['control']\n",
      "{'column': 'Header.Parameter', 'label': 'group', 'units': [], 'link': None, 'name': None}\n",
      "['control', 'mtx treated']\n",
      "{'column': 'Header.Parameter', 'label': 'group', 'units': [], 'link': None, 'name': None}\n",
      "['control', 'mtx treated', 'asp + mtx treated']\n",
      "                               group animal_model sample_size terminal_time  \\\n",
      "control                      Control         None        None          None   \n",
      "mtx treated              MTX treated         None        None          None   \n",
      "asp + mtx treated  Asp + MTX treated         None        None          None   \n",
      "\n",
      "                  treatment1 dose1 units1 treatment2 dose2 units2  \n",
      "control                 None  None   None       None  None   None  \n",
      "mtx treated             None  None   None       None  None   None  \n",
      "asp + mtx treated       None  None   None       None  None   None  \n"
     ]
    }
   ],
   "source": [
    "for tid, info in formatted_tables.items():\n",
    "    if info[\"label\"] != \"b\": continue\n",
    "    table_groups = []\n",
    "    group_names = []\n",
    "    info[\"df\"].reset_index(drop=True, inplace=True)\n",
    "    for idx, row in info[\"df\"].iterrows():\n",
    "        group_row = {\"group\": None, \"animal_model\": None, \"sample_size\": None, \"terminal_time\": None,\"treatment1\": None, \"dose1\": None, \"units1\": None,\"treatment2\": None, \"dose2\": None, \"units2\": None}\n",
    "        group_name = \"\"\n",
    "        for label in info[\"group_cols\"]:\n",
    "            print(label)\n",
    "            col_val = info[\"df\"].loc[idx, label[\"column\"]]\n",
    "            if label[\"label\"] == \"group\":\n",
    "                group_name += f\".{col_val}\"\n",
    "            elif label[\"label\"] == \"dose\":\n",
    "                if not group_row[\"dose1\"]:\n",
    "                    group_row[\"dose1\"] = col_val\n",
    "                    if label[\"name\"]:\n",
    "                        group_row[\"treatment1\"] = label[\"name\"]\n",
    "                    elif label[\"link\"]:\n",
    "                        group_row[\"treatment1\"] = info[\"df\"].loc[idx, label[\"link\"]]\n",
    "                elif not group_row[\"dose2\"]:\n",
    "                    group_row[\"dose2\"] = col_val\n",
    "                    if label[\"name\"]:\n",
    "                        group_row[\"treatment2\"] = label[\"name\"]\n",
    "                    elif label[\"link\"]:\n",
    "                        group_row[\"treatment2\"] = info[\"df\"].loc[idx, label[\"link\"]]\n",
    "                else:\n",
    "                    print(\"ERROR: >2 treatments\")\n",
    "            elif label[\"label\"] == \"size\":\n",
    "                group_row[\"sample_size\"] = col_val\n",
    "            elif label[\"label\"] == \"animal\":\n",
    "                group_row[\"animal_model\"] = col_val\n",
    "            elif label[\"label\"] == \"time\":\n",
    "                group_row[\"terminal_time\"] = col_val\n",
    "        table_groups.append(group_row)\n",
    "        group_name = group_name.lstrip(\".\") if group_name else f\"Group {idx}\"\n",
    "        group_names.append(group_name.lower() if group_name else group_name)\n",
    "        group_row[\"group\"] = group_name\n",
    "        print(group_names)\n",
    "    info[\"df\"].index = [name.lower().strip() for name in group_names]\n",
    "    info[\"table_groups\"] = pd.DataFrame(data=table_groups, index=group_names)\n",
    "    print(info[\"table_groups\"])\n",
    "\n",
    "# group_label = {\"column\": label[0], \"label\": label[1].strip().lower(), \"type\": None, \"units\": [], \"link\": None, \"name\": None}"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "5fa3d359",
   "metadata": {},
   "source": [
    "Deriving Groups form Text via LLM"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "id": "db40feaf",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "https://pharmacoinfo-openai-5.openai.azure.com/openai/deployments/gpt-4o/chat/completions?api-version=2024-10-21\n",
      "ChatGPT Request Status: 200\n",
      "<control|rat|6|90 days||||||>  \n",
      "<MTXcontrol|rat|6|90 days|methotrexate|0.2|mg/kg||||>  \n",
      "<MTXASP|rat|6|90 days|methotrexate|0.2|mg/kg|aspartame|40|mg/kg>  \n",
      "[['control', 'rat', '6', '90 days', '', '', '', '', '', ''], ['MTXcontrol', 'rat', '6', '90 days', 'methotrexate', '0.2', 'mg/kg', '', '', '', ''], ['MTXASP', 'rat', '6', '90 days', 'methotrexate', '0.2', 'mg/kg', 'aspartame', '40', 'mg/kg']]\n"
     ]
    }
   ],
   "source": [
    "text_groups = []\n",
    "group_names = []\n",
    "for row in utils.groups_from_text(PMC, abstract, methods, results):\n",
    "    if len(row) not in [4, 7, 10] and len(row) < 10:\n",
    "        print(\"ERROR: Invalid text_groups label\")\n",
    "        continue\n",
    "    text_group = {}\n",
    "    text_group[\"group\"] = row[0]\n",
    "    text_group[\"animal_model\"] = row[1] if row[1] else None\n",
    "    text_group[\"sample_size\"] = row[2] if row[2] else None\n",
    "    text_group[\"terminal_time\"] = row[3] if row[3] else None\n",
    "    if len(row) == 4:\n",
    "        text_group[\"treatment1\"] = None\n",
    "        text_group[\"dose1\"] = None\n",
    "        text_group[\"units1\"] = None\n",
    "        text_group[\"treatment2\"] = None\n",
    "        text_group[\"dose2\"] = None\n",
    "        text_group[\"units2\"] = None\n",
    "    elif len(row) == 7:\n",
    "        text_group[\"treatment1\"] = row[4]\n",
    "        text_group[\"dose1\"] = row[5]\n",
    "        text_group[\"units1\"] = row[6]\n",
    "        text_group[\"treatment2\"] = None\n",
    "        text_group[\"dose2\"] = None\n",
    "        text_group[\"units2\"] = None\n",
    "    else:\n",
    "        text_group[\"treatment1\"] = row[4]\n",
    "        text_group[\"dose1\"] = row[5]\n",
    "        text_group[\"units1\"] = row[6]\n",
    "        text_group[\"treatment2\"] = row[7]\n",
    "        text_group[\"dose2\"] = row[8]\n",
    "        text_group[\"units2\"] = row[9]\n",
    "    text_groups.append(text_group)\n",
    "    group_names.append(row[0].strip().lower())\n",
    "text_groups_df = pd.DataFrame(data=text_groups, index=group_names)\n",
    "    "
   ]
  },
  {
   "cell_type": "markdown",
   "id": "8da82e0a",
   "metadata": {},
   "source": [
    "Linking table and text treatment groups"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "d2dd7ab1",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "https://pharmacoinfo-openai-5.openai.azure.com/openai/deployments/gpt-4o/chat/completions?api-version=2024-10-21\n",
      "ChatGPT Request Status: 200\n"
     ]
    }
   ],
   "source": [
    "for tid, info in formatted_tables.items():\n",
    "    if info[\"label\"] != \"b\": continue\n",
    "    utils.link_groups(PMC, tid, methods, info[\"table_groups\"], text_groups_df)\n",
    "    print(table_utils.dataframe_to_markdown(info[\"table_groups\"]))"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "ab178d08",
   "metadata": {},
   "source": [
    "Process Biomarkers"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "d8545838",
   "metadata": {},
   "outputs": [],
   "source": [
    "for tid, info in formatted_tables.items():\n",
    "    if info[\"label\"] != \"b\": continue\n",
    "    mean_columns = defaultdict(lambda: {\"mean_col\": None, \"variance_col\": None, \"units\": None})\n",
    "    freq_columns = defaultdict(lambda: {\"freq_col\": None, \"severity_col\": None, \"units\": None})\n",
    "    for label in info[\"biomarker_cols\"]:\n",
    "        if label[\"type\"] == \"mean\":\n",
    "            mean_columns[label[\"name\"]][\"mean_col\"] = label[\"column\"]\n",
    "            if label[\"units\"]: mean_columns[label[\"name\"]][\"units\"] = label[\"units\"]\n",
    "        elif label[\"type\"] == \"variance\" or label[\"type\"] == \"variation\":\n",
    "            mean_columns[label[\"name\"]][\"variance_col\"] = label[\"column\"]\n",
    "            if label[\"units\"]: mean_columns[label[\"name\"]][\"units\"] = label[\"units\"]\n",
    "        elif label[\"type\"] == \"frequency\":\n",
    "            freq_columns[label[\"name\"]][\"freq_col\"] = label[\"column\"]\n",
    "            if label[\"units\"]: mean_columns[label[\"name\"]][\"units\"] = label[\"units\"]\n",
    "        elif label[\"type\"] == \"severity\":\n",
    "            freq_columns[label[\"name\"]][\"severity_col\"] = label[\"column\"]\n",
    "    info[\"mean_cols\"] = mean_columns\n",
    "    info[\"freq_cols\"] = freq_columns"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "id": "77aee84f",
   "metadata": {},
   "outputs": [],
   "source": [
    "for tid, info in formatted_tables.items():\n",
    "    if info[\"label\"] != \"b\": continue\n",
    "    table_df = table_utils.markdown_to_dataframe(info[\"markdown\"])\n",
    "    table_df.index = info[\"table_groups\"].index\n",
    "    output_mean = []\n",
    "    output_freq = []\n",
    "    for row_index in table_df.index:\n",
    "        for col_index, col_params in info[\"mean_cols\"].items():\n",
    "            if not table_df.at[row_index, col_params[\"mean_col\"]]: continue\n",
    "            output_mean.append({\n",
    "                \"Animal Model\": info[\"table_groups\"].at[row_index, \"animal_model\"],\n",
    "                \"Sample Size\": info[\"table_groups\"].at[row_index, \"sample_size\"],\n",
    "                \"Treatment 1\": info[\"table_groups\"].at[row_index, \"treatment1\"],\n",
    "                \"Dose 1\": info[\"table_groups\"].at[row_index, \"dose1\"],\n",
    "                \"Units 1\": info[\"table_groups\"].at[row_index, \"units1\"],\n",
    "                \"Treatment 2\": info[\"table_groups\"].at[row_index, \"treatment2\"],\n",
    "                \"Dose 2\": info[\"table_groups\"].at[row_index, \"dose2\"],\n",
    "                \"Units 2\": info[\"table_groups\"].at[row_index, \"units2\"],\n",
    "                \"Terminal Time\": info[\"table_groups\"].at[row_index, \"terminal_time\"],\n",
    "                \"Biomarker\": col_index,\n",
    "                \"Value\": table_df.at[row_index, col_params[\"mean_col\"]],\n",
    "                \"Units\": col_params[\"units\"],\n",
    "                \"Variance\": table_df.at[row_index, col_params[\"variance_col\"]] if col_params[\"variance_col\"] else None,\n",
    "            })\n",
    "        for col_index, col_params in info[\"freq_cols\"].items():\n",
    "            if not table_df.at[row_index, col_params[\"freq_col\"]] and not table_df.at[row_index, col_params[\"severity_col\"]]: continue\n",
    "            if table_df.at[row_index, col_params[\"freq_col\"]]:\n",
    "                if col_params[\"units\"] in [\"percent\", \"percents\", \"percentage\", \"percentages\", \"%\"]:\n",
    "                    frequency = table_df.at[row_index, col_params[\"frequency\"]]/info[\"table_groups\"].at[row_index, \"sample_size\"] * 100\n",
    "                elif col_params[\"units\"] in [\"count\", \"counts\"]:\n",
    "                    frequency = table_df.at[row_index, col_params[\"frequency\"]]/info[\"table_groups\"].at[row_index, \"sample_size\"]\n",
    "                else:\n",
    "                    frequency = table_df.at[row_index, col_params[\"frequency\"]]\n",
    "            output_mean.append({\n",
    "                \"Animal Model\": info[\"table_groups\"].at[row_index, \"animal_model\"],\n",
    "                \"Sample Size\": info[\"table_groups\"].at[row_index, \"sample_size\"],\n",
    "                \"Treatment 1\": info[\"table_groups\"].at[row_index, \"treatment1\"],\n",
    "                \"Dose 1\": info[\"table_groups\"].at[row_index, \"dose1\"],\n",
    "                \"Units 1\": info[\"table_groups\"].at[row_index, \"units1\"],\n",
    "                \"Treatment 2\": info[\"table_groups\"].at[row_index, \"treatment2\"],\n",
    "                \"Dose 2\": info[\"table_groups\"].at[row_index, \"dose2\"],\n",
    "                \"Units 2\": info[\"table_groups\"].at[row_index, \"units2\"],\n",
    "                \"Terminal Time\": info[\"table_groups\"].at[row_index, \"terminal_time\"],\n",
    "                \"Biomarker\": col_index,\n",
    "                \"Frequency\": frequency,\n",
    "                \"Severity\": table_df.at[row_index, col_params[\"variance_col\"]],\n",
    "            })\n",
    "    info[\"output_mean\"] = pd.DataFrame(output_mean)\n",
    "    info[\"output_freq\"] = pd.DataFrame(output_freq)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "id": "d13a4166",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "   Animal Model Sample Size   Treatment 1 Dose 1 Units 1 Treatment 2 Dose 2  \\\n",
      "0           rat           6                                                   \n",
      "1           rat           6                                                   \n",
      "2           rat           6                                                   \n",
      "3           rat           6                                                   \n",
      "4           rat           6                                                   \n",
      "5           rat           6                                                   \n",
      "6           rat           6                                                   \n",
      "7           rat           6                                                   \n",
      "8           rat           6                                                   \n",
      "9           rat           6                                                   \n",
      "10          rat           6                                                   \n",
      "11          rat           6                                                   \n",
      "12          rat           6                                                   \n",
      "13          rat           6                                                   \n",
      "14          rat           6                                                   \n",
      "15          rat           6                                                   \n",
      "16          rat           6                                                   \n",
      "17          rat           6  methotrexate    0.2   mg/kg                      \n",
      "18          rat           6  methotrexate    0.2   mg/kg                      \n",
      "19          rat           6  methotrexate    0.2   mg/kg                      \n",
      "20          rat           6  methotrexate    0.2   mg/kg                      \n",
      "21          rat           6  methotrexate    0.2   mg/kg                      \n",
      "22          rat           6  methotrexate    0.2   mg/kg                      \n",
      "23          rat           6  methotrexate    0.2   mg/kg                      \n",
      "24          rat           6  methotrexate    0.2   mg/kg                      \n",
      "25          rat           6  methotrexate    0.2   mg/kg                      \n",
      "26          rat           6  methotrexate    0.2   mg/kg                      \n",
      "27          rat           6  methotrexate    0.2   mg/kg                      \n",
      "28          rat           6  methotrexate    0.2   mg/kg                      \n",
      "29          rat           6  methotrexate    0.2   mg/kg                      \n",
      "30          rat           6  methotrexate    0.2   mg/kg                      \n",
      "31          rat           6  methotrexate    0.2   mg/kg                      \n",
      "32          rat           6  methotrexate    0.2   mg/kg                      \n",
      "33          rat           6  methotrexate    0.2   mg/kg                      \n",
      "34          rat           6  methotrexate    0.2   mg/kg   aspartame     40   \n",
      "35          rat           6  methotrexate    0.2   mg/kg   aspartame     40   \n",
      "36          rat           6  methotrexate    0.2   mg/kg   aspartame     40   \n",
      "37          rat           6  methotrexate    0.2   mg/kg   aspartame     40   \n",
      "38          rat           6  methotrexate    0.2   mg/kg   aspartame     40   \n",
      "39          rat           6  methotrexate    0.2   mg/kg   aspartame     40   \n",
      "40          rat           6  methotrexate    0.2   mg/kg   aspartame     40   \n",
      "41          rat           6  methotrexate    0.2   mg/kg   aspartame     40   \n",
      "42          rat           6  methotrexate    0.2   mg/kg   aspartame     40   \n",
      "43          rat           6  methotrexate    0.2   mg/kg   aspartame     40   \n",
      "44          rat           6  methotrexate    0.2   mg/kg   aspartame     40   \n",
      "45          rat           6  methotrexate    0.2   mg/kg   aspartame     40   \n",
      "46          rat           6  methotrexate    0.2   mg/kg   aspartame     40   \n",
      "47          rat           6  methotrexate    0.2   mg/kg   aspartame     40   \n",
      "48          rat           6  methotrexate    0.2   mg/kg   aspartame     40   \n",
      "49          rat           6  methotrexate    0.2   mg/kg   aspartame     40   \n",
      "50          rat           6  methotrexate    0.2   mg/kg   aspartame     40   \n",
      "\n",
      "   Units 2 Terminal Time                      Biomarker Value  \\\n",
      "0                90 days  Cerebral cortex Na+/K+ ATPase  0.73   \n",
      "1                90 days     Cerebral cortex Ca+ ATPase  0.44   \n",
      "2                90 days    Cerebral cortex Mg2+ ATPase  0.70   \n",
      "3                90 days       Cerebellum Na+/K+ ATPase  0.53   \n",
      "4                90 days          Cerebellum Ca+ ATPase  0.34   \n",
      "5                90 days         Cerebellum Mg2+ ATPase  0.49   \n",
      "6                90 days         Midbrain Na+/K+ ATPase  0.70   \n",
      "7                90 days            Midbrain Ca+ ATPase  0.45   \n",
      "8                90 days           Midbrain Mg2+ ATPase  0.53   \n",
      "9                90 days     Pons medulla Na+/K+ ATPase  0.63   \n",
      "10               90 days        Pons medulla Ca+ ATPase  0.39   \n",
      "11               90 days       Pons medulla Mg2+ ATPase  0.70   \n",
      "12               90 days      Hippocampus Na+/K+ ATPase  0.86   \n",
      "13               90 days         Hippocampus Ca+ ATPase  0.43   \n",
      "14               90 days        Hippocampus Mg2+ ATPase  0.53   \n",
      "15               90 days     Hypothalamus Na+/K+ ATPase  0.53   \n",
      "16               90 days        Hypothalamus Ca+ ATPase  0.45   \n",
      "17               90 days  Cerebral cortex Na+/K+ ATPase  0.71   \n",
      "18               90 days     Cerebral cortex Ca+ ATPase  0.45   \n",
      "19               90 days    Cerebral cortex Mg2+ ATPase  0.69   \n",
      "20               90 days       Cerebellum Na+/K+ ATPase  0.52   \n",
      "21               90 days          Cerebellum Ca+ ATPase  0.34   \n",
      "22               90 days         Cerebellum Mg2+ ATPase  0.51   \n",
      "23               90 days         Midbrain Na+/K+ ATPase  0.69   \n",
      "24               90 days            Midbrain Ca+ ATPase  0.47   \n",
      "25               90 days           Midbrain Mg2+ ATPase  0.53   \n",
      "26               90 days     Pons medulla Na+/K+ ATPase  0.63   \n",
      "27               90 days        Pons medulla Ca+ ATPase  0.39   \n",
      "28               90 days       Pons medulla Mg2+ ATPase  0.72   \n",
      "29               90 days      Hippocampus Na+/K+ ATPase  0.85   \n",
      "30               90 days         Hippocampus Ca+ ATPase  0.41   \n",
      "31               90 days        Hippocampus Mg2+ ATPase  0.51   \n",
      "32               90 days     Hypothalamus Na+/K+ ATPase  0.50   \n",
      "33               90 days        Hypothalamus Ca+ ATPase  0.44   \n",
      "34   mg/kg       90 days  Cerebral cortex Na+/K+ ATPase  0.29   \n",
      "35   mg/kg       90 days     Cerebral cortex Ca+ ATPase  0.28   \n",
      "36   mg/kg       90 days    Cerebral cortex Mg2+ ATPase  0.38   \n",
      "37   mg/kg       90 days       Cerebellum Na+/K+ ATPase  0.35   \n",
      "38   mg/kg       90 days          Cerebellum Ca+ ATPase  0.20   \n",
      "39   mg/kg       90 days         Cerebellum Mg2+ ATPase  0.22   \n",
      "40   mg/kg       90 days         Midbrain Na+/K+ ATPase  0.28   \n",
      "41   mg/kg       90 days            Midbrain Ca+ ATPase  0.25   \n",
      "42   mg/kg       90 days           Midbrain Mg2+ ATPase  0.26   \n",
      "43   mg/kg       90 days     Pons medulla Na+/K+ ATPase  0.41   \n",
      "44   mg/kg       90 days        Pons medulla Ca+ ATPase  0.27   \n",
      "45   mg/kg       90 days       Pons medulla Mg2+ ATPase  0.38   \n",
      "46   mg/kg       90 days      Hippocampus Na+/K+ ATPase  0.49   \n",
      "47   mg/kg       90 days         Hippocampus Ca+ ATPase  0.27   \n",
      "48   mg/kg       90 days        Hippocampus Mg2+ ATPase  0.29   \n",
      "49   mg/kg       90 days     Hypothalamus Na+/K+ ATPase  0.34   \n",
      "50   mg/kg       90 days        Hypothalamus Ca+ ATPase  0.36   \n",
      "\n",
      "                     Units Variance  \n",
      "0   mumoles/min/mg protein     None  \n",
      "1   mumoles/min/mg protein     None  \n",
      "2   mumoles/min/mg protein     None  \n",
      "3   mumoles/min/mg protein     None  \n",
      "4   mumoles/min/mg protein     None  \n",
      "5   mumoles/min/mg protein     None  \n",
      "6   mumoles/min/mg protein     None  \n",
      "7   mumoles/min/mg protein     None  \n",
      "8   mumoles/min/mg protein     None  \n",
      "9   mumoles/min/mg protein     None  \n",
      "10  mumoles/min/mg protein     None  \n",
      "11  mumoles/min/mg protein     None  \n",
      "12  mumoles/min/mg protein     None  \n",
      "13  mumoles/min/mg protein     None  \n",
      "14  mumoles/min/mg protein     None  \n",
      "15  mumoles/min/mg protein     None  \n",
      "16  mumoles/min/mg protein     None  \n",
      "17  mumoles/min/mg protein     None  \n",
      "18  mumoles/min/mg protein     None  \n",
      "19  mumoles/min/mg protein     None  \n",
      "20  mumoles/min/mg protein     None  \n",
      "21  mumoles/min/mg protein     None  \n",
      "22  mumoles/min/mg protein     None  \n",
      "23  mumoles/min/mg protein     None  \n",
      "24  mumoles/min/mg protein     None  \n",
      "25  mumoles/min/mg protein     None  \n",
      "26  mumoles/min/mg protein     None  \n",
      "27  mumoles/min/mg protein     None  \n",
      "28  mumoles/min/mg protein     None  \n",
      "29  mumoles/min/mg protein     None  \n",
      "30  mumoles/min/mg protein     None  \n",
      "31  mumoles/min/mg protein     None  \n",
      "32  mumoles/min/mg protein     None  \n",
      "33  mumoles/min/mg protein     None  \n",
      "34  mumoles/min/mg protein     None  \n",
      "35  mumoles/min/mg protein     None  \n",
      "36  mumoles/min/mg protein     None  \n",
      "37  mumoles/min/mg protein     None  \n",
      "38  mumoles/min/mg protein     None  \n",
      "39  mumoles/min/mg protein     None  \n",
      "40  mumoles/min/mg protein     None  \n",
      "41  mumoles/min/mg protein     None  \n",
      "42  mumoles/min/mg protein     None  \n",
      "43  mumoles/min/mg protein     None  \n",
      "44  mumoles/min/mg protein     None  \n",
      "45  mumoles/min/mg protein     None  \n",
      "46  mumoles/min/mg protein     None  \n",
      "47  mumoles/min/mg protein     None  \n",
      "48  mumoles/min/mg protein     None  \n",
      "49  mumoles/min/mg protein     None  \n",
      "50  mumoles/min/mg protein     None  \n",
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: []\n"
     ]
    }
   ],
   "source": [
    "for tid, info in formatted_tables.items():\n",
    "    if info[\"label\"] != \"b\": continue\n",
    "    print(info[\"output_mean\"])\n",
    "    print(info[\"output_freq\"])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "ee89a743",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "env",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.12.10"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
